<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="grantpubcite">
<title>Case study: specific projects â€¢ grantpubcite</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.2.2/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.2.2/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Case study: specific projects">
<meta property="og:description" content="grantpubcite">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">grantpubcite</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="Released version">0.0.1.9004</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="../articles/grantpubcite.html">Get started</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="active nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/case_study_iotn.html">Case study: IOTN</a>
    <a class="dropdown-item" href="../articles/case_study_itcr.html">Case study: ITCR</a>
    <a class="dropdown-item" href="../articles/case_study_project_nums_quick.html">Case study: specific projects</a>
    <a class="dropdown-item" href="../articles/implementation_notes.html">Implementation notes</a>
  </div>
</li>
<li class="nav-item">
  <a class="nav-link" href="../news/index.html">Changelog</a>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/mtmorgan/grantpubcite/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-article">



<script src="case_study_project_nums_quick_files/htmlwidgets-1.6.2/htmlwidgets.js"></script><link href="case_study_project_nums_quick_files/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="case_study_project_nums_quick_files/datatables-binding-0.27/datatables.js"></script><link href="case_study_project_nums_quick_files/dt-core-1.12.1/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="case_study_project_nums_quick_files/dt-core-1.12.1/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="case_study_project_nums_quick_files/dt-core-1.12.1/js/jquery.dataTables.min.js"></script><link href="case_study_project_nums_quick_files/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet">
<script src="case_study_project_nums_quick_files/crosstalk-1.2.0/js/crosstalk.min.js"></script><div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Case study: specific projects</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/mtmorgan/grantpubcite/blob/HEAD/vignettes/case_study_project_nums_quick.Rmd" class="external-link"><code>vignettes/case_study_project_nums_quick.Rmd</code></a></small>
      <div class="d-none name"><code>case_study_project_nums_quick.Rmd</code></div>
    </div>

    
    
<p>Compiled: 2023-04-03</p>
<p>This article illustrates how one might explore specific project
numbers. Start by attaching our package and others we will use.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://mtmorgan.github.io/grantpubcite/">grantpubcite</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/rstudio/DT" class="external-link">DT</a></span><span class="op">)</span></span></code></pre></div>
<div class="section level2">
<h2 id="projects">Projects<a class="anchor" aria-label="anchor" href="#projects"></a>
</h2>
<p>Suppose one is interested in specific projects, and the grant project
numbers are known. Define and search for them.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">project_nums</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"U01CA190234"</span>, <span class="st">"U24CA194354"</span><span class="op">)</span></span></code></pre></div>
<p>Query the NIH Reporter for the projects of interest. The use of
<code>project_nums=</code> for the argument required some reflection on
the API schema provided at the end of the <a href="https://api.reporter.nih.gov/#!/Search/Projects" class="external-link">API projects
endpoint</a>.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">projects</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/reporter_and_icite.html">reporter_projects</a></span><span class="op">(</span>project_nums <span class="op">=</span> <span class="va">project_nums</span><span class="op">)</span></span></code></pre></div>
<p>Explore the data, for instance finding the Funding Opportunity
Announcement the grants were reward under, and the fiscal years in which
they were active.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">projects</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/group_by.html" class="external-link">group_by</a></span><span class="op">(</span><span class="va">core_project_num</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/summarise.html" class="external-link">summarize</a></span><span class="op">(</span></span>
<span>        foa <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">full_foa</span><span class="op">)</span>,</span>
<span>        first_year <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/Extremes.html" class="external-link">min</a></span><span class="op">(</span><span class="va">fiscal_year</span><span class="op">)</span>,</span>
<span>        last_year <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/Extremes.html" class="external-link">max</a></span><span class="op">(</span><span class="va">fiscal_year</span><span class="op">)</span>,</span>
<span>        project_title <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">project_title</span><span class="op">)</span></span>
<span>    <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/gpc_utilities.html">gpc_datatable</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-d6fc62eda57fa8dafbe5" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-d6fc62eda57fa8dafbe5">{"x":{"filter":"none","vertical":false,"data":[["U01CA190234","U24CA194354"],["PAR-11-150","PAR-13-294"],[2015,2015],[2019,2019],["Genotype and Imaging Phenotype Biomarkers in Lung Cancer","Quantitative Radiomics System Decoding the Tumor Phenotype"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>project<\/th>\n      <th>foa<\/th>\n      <th>first_year<\/th>\n      <th>last_year<\/th>\n      <th>title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"scrollX":true,"columnDefs":[{"targets":4,"render":"\n        function(data, type, row, meta) {\n            test = type === 'display' && data != null && data.length > 40\n            return test ?\n                '<span title=\\\"' + data + '\\\">' + data.substr(0, 40) + '...<\/span>' :\n                data;\n        }\n    "},{"className":"dt-right","targets":[2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100],"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[-1]; $(this.api().cell(row, -1).node()).css({'vertical-align':'top'});\nvar value=data[0]; $(this.api().cell(row, 0).node()).css({'vertical-align':'top'});\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'vertical-align':'top'});\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'vertical-align':'top'});\nvar value=data[3]; $(this.api().cell(row, 3).node()).css({'vertical-align':'top'});\n}"}},"evals":["options.columnDefs.0.render","options.rowCallback"],"jsHooks":[]}</script>
</div>
<div class="section level2">
<h2 id="publications">Publications<a class="anchor" aria-label="anchor" href="#publications"></a>
</h2>
<p>Use the core project numbers to retrieve known publications</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">core_project_nums</span> <span class="op">&lt;-</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/pull.html" class="external-link">pull</a></span><span class="op">(</span><span class="va">projects</span>, <span class="st">"core_project_num"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="va">publications</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/reporter_and_icite.html">reporter_publications</a></span><span class="op">(</span>core_project_nums <span class="op">=</span> <span class="va">core_project_nums</span><span class="op">)</span></span></code></pre></div>
<p>The number of publications associated with each grant are as
follows:</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">publications</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count</a></span><span class="op">(</span><span class="va">coreproject</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/gpc_utilities.html">gpc_datatable</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-0bfe1e4ecebcd92e6450" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0bfe1e4ecebcd92e6450">{"x":{"filter":"none","vertical":false,"data":[["U01CA190234","U24CA194354"],[58,46]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>project<\/th>\n      <th>n<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"scrollX":true,"columnDefs":[[],{"className":"dt-right","targets":1}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100],"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[-1]; $(this.api().cell(row, -1).node()).css({'vertical-align':'top'});\nvar value=data[0]; $(this.api().cell(row, 0).node()).css({'vertical-align':'top'});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script><p>Many publications were attributed to both grants</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">publications</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count</a></span><span class="op">(</span><span class="va">pmid</span>, name <span class="op">=</span> <span class="st">"cited_by"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count</a></span><span class="op">(</span><span class="va">cited_by</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/gpc_utilities.html">gpc_datatable</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-a5a4e411de59feebafda" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-a5a4e411de59feebafda">{"x":{"filter":"none","vertical":false,"data":[[1,2],[16,44]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>cited_by<\/th>\n      <th>n<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"scrollX":true,"columnDefs":[[],{"className":"dt-right","targets":[0,1]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100],"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[-1]; $(this.api().cell(row, -1).node()).css({'vertical-align':'top'});\nvar value=data[0]; $(this.api().cell(row, 0).node()).css({'vertical-align':'top'});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div class="section level2">
<h2 id="citations">Citations<a class="anchor" aria-label="anchor" href="#citations"></a>
</h2>
<p>Obtain citation information (including information about the orginal
publication) using the <code>publications</code> tibble</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">citations</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/reporter_and_icite.html">icite</a></span><span class="op">(</span><span class="va">publications</span><span class="op">)</span></span></code></pre></div>
<p>There are 60 citation records, one for each publication. Explore
these interactively</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">citations</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="va">pmid</span>, <span class="va">citation_count</span>, <span class="va">relative_citation_ratio</span>, <span class="va">year</span>, <span class="va">title</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/gpc_utilities.html">gpc_datatable</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-62a8f5648d6d7e2bc0d9" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-62a8f5648d6d7e2bc0d9">{"x":{"filter":"none","vertical":false,"data":[[25746350,26173225,26251068,26264429,26278466,26337765,26576732,26697407,26738433,27064691,27085484,27269645,27498015,27502180,27645803,27903462,27987576,28046060,28218293,28275097,28334344,28426673,28499022,28566328,28594880,28615677,28706185,28731408,28851834,28899340,28974199,29036413,29069589,29092929,29092951,29097379,29145421,29363773,29581134,29777175,30240647,30383900,30388114,30500819,30720861,30895304,31010833,31069578,31263116,31753987,31806877,32579922,32772385,32843739,33514711,33536619,33674717,33727623,33930310,36028289],[442,12,288,145,533,63,59,228,28,217,201,577,28,46,140,130,13,61,31,233,8,6,68,214,23,62,129,182,53,29,26,40,173,27,2020,32,77,12,72,915,9,82,70,232,510,236,190,6,130,52,18,99,3,73,46,21,3,24,51,1],[21.17,0.56,13.7,6.7,27.02,2.78,2.99,10.54,1.67,11.84,10.42,32.48,1.96,2.25,7.09,7.16,0.5,3.82,1.42,13.92,0.31,0.28,3.47,11.02,1.5,3.57,8.64,9.38,1.85,1.18,0.99,2.09,6.72,1.22,123.72,1.92,4.85,0.69,4.4,65.56,0.42,4.77,4.44,13.96,40.2,14.7,15.52,0.27,12.82,3.46,1.4,8.87,0.41,8.14,9.84,2.42,0.72,6.48,8.5,null],[2015,2016,2015,2015,2015,2015,2015,2015,2016,2016,2016,2016,2016,2016,2016,2017,2016,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2018,2017,2017,2017,2018,2017,2018,2018,2018,2018,2018,2018,2018,2019,2019,2019,2019,2019,2019,2020,2020,2020,2020,2021,2021,2021,2021,2021,2022],["CT-based radiomic signature predicts distant metastasis in lung adenocarcinoma.","Characterization of Conserved Toxicogenomic Responses in Chemically Exposed Hepatocytes across Species and Platforms.","Radiomic feature clusters and prognostic signatures specific for Lung and Head &amp;amp; Neck cancer.","External validation of a prognostic CT-based radiomic signature in oropharyngeal squamous cell carcinoma.","Machine Learning methods for Quantitative Radiomic Biomarkers.","Somatic mutations associated with MRI-derived volumetric features in glioblastoma.","Fully automatic GBM segmentation in the TCGA-GBM dataset: Prognosis and correlation with VASARI features.","Radiomic Machine-Learning Classifiers for Prognostic Biomarkers of Head and Neck Cancer.","Use of registration-based contour propagation in texture analysis for esophageal cancer pathologic response prediction.","Exploratory Study to Identify Radiomics Classifiers for Lung Cancer Histology.","Radiomic phenotype features predict pathological response in non-small cell lung cancer.","Applications and limitations of radiomics.","Increasing the impact of medical image computing using community-based open-access hackathons: The NA-MIC and 3D Slicer experience.","Imaging-genomics reveals driving pathways of MRI derived volumetric tumor phenotype features in Glioblastoma.","Defining a Radiomic Response Phenotype: A Pilot Study using targeted therapy in NSCLC.","Radiomic-Based Pathological Response Prediction from Primary Tumors and Lymph Nodes in NSCLC.","Radiologic-pathologic correlation of response to chemoradiation in resectable locally advanced NSCLC.","Associations of Radiomic Data Extracted from Static and Respiratory-Gated CT Scans with Disease Recurrence in Lung Cancer Patients Treated with SBRT.","Plasma Exosome Profiling of Cancer Patients by a Next Generation Systems Biology Approach.","Simplified and representative bacterial community of maize roots.","Estimating gene regulatory networks with pandaR.","Lymph node volume predicts survival but not nodal clearance in Stage IIIA-IIIB NSCLC.","Quantitative imaging biomarkers for risk stratification of patients with recurrent glioblastoma treated with bevacizumab.","Somatic Mutations Drive Distinct Imaging Phenotypes in Lung Cancer.","Application of the 3D slicer chest imaging platform segmentation algorithm for large lung nodule delineation.","Associations between radiologist-defined semantic and automatically computed radiomic features in non-small cell lung cancer.","Deep Learning for Fully-Automated Localization and Segmentation of Rectal Cancer on Multiparametric MR.","Defining the biological basis of radiomic phenotypes in lung cancer.","Exploring regulation in tissues with eQTL networks.","Regulatory network changes between cell lines and their tissues of origin.","Tissue-aware RNA-Seq processing and normalization for heterogeneous and sparse data.","Smooth quantile normalization.","Understanding Tissue-Specific Gene Regulation.","WebMeV: A Cloud Platform for Analyzing and Visualizing Cancer Genomic Data.","Computational Radiomics System to Decode the Radiographic Phenotype.","Data Science in Radiology: A Path Forward.","Radiographic prediction of meningioma grade by semantic and radiomic features.","Semi-automated pulmonary nodule interval segmentation using the NLST data.","Data Analysis Strategies in Medical Imaging.","Artificial intelligence in radiology.","Histopathological Image QTL Discovery of Immune Infiltration Variants.","Quantitative imaging of cancer in the postgenomic era: Radio(geno)mics, deep learning, and habitats.","Peritumoral radiomics features predict distant metastasis in locally advanced NSCLC.","Deep learning for lung cancer prognostication: A retrospective multi-cohort radiomics study.","Artificial intelligence in cancer imaging: Clinical challenges and applications.","Predicting response to cancer immunotherapy using noninvasive radiomic biomarkers.","Deep Learning Predicts Lung Cancer Treatment Response from Serial Medical Imaging.","Proceedings of the fourth international molecular pathological epidemiology (MPE) meeting.","Repeatability of Multiparametric Prostate MRI Radiomics Features.","Artificial intelligence for global health.","Nongenic cancer-risk SNPs affect oncogenes, tumour-suppressor genes, and immune function.","Sex Differences in Gene Expression and Regulatory Networks across 29 Human Tissues.","DICOM re-encoding of volumetrically annotated Lung Imaging Database Consortium (LIDC) nodules.","Artificial intelligence in radiation oncology.","Deep convolutional neural networks to predict cardiovascular risk from computed tomography.","BRCA1 and RNAi factors promote repair mediated by small RNAs and PALB2-RAD52.","Deep-learning system to improve the quality and efficiency of volumetric heart segmentation for breast cancer.","Deep learning classification of lung cancer histology using CT images.","Artificial intelligence for clinical oncology.","Clinical validation of deep learning algorithms for radiotherapy targeting of non-small-cell lung cancer: an observational study."]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>pmid<\/th>\n      <th>citn<\/th>\n      <th>rcr<\/th>\n      <th>year<\/th>\n      <th>title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"scrollX":true,"columnDefs":[{"targets":4,"render":"\n        function(data, type, row, meta) {\n            test = type === 'display' && data != null && data.length > 40\n            return test ?\n                '<span title=\\\"' + data + '\\\">' + data.substr(0, 40) + '...<\/span>' :\n                data;\n        }\n    "},{"className":"dt-right","targets":[0,1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100],"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[-1]; $(this.api().cell(row, -1).node()).css({'vertical-align':'top'});\nvar value=data[0]; $(this.api().cell(row, 0).node()).css({'vertical-align':'top'});\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'vertical-align':'top'});\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'vertical-align':'top'});\nvar value=data[3]; $(this.api().cell(row, 3).node()).css({'vertical-align':'top'});\n}"}},"evals":["options.columnDefs.0.render","options.rowCallback"],"jsHooks":[]}</script><p>Use a â€˜joinâ€™ between tables to associate publications and
citations</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">publications</span>, <span class="va">citations</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/group_by.html" class="external-link">group_by</a></span><span class="op">(</span><span class="va">coreproject</span>, <span class="va">year</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/summarise.html" class="external-link">summarize</a></span><span class="op">(</span></span>
<span>        n_publications <span class="op">=</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/context.html" class="external-link">n</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>        total_citations <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/sum.html" class="external-link">sum</a></span><span class="op">(</span><span class="va">citation_count</span><span class="op">)</span></span>
<span>    <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/gpc_utilities.html">gpc_datatable</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; Joining with `by = join_by(pmid)`</span></span>
<span><span class="co">#&gt; `summarise()` has grouped output by 'coreproject'. You can override using the</span></span>
<span><span class="co">#&gt; `.groups` argument.</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-5d8d98eec527c624e071" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-5d8d98eec527c624e071">{"x":{"filter":"none","vertical":false,"data":[["U01CA190234","U01CA190234","U01CA190234","U01CA190234","U01CA190234","U01CA190234","U01CA190234","U01CA190234","U24CA194354","U24CA194354","U24CA194354","U24CA194354","U24CA194354","U24CA194354","U24CA194354","U24CA194354"],[2015,2016,2017,2018,2019,2020,2021,2022,2015,2016,2017,2018,2019,2020,2021,2022],[7,9,19,8,5,4,5,1,4,8,13,8,6,1,5,1],[1758,1262,3552,1452,1118,193,145,1,883,1250,3030,1424,1124,73,145,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>project<\/th>\n      <th>year<\/th>\n      <th>n_publications<\/th>\n      <th>total_citations<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"scrollX":true,"columnDefs":[[],{"className":"dt-right","targets":[1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100],"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[-1]; $(this.api().cell(row, -1).node()).css({'vertical-align':'top'});\nvar value=data[0]; $(this.api().cell(row, 0).node()).css({'vertical-align':'top'});\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'vertical-align':'top'});\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'vertical-align':'top'});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div class="section level2">
<h2 id="session-information">Session information<a class="anchor" aria-label="anchor" href="#session-information"></a>
</h2>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; R version 4.2.3 (2023-03-15)</span></span>
<span><span class="co">#&gt; Platform: x86_64-pc-linux-gnu (64-bit)</span></span>
<span><span class="co">#&gt; Running under: Ubuntu 22.04.2 LTS</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Matrix products: default</span></span>
<span><span class="co">#&gt; BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3</span></span>
<span><span class="co">#&gt; LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.20.so</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; locale:</span></span>
<span><span class="co">#&gt;  [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       </span></span>
<span><span class="co">#&gt;  [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   </span></span>
<span><span class="co">#&gt;  [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          </span></span>
<span><span class="co">#&gt; [10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; attached base packages:</span></span>
<span><span class="co">#&gt; [1] stats     graphics  grDevices utils     datasets  methods   base     </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; other attached packages:</span></span>
<span><span class="co">#&gt; [1] DT_0.27                 grantpubcite_0.0.1.9004 dplyr_1.1.1            </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; loaded via a namespace (and not attached):</span></span>
<span><span class="co">#&gt;  [1] bslib_0.4.2       compiler_4.2.3    pillar_1.9.0      jquerylib_0.1.4  </span></span>
<span><span class="co">#&gt;  [5] tools_4.2.3       bit_4.0.5         digest_0.6.31     jsonlite_1.8.4   </span></span>
<span><span class="co">#&gt;  [9] evaluate_0.20     memoise_2.0.1     lifecycle_1.0.3   tibble_3.2.1     </span></span>
<span><span class="co">#&gt; [13] pkgconfig_2.0.3   rlang_1.1.0       cli_3.6.1         crosstalk_1.2.0  </span></span>
<span><span class="co">#&gt; [17] parallel_4.2.3    curl_5.0.0        yaml_2.3.7        pkgdown_2.0.7    </span></span>
<span><span class="co">#&gt; [21] xfun_0.38         fastmap_1.1.1     withr_2.5.0       httr_1.4.5       </span></span>
<span><span class="co">#&gt; [25] stringr_1.5.0     knitr_1.42        rjsoncons_1.0.0   hms_1.1.3        </span></span>
<span><span class="co">#&gt; [29] desc_1.4.2        generics_0.1.3    fs_1.6.1          vctrs_0.6.1      </span></span>
<span><span class="co">#&gt; [33] sass_0.4.5        htmlwidgets_1.6.2 systemfonts_1.0.4 bit64_4.0.5      </span></span>
<span><span class="co">#&gt; [37] rprojroot_2.0.3   tidyselect_1.2.0  glue_1.6.2        R6_2.5.1         </span></span>
<span><span class="co">#&gt; [41] textshaping_0.3.6 fansi_1.0.4       vroom_1.6.1       rmarkdown_2.21   </span></span>
<span><span class="co">#&gt; [45] tzdb_0.3.0        readr_2.1.4       purrr_1.0.1       magrittr_2.0.3   </span></span>
<span><span class="co">#&gt; [49] ellipsis_0.3.2    htmltools_0.5.5   ragg_1.2.5        utf8_1.2.3       </span></span>
<span><span class="co">#&gt; [53] stringi_1.7.12    cachem_1.0.7      crayon_1.5.2</span></span></code></pre></div>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by Martin Morgan, NIH NCI ITCR U24CA180996.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
