<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="grantpubcite">
<title>Case study: ITCR • grantpubcite</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.2.2/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.2.2/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Case study: ITCR">
<meta property="og:description" content="grantpubcite">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">grantpubcite</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.0.0.9002</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="../articles/grantpubcite.html">Get started</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="active nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/case_study_itcr.html">Case study: ITCR</a>
  </div>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav"></ul>
</div>

    
  </div>
</nav><div class="container template-article">



<script src="case_study_itcr_files/htmlwidgets-1.6.1/htmlwidgets.js"></script><link href="case_study_itcr_files/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="case_study_itcr_files/datatables-binding-0.27/datatables.js"></script><link href="case_study_itcr_files/dt-core-1.12.1/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="case_study_itcr_files/dt-core-1.12.1/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="case_study_itcr_files/dt-core-1.12.1/js/jquery.dataTables.min.js"></script><link href="case_study_itcr_files/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet">
<script src="case_study_itcr_files/crosstalk-1.2.0/js/crosstalk.min.js"></script><div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Case study: ITCR</h1>
            
      
      
      <div class="d-none name"><code>case_study_itcr.Rmd</code></div>
    </div>

    
    
<!-- badges: start -->
<!-- badges: end -->
<p>The ‘grantpubcite’ package can be used to query the <a href="https://reporter.nih.gov/" class="external-link">NIH Reporter</a> database for funded
grants and, optionally, publications associated with grants in NIH
Reporter. The citation history of publications can discovered using <a href="https://icite.od.nih.gov/" class="external-link">iCite</a>. The ‘grantpubcite’ package
help pages and resources make extensive use of ‘tidyverse’ concepts.</p>
<p>See the “Introduction to ‘grantpubcite’” article for installation and
basic use.</p>
<p>Core tidyverse functions used in this article include:</p>
<ul>
<li>
<code><a href="https://tibble.tidyverse.org/reference/tibble.html" class="external-link">tibble()</a></code> – representation of a <code>data.frame</code>,
with better display of long and wide data frames. <code><a href="https://tibble.tidyverse.org/reference/tribble.html" class="external-link">tribble()</a></code>
constructs a tibble in a way that makes the relationship between data
across rows more transparent.</li>
<li>
<code><a href="https://pillar.r-lib.org/reference/glimpse.html" class="external-link">glimpse()</a></code> – providing a quick look into the columns and
data in the tibble by transposing the tibble and display each ‘column’
on a single line.</li>
<li>
<code><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select()</a></code> – column selection.</li>
<li>
<code><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter()</a></code>, <code><a href="https://dplyr.tidyverse.org/reference/slice.html" class="external-link">slice()</a></code> – row selection.</li>
<li>
<code><a href="https://dplyr.tidyverse.org/reference/pull.html" class="external-link">pull()</a></code> – extract a single column as a vector.</li>
<li>
<code><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate()</a></code> – column transformation.</li>
<li>
<code><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count()</a></code> – count occurences in one or more columns.</li>
<li>
<code><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange()</a></code> – order rows by values in one or more
columns.</li>
<li>
<code><a href="https://dplyr.tidyverse.org/reference/distinct.html" class="external-link">distinct()</a></code> – reduce a tibble to only unique rows.</li>
<li>
<code><a href="https://dplyr.tidyverse.org/reference/group_by.html" class="external-link">group_by()</a></code> – perform computations on groups defined by
one or several columns.</li>
<li>
<code><a href="https://dplyr.tidyverse.org/reference/summarise.html" class="external-link">summarize()</a></code> – calculate summary statstics for
groups.</li>
<li>
<code><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join()</a></code>, <code><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">right_join()</a></code> – merge two
tibbles based on shared columns, preserving all rows in the first
(<code><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join()</a></code>) or second (<code><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">right_join()</a></code>)
tibble.</li>
</ul>
<p>Load the library and other packages to be used in this article.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://mtmorgan.github.io/grantpubcite/">grantpubcite</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://ggplot2.tidyverse.org" class="external-link">ggplot2</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/rstudio/DT" class="external-link">DT</a></span><span class="op">)</span></span></code></pre></div>
<p>As a case study, suppose we are interested in grants funded under the
Information Technology in Cancer Research (<a href="https://itcr.cancer.gov/" class="external-link">ITCR</a>) program.</p>
<div class="section level2">
<h2 id="project-information">Project information<a class="anchor" aria-label="anchor" href="#project-information"></a>
</h2>
<p>The relevant funding opportunity announcements (FOA) are summarized
in a <code>tibble</code> (the ‘tidy’ version of
<code>data.frame</code>)</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">foas</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://tibble.tidyverse.org/reference/tribble.html" class="external-link">tribble</a></span><span class="op">(</span></span>
<span>    <span class="op">~</span><span class="va">full_foa</span>,    <span class="op">~</span><span class="va">reissue_of</span>,  <span class="op">~</span><span class="va">foa_tag</span>,      <span class="op">~</span><span class="va">description</span>,</span>
<span>    <span class="co">## current FOAs</span></span>
<span>    <span class="st">"PAR-15-331"</span>, <span class="st">"PAR-13-294"</span>, <span class="st">"Advanced"</span>,    <span class="st">"Advanced Development of Informatics Technologies for Cancer Research and Management (U24)"</span>,</span>
<span>    <span class="st">"PAR-15-332"</span>, <span class="st">"PAR-12-288"</span>, <span class="st">"Early-Stage"</span>, <span class="st">"Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)"</span>,</span>
<span>    <span class="st">"PAR-15-333"</span>, <span class="cn">NA</span>,           <span class="st">"Sustained"</span>,   <span class="st">"Sustained Support for Informatics Resources for Cancer Research and Management (U24)"</span>,</span>
<span>    <span class="st">"PAR-15-334"</span>, <span class="cn">NA</span>,           <span class="st">"Innovative"</span>,  <span class="st">"Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)"</span>,</span>
<span>    <span class="co">## earlier FOAs</span></span>
<span>    <span class="st">"PAR-13-294"</span>, <span class="st">"PAR-12-287"</span>, <span class="st">"Advanced"</span>,    <span class="st">"Advanced Development of Informatics Technology (U24)"</span>,</span>
<span>    <span class="st">"PAR-12-287"</span>, <span class="cn">NA</span>,           <span class="st">"Advanced"</span>,    <span class="st">"Advanced Development of Informatics Technology (U24)"</span>,</span>
<span>    <span class="st">"PAR-12-288"</span>, <span class="cn">NA</span>,           <span class="st">"Early-Stage"</span>, <span class="st">"Early-Stage Development of Informatics Technology (U01)"</span></span>
<span>    <span class="co">## FIXME: add 'Revsion of...' FOAs</span></span>
<span><span class="op">)</span></span>
<span><span class="va">foas</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="va">foas</span>, <span class="op">-</span><span class="va">reissue_of</span><span class="op">)</span> <span class="co"># drop 'reissue_of' for clarity</span></span></code></pre></div>
<p>The NIH Reporter contains quite a bit of information about each
grant. Get a sense of this by using <code><a href="../reference/user_interface.html">reporter_projects()</a></code> to
query for all fields of a single record; most fields included in the
result can also be used for queries.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/user_interface.html">reporter_projects</a></span><span class="op">(</span>foa <span class="op">=</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/pull.html" class="external-link">pull</a></span><span class="op">(</span><span class="va">foas</span>, <span class="st">"full_foa"</span><span class="op">)</span>, limit <span class="op">=</span> <span class="fl">1</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://pillar.r-lib.org/reference/glimpse.html" class="external-link">glimpse</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; Rows: 1</span></span>
<span><span class="co">#&gt; Columns: 44</span></span>
<span><span class="co">#&gt; $ appl_id                  <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> 9676260</span></span>
<span><span class="co">#&gt; $ subproject_id            <span style="color: #949494; font-style: italic;">&lt;lgl&gt;</span> NA</span></span>
<span><span class="co">#&gt; $ fiscal_year              <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> 2019</span></span>
<span><span class="co">#&gt; $ project_num              <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "5U24CA220242-02"</span></span>
<span><span class="co">#&gt; $ project_serial_num       <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "CA220242"</span></span>
<span><span class="co">#&gt; $ organization             <span style="color: #949494; font-style: italic;">&lt;df[,17]&gt;</span> &lt;data.frame[1 x 17]&gt;</span></span>
<span><span class="co">#&gt; $ award_type               <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "5"</span></span>
<span><span class="co">#&gt; $ activity_code            <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "U24"</span></span>
<span><span class="co">#&gt; $ award_amount             <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> 550158</span></span>
<span><span class="co">#&gt; $ is_active                <span style="color: #949494; font-style: italic;">&lt;lgl&gt;</span> FALSE</span></span>
<span><span class="co">#&gt; $ project_num_split        <span style="color: #949494; font-style: italic;">&lt;df[,7]&gt;</span> &lt;data.frame[1 x 7]&gt;</span></span>
<span><span class="co">#&gt; $ principal_investigators  <span style="color: #949494; font-style: italic;">&lt;list&gt;</span> [&lt;data.frame[1 x 8]&gt;]</span></span>
<span><span class="co">#&gt; $ contact_pi_name          <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "ABYZOV, ALEXEJ "</span></span>
<span><span class="co">#&gt; $ program_officers         <span style="color: #949494; font-style: italic;">&lt;list&gt;</span> [&lt;data.frame[1 x 5]&gt;]</span></span>
<span><span class="co">#&gt; $ agency_ic_admin          <span style="color: #949494; font-style: italic;">&lt;df[,3]&gt;</span> &lt;data.frame[1 x 3]&gt;</span></span>
<span><span class="co">#&gt; $ agency_ic_fundings       <span style="color: #949494; font-style: italic;">&lt;list&gt;</span> [&lt;data.frame[1 x 5]&gt;]</span></span>
<span><span class="co">#&gt; $ cong_dist                <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "MN-01"</span></span>
<span><span class="co">#&gt; $ spending_categories      <span style="color: #949494; font-style: italic;">&lt;list&gt;</span> &lt;108, 132, 1393, 276, 320, 3070, 3724&gt;</span></span>
<span><span class="co">#&gt; $ project_start_date       <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "2018-05-01T12:05:00Z"</span></span>
<span><span class="co">#&gt; $ project_end_date         <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "2023-04-30T12:04:00Z"</span></span>
<span><span class="co">#&gt; $ organization_type        <span style="color: #949494; font-style: italic;">&lt;df[,3]&gt;</span> &lt;data.frame[1 x 3]&gt;</span></span>
<span><span class="co">#&gt; $ full_foa                 <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "PAR-15-331"</span></span>
<span><span class="co">#&gt; $ full_study_section       <span style="color: #949494; font-style: italic;">&lt;df[,11]&gt;</span> &lt;data.frame[1 x 11]&gt;</span></span>
<span><span class="co">#&gt; $ award_notice_date        <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "2019-05-01T12:05:00Z"</span></span>
<span><span class="co">#&gt; $ is_new                   <span style="color: #949494; font-style: italic;">&lt;lgl&gt;</span> FALSE</span></span>
<span><span class="co">#&gt; $ mechanism_code_dc        <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "OR"</span></span>
<span><span class="co">#&gt; $ core_project_num         <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "U24CA220242"</span></span>
<span><span class="co">#&gt; $ terms                    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "&lt;Aftercare&gt;&lt;post treatment&gt;&lt;After-Treatment&gt;…</span></span>
<span><span class="co">#&gt; $ pref_terms               <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "Address;Aftercare;Area;Attention;Basic Scien…</span></span>
<span><span class="co">#&gt; $ abstract_text            <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "Project Summary/Abstract\n Progress in techn…</span></span>
<span><span class="co">#&gt; $ project_title            <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "Detection of somatic, subclonal and mosaic C…</span></span>
<span><span class="co">#&gt; $ phr_text                 <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "Narrative\nThe analytical tools that will be…</span></span>
<span><span class="co">#&gt; $ spending_categories_desc <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "Biotechnology; Cancer; Cancer Genomics; …</span></span>
<span><span class="co">#&gt; $ agency_code              <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "NIH"</span></span>
<span><span class="co">#&gt; $ covid_response           <span style="color: #949494; font-style: italic;">&lt;lgl&gt;</span> NA</span></span>
<span><span class="co">#&gt; $ arra_funded              <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "N"</span></span>
<span><span class="co">#&gt; $ budget_start             <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "2019-05-01T12:05:00Z"</span></span>
<span><span class="co">#&gt; $ budget_end               <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "2020-04-30T12:04:00Z"</span></span>
<span><span class="co">#&gt; $ cfda_code                <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "399"</span></span>
<span><span class="co">#&gt; $ funding_mechanism        <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "Other Research-Related"</span></span>
<span><span class="co">#&gt; $ direct_cost_amt          <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> 348999</span></span>
<span><span class="co">#&gt; $ indirect_cost_amt        <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> 201159</span></span>
<span><span class="co">#&gt; $ project_detail_url       <span style="color: #949494; font-style: italic;">&lt;lgl&gt;</span> NA</span></span>
<span><span class="co">#&gt; $ date_added               <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> "2019-05-04T07:05:16Z"</span></span></code></pre></div>
<p>For our purpose we restrict fields to a few of particular interest; a
fuller description of fields available as criteria for querying and
inclusion in the response are available in the technical description of
the NIH Reporter <a href="https://api.reporter.nih.gov/#!/Search/Projects" class="external-link">project search
API</a>.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">include_fields</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"full_foa"</span>,</span>
<span>    <span class="st">"core_project_num"</span>,</span>
<span>    <span class="st">"fiscal_year"</span>,</span>
<span>    <span class="st">"award_amount"</span>,</span>
<span>    <span class="st">"contact_pi_name"</span>,</span>
<span>    <span class="st">"project_title"</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p>Retrieve these fields for all projects in NIH reporter associated
with the funding announcements of interest. Perform some minor data
cleaning by removing leading and trailing whitespace from contact PI
names</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">projects</span> <span class="op">&lt;-</span></span>
<span>    <span class="fu"><a href="../reference/user_interface.html">reporter_projects</a></span><span class="op">(</span></span>
<span>        foa <span class="op">=</span> <span class="va">foas</span><span class="op">$</span><span class="va">full_foa</span>,</span>
<span>        include_fields <span class="op">=</span> <span class="va">include_fields</span></span>
<span>    <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>contact_pi_name <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/trimws.html" class="external-link">trimws</a></span><span class="op">(</span><span class="va">contact_pi_name</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<p>The result is a tidyverse <code>tibble</code>, and we use ‘tidy’
semantics to explore the data. For instance, although there are 341
returned by NIH Reporter, each project is associated with funding over
multiple fiscal years and perhaps mechanisms (e.g., through
administrative supplements).</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">projects</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/distinct.html" class="external-link">distinct</a></span><span class="op">(</span><span class="va">full_foa</span>, <span class="va">core_project_num</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">foas</span>, by <span class="op">=</span> <span class="st">"full_foa"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-153318b252380d271bc7" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-153318b252380d271bc7">{"x":{"filter":"none","vertical":false,"data":[["PAR-15-331","PAR-13-294","PAR-15-334","PAR-15-332","PAR-15-334","PAR-15-331","PAR-12-287","PAR-12-288","PAR-12-288","PAR-15-334","PAR-15-334","PAR-15-334","PAR-15-332","PAR-15-334","PAR-12-288","PAR-15-334","PAR-15-332","PAR-15-332","PAR-13-294","PAR-15-332","PAR-15-331","PAR-15-331","PAR-15-331","PAR-15-331","PAR-13-294","PAR-15-332","PAR-15-331","PAR-13-294","PAR-15-334","PAR-15-331","PAR-13-294","PAR-12-287","PAR-15-331","PAR-12-288","PAR-12-288","PAR-12-287","PAR-15-331","PAR-12-287","PAR-15-332","PAR-12-288","PAR-15-332","PAR-13-294","PAR-12-288","PAR-13-294","PAR-15-334","PAR-12-287","PAR-15-332","PAR-12-288","PAR-15-332","PAR-15-332","PAR-15-331","PAR-15-331","PAR-12-288","PAR-13-294","PAR-13-294","PAR-15-334","PAR-15-331","PAR-15-332","PAR-13-294","PAR-15-333","PAR-15-333","PAR-15-331","PAR-13-294","PAR-15-334","PAR-15-334","PAR-12-288","PAR-15-334","PAR-13-294","PAR-15-334","PAR-15-333","PAR-15-334","PAR-15-333","PAR-12-287","PAR-15-334","PAR-15-332","PAR-13-294","PAR-15-333","PAR-15-334","PAR-15-332","PAR-15-332","PAR-15-334","PAR-15-332","PAR-12-288","PAR-13-294","PAR-15-334","PAR-12-288","PAR-15-334","PAR-13-294","PAR-12-288","PAR-15-334"],["U24CA220242","U24CA194354","R21CA220352","U01CA242871","R21CA231892","U24CA180922","U24CA180921","U01CA184826","U01CA180964","R21CA220398","R21CA242940","R21CA226188","U01CA214846","R21CA231911","U01CA204826","R21CA209848","U01CA242954","U01CA220401","U24CA189523","U01CA231860","U24CA204800","U24CA215123","U24CA209996","U24CA231877","U24CA199347","U01CA209936","U24CA237719","U24CA194362","R21CA220299","U24CA204863","U24CA199460","U24CA180951","U24CA220441","U01CA180982","U01CA188547","U24CA184427","U24CA184427","U24CA184407","U01CA239055","U01CA180940","U01CA220413","U24CA180927","U01CA180956","U24CA204817","R21CA209941","U24CA180918","U01CA242936","U01CA188590","U01CA226196","U01CA237711","U24CA209851","U24CA237617","U01CA180980","U24CA199374","U24CA204854","R21CA226106","U24CA209999","U01CA215010","U24CA194107","U24CA220341","U24CA210004","U24CA180996","U24CA180996","R21CA231853","R21CA220411","U01CA199459","R21CA209875","U24CA204781","R21CA209940","U24CA215109","R21CA214845","U24CA231846","U24CA180922","R21CA237686","U01CA242879","U24CA180924","U24CA220457","R21CA242933","U01CA231782","U01CA226187","R21CA242861","U01CA231840","U01CA184783","U24CA199461","R21CA209874","U01CA204695","R21CA237493","U24CA194215","U01CA180975","R21CA231904"],["Advanced","Advanced","Innovative","Early-Stage","Innovative","Advanced","Advanced","Early-Stage","Early-Stage","Innovative","Innovative","Innovative","Early-Stage","Innovative","Early-Stage","Innovative","Early-Stage","Early-Stage","Advanced","Early-Stage","Advanced","Advanced","Advanced","Advanced","Advanced","Early-Stage","Advanced","Advanced","Innovative","Advanced","Advanced","Advanced","Advanced","Early-Stage","Early-Stage","Advanced","Advanced","Advanced","Early-Stage","Early-Stage","Early-Stage","Advanced","Early-Stage","Advanced","Innovative","Advanced","Early-Stage","Early-Stage","Early-Stage","Early-Stage","Advanced","Advanced","Early-Stage","Advanced","Advanced","Innovative","Advanced","Early-Stage","Advanced","Sustained","Sustained","Advanced","Advanced","Innovative","Innovative","Early-Stage","Innovative","Advanced","Innovative","Sustained","Innovative","Sustained","Advanced","Innovative","Early-Stage","Advanced","Sustained","Innovative","Early-Stage","Early-Stage","Innovative","Early-Stage","Early-Stage","Advanced","Innovative","Early-Stage","Innovative","Advanced","Early-Stage","Innovative"],["Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Advanced Development of Informatics Technology (U24)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Advanced Development of Informatics Technology (U24)","Early-Stage Development of Informatics Technology (U01)","Early-Stage Development of Informatics Technology (U01)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Early-Stage Development of Informatics Technology (U01)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Advanced Development of Informatics Technology (U24)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Advanced Development of Informatics Technology (U24)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Advanced Development of Informatics Technology (U24)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Advanced Development of Informatics Technology (U24)","Advanced Development of Informatics Technology (U24)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Early-Stage Development of Informatics Technology (U01)","Early-Stage Development of Informatics Technology (U01)","Advanced Development of Informatics Technology (U24)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Advanced Development of Informatics Technology (U24)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Early-Stage Development of Informatics Technology (U01)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Advanced Development of Informatics Technology (U24)","Early-Stage Development of Informatics Technology (U01)","Advanced Development of Informatics Technology (U24)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Advanced Development of Informatics Technology (U24)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Early-Stage Development of Informatics Technology (U01)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Early-Stage Development of Informatics Technology (U01)","Advanced Development of Informatics Technology (U24)","Advanced Development of Informatics Technology (U24)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Advanced Development of Informatics Technology (U24)","Sustained Support for Informatics Resources for Cancer Research and Management (U24)","Sustained Support for Informatics Resources for Cancer Research and Management (U24)","Advanced Development of Informatics Technologies for Cancer Research and Management (U24)","Advanced Development of Informatics Technology (U24)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Early-Stage Development of Informatics Technology (U01)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Advanced Development of Informatics Technology (U24)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Sustained Support for Informatics Resources for Cancer Research and Management (U24)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Sustained Support for Informatics Resources for Cancer Research and Management (U24)","Advanced Development of Informatics Technology (U24)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Advanced Development of Informatics Technology (U24)","Sustained Support for Informatics Resources for Cancer Research and Management (U24)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Early-Stage Development of Informatics Technologies for Cancer Research and Management (U01)","Early-Stage Development of Informatics Technology (U01)","Advanced Development of Informatics Technology (U24)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Early-Stage Development of Informatics Technology (U01)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)","Advanced Development of Informatics Technology (U24)","Early-Stage Development of Informatics Technology (U01)","Development of Innovative Informatics Methods and Algorithms for Cancer Research and Management (R21)"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>full_foa<\/th>\n      <th>core_project_num<\/th>\n      <th>foa_tag<\/th>\n      <th>description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script><p>The number of projects funded by each FOA is</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">projects</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">foas</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/distinct.html" class="external-link">distinct</a></span><span class="op">(</span><span class="va">foa_tag</span>, <span class="va">core_project_num</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count</a></span><span class="op">(</span><span class="va">foa_tag</span><span class="op">)</span></span>
<span><span class="co">#&gt; Joining with `by = join_by(full_foa)`</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 4 × 2</span></span></span>
<span><span class="co">#&gt;   foa_tag         n</span></span>
<span><span class="co">#&gt;   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;int&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">1</span> Advanced       32</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">2</span> Early-Stage    29</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">3</span> Innovative     21</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">4</span> Sustained       5</span></span></code></pre></div>
<p>The number of projects funded per fiscal year is</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">projects</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/distinct.html" class="external-link">distinct</a></span><span class="op">(</span><span class="va">full_foa</span>, <span class="va">core_project_num</span>, <span class="va">fiscal_year</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count</a></span><span class="op">(</span><span class="va">fiscal_year</span><span class="op">)</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 11 × 2</span></span></span>
<span><span class="co">#&gt;    fiscal_year     n</span></span>
<span><span class="co">#&gt;          <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> <span style="color: #949494; font-style: italic;">&lt;int&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span>        <span style="text-decoration: underline;">2</span>013     7</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span>        <span style="text-decoration: underline;">2</span>014    17</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span>        <span style="text-decoration: underline;">2</span>015    27</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span>        <span style="text-decoration: underline;">2</span>016    38</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span>        <span style="text-decoration: underline;">2</span>017    45</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span>        <span style="text-decoration: underline;">2</span>018    51</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span>        <span style="text-decoration: underline;">2</span>019    63</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span>        <span style="text-decoration: underline;">2</span>020    42</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span>        <span style="text-decoration: underline;">2</span>021    22</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">10</span>        <span style="text-decoration: underline;">2</span>022    10</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">11</span>        <span style="text-decoration: underline;">2</span>023     1</span></span></code></pre></div>
<p>A few projects changed names within the same award; for convenience
we create a table of most-recent project titles</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">project_titles_h</span> <span class="op">&lt;-</span></span>
<span>    <span class="va">projects</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="va">fiscal_year</span>, <span class="va">core_project_num</span>, <span class="va">project_title</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/desc.html" class="external-link">desc</a></span><span class="op">(</span><span class="va">fiscal_year</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/duplicated.html" class="external-link">duplicated</a></span><span class="op">(</span><span class="va">core_project_num</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="va">core_project_num</span>, <span class="va">project_title</span><span class="op">)</span></span></code></pre></div>
<p>The 87 projects funded by ITCR are</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="va">project_titles_h</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-22afea06269c45d49594" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-22afea06269c45d49594">{"x":{"filter":"none","vertical":false,"data":[["U24CA237617","U24CA220242","U24CA215123","U24CA231877","U24CA237719","U24CA180922","U01CA239055","U24CA220341","U24CA231846","U24CA180996","U01CA242871","U01CA242954","U01CA220401","U01CA231860","U24CA204800","U24CA209996","U24CA204863","U01CA242936","U01CA237711","U24CA184427","U24CA215109","U01CA242879","U24CA220457","R21CA242940","R21CA226188","R21CA231911","U24CA220441","U24CA204817","U01CA226196","U24CA209851","U24CA204854","R21CA226106","U24CA209999","U24CA210004","U24CA204781","R21CA237686","R21CA242933","U01CA231782","U01CA226187","R21CA242861","U01CA231840","R21CA237493","U24CA194215","U24CA194354","R21CA220352","R21CA231892","R21CA220398","U01CA214846","U24CA189523","U24CA199347","U24CA194362","R21CA220299","U24CA199460","U01CA220413","U24CA199374","U01CA215010","U24CA194107","R21CA231853","R21CA220411","R21CA214845","U24CA180924","U24CA199461","R21CA231904","U01CA209936","U24CA180927","R21CA209875","R21CA209940","U24CA184407","U01CA204826","U01CA204695","U24CA180921","R21CA209848","U24CA180951","U01CA188547","R21CA209941","U24CA180918","U01CA188590","U01CA199459","R21CA209874","U01CA184826","U01CA180982","U01CA180940","U01CA184783","U01CA180975","U01CA180964","U01CA180956","U01CA180980"],["Developing Informatics Technologies to Model Cancer Gene Regulation","Detection of somatic, subclonal and mosaic CNVs from sequencing","The TOPAS Tool for Particle Simulation, a Monte Carlo Simulation Tool for Physics, Biology and Clinical Research","A Federated Galaxy for user-friendly large-scale cancer genomics research","Standardized and Genome-Wide Clinical Interpretation of Complex Genotypes for Cancer Precision Medicine","Trinity: Transcriptome assembly for genetic and functional analysis of cancer","HistoTools:  A suite of digital pathology tools for quality control, annotation and dataset identification","Supporting and evolving Gene Set Enrichment Analysis and the Molecular Signatures Database for cancer research","WebMeV: A Robust Platform for Intuitive Genomic Data Analysis","Cancer Genomics: Integrative and Scalable Solutions in R/Bioconductor","The Federated Tumor Segmentation (FeTS) platform: An intuitive tool facilitating secure multi-institutional collaboration","Eliminating variants of uncertain significance in BRCA1, BRCA2 and beyond","Informatics Tools for Quantitative Digital Pathology Profiling and Integrated Prognostic Modeling","Open-access X-ray and CT simulation toolkit for research in cancer imaging and dosimetry","Scalable Clinical Decision Support for Individualized Cancer Risk Management","Building protected data sharing networks to advance cancer risk assessment and treatment","Advanced Development and Dissemination of EMERSE for Cancer Phenotyping from Medical Records","Computational pathology software for integrative cancer research with three-dimensional digital slides","Informatics resources for liquid biopsy research","NDEx - the Network Data Exchange A Network Commons for Biologists","TCIA Sustainment and Scalability - Platforms for Quantitative Imaging Informatics in Precision Medicine","Distributed Learning of Deep Learning Models for Cancer Research","The cBioPortal for Cancer Genomics","Semi-supervised Algorithms for Risk Assessment with Noisy EHR Data","Enhanced deconvolution and prediction of mutational signatures","STAN-CT: Standardization and Normalization of CT images for Lung Cancer Patients","Developing the JBrowse Genome Browser to Visualize Structural Variants and Cancer Genomics Data","Informatics Tools for High-throughput Analysis of Cancer Mutations","Bioinformatics Technology to Characterize Tumor Infiltrating Immune Repertoires","TCPA: an Integrated Bioinformatics Resource for Functional Cancer Proteomic Data","INTEGRATIVE IMAGING INFORMATICS FOR CANCER RESEARCH","Conservation and functional-characterization of tumor methylation sites","Monitoring tumor subclonal heterogeneity over time and space","The Integrative Genomics Viewer (IGV): visualization supporting cancer research","The PDX Integrator- Re-purposing KOMP2 informatics to widely disseminate data from Patient Derived Xenografts","Personalized dosimetry for liver cancer radioembolization using fluid dynamics simulation","Data-driven QSP software for personalized colon cancer treatment","A Unified Machine Learning Package for Cancer Diagnosis","Tools for annotating mutations in the 3D cancer genome","A statistical framework to systematically characterize cancer driver mutations in noncoding genomic regions","Network Analysis for a Data-Driven Approach to Cancer Care","Development and Dissemination of KiNet: A Novel Imaging Informatics Tool for Gastrointestinal and Pancreatic Neuroendocrine Tumors","Advancing Cancer Pharmacoepidemiology Research Through EHRs and Informatics","Quantitative Radiomics System Decoding the Tumor Phenotype","Predicting the Presence of Clinically Significant Prostate Cancer using Multiparametric MRI and MR-US Fusion Biopsy","Bayesian learning algorithms for identifying and classifying heterogeneity of cell types in variety of solid tumors","CGDnet: Cancer Gene Drug Network: Using patient-specific drug-gene networks for recommending targeted cancer therapies.","Accelerating Cancer Genomics with Cloud-scale Bioconductor","Cancer imaging phenomics software suite: application to brain and breast cancer","A Galaxy‐based multi‐omic informatics hub for cancer researchers","Advanced Development of an Open-source Platform for Web-based Integrative Digital Image Analysis in Cancer","Proteomic-based integrated subject-specific networks in cancer","Extensible open-source zero-footprint web viewer for oncologic imaging research","An interactive analysis toolkit for single cell RNA-seq in cancer research","Pathology Image Informatics Platform for visualization, analysis and management","Integration of comprehensive cancer mutation and expression-associated data for biomarker evaluation and discovery","GenePattern Notebooks for Cancer Research","Inferring Kinase Activity Profiles from Phosphoproteomic Data","Somatic variant calling and phasing using colored de Bruijn graphs in heterogeneous tumors","Statistical Evaluation and Selection of Normalization Methods for microRNA Sequencing Data in Cancer Biomarker Studies","Tools to Analyze Morphology and Spatially Mapped Molecular Data","Next-Generation Clustered Heat Maps for fluent, interactive exploration of omic data","An Ontology-driven Faceted Query Engine for the Kentucky Cancer Registry","DEVELOPMENT OF INFORMATICS RESOURCES FOR INTERPRETATION OF CLINICALLY ACTIONABLE VARIANTS IN CANCER","Informatics Tools for Optimized Imaging Biomarkers for Cancer Research&amp;Discovery","A network-based approach for personalized treatment of Multiple Myeloma","Integrative genomic framework for dissecting regulatory mechanisms underlying hepatocellular carcinoma","Cancer Deep Phenotype Extraction from Electronic Medical Records","Informatics Tools for Tumor Heterogeneity in Multiplexed Fluorescence Images","Informatic tools for single-nucleotide analysis of cancer RNA-Seq","Advanced Development of TIES-Enhancing Access to Tissue for Cancer Research","Algorithms for Literature-Guided Multi-Platform Identification of Cancer Subtypes","Cloud Based Resource for Data Hosting, Visualization and Analysis Using UCSC Canc","Informatics Links Between Histological Features and Genetics in Cancer","Structure-based selection of tumor-antigens for T-cell based immunotherapy","Quantitative Image Informatics for Cancer Research (QIICR)","Computational Framework for Single-Cell Genomics of Tumors","Open source diffusion MRI technology for brain cancer research","Developing Enabling PET-CT Image Analysis Tools for Predicting Response in Radiation Cancer Therapy","Software Tools For Regulatory Analysis of Large Cancer Methylome Datasets","OmniSearch: A semantic search tool for discovering microRNAs' critical roles in h","caCDE-QA: A Quality Assurance Platform for Cancer Study Common Data Elements","Interactive Informatics Resource for Research-driven Cancer Proteomics","EMR Adverse Drug Event Detection for Pharmacovigilance","Informatics to enable routine personalized cancer therapy","Informatics Tools for High-throughput Analysis of Cancer Mutations","Developing Informatics Technologies to Model Cancer Gene Regulation"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>core_project_num<\/th>\n      <th>project_title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script><p>We update <code>projects</code> to use harmonized project titles.</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">projects</span> <span class="op">&lt;-</span></span>
<span>    <span class="va">projects</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="op">-</span><span class="va">project_title</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">project_titles_h</span>, by <span class="op">=</span> <span class="st">"core_project_num"</span><span class="op">)</span></span></code></pre></div>
<p>Three project numbers have been funded under more than one FOA.</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">projects</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/distinct.html" class="external-link">distinct</a></span><span class="op">(</span><span class="va">full_foa</span>, <span class="va">core_project_num</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count</a></span><span class="op">(</span><span class="va">core_project_num</span>, sort <span class="op">=</span>  <span class="cn">TRUE</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">n</span> <span class="op">&gt;</span> <span class="fl">1L</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">project_titles_h</span><span class="op">)</span></span>
<span><span class="co">#&gt; Joining with `by = join_by(core_project_num)`</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 3 × 3</span></span></span>
<span><span class="co">#&gt;   core_project_num     n project_title                                          </span></span>
<span><span class="co">#&gt;   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>            <span style="color: #949494; font-style: italic;">&lt;int&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>                                                  </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">1</span> U24CA180922          2 Trinity: Transcriptome assembly for genetic and functi…</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">2</span> U24CA180996          2 Cancer Genomics: Integrative and Scalable Solutions in…</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">3</span> U24CA184427          2 NDEx - the Network Data Exchange A Network Commons for…</span></span></code></pre></div>
<p>The following summarizes project funding across years; there are
likely many caveats to this.</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">project_summaries</span> <span class="op">&lt;-</span></span>
<span>    <span class="va">projects</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/group_by.html" class="external-link">group_by</a></span><span class="op">(</span><span class="va">core_project_num</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/summarise.html" class="external-link">summarize</a></span><span class="op">(</span></span>
<span>        n_years <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/length.html" class="external-link">length</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">fiscal_year</span><span class="op">)</span><span class="op">)</span>,</span>
<span>        amount <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/sum.html" class="external-link">sum</a></span><span class="op">(</span><span class="va">award_amount</span><span class="op">)</span></span>
<span>    <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">project_titles_h</span>, by <span class="op">=</span> <span class="st">"core_project_num"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/desc.html" class="external-link">desc</a></span><span class="op">(</span><span class="va">n_years</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="va">project_summaries</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span><span class="va">n_years</span>, <span class="va">amount</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_point.html" class="external-link">geom_point</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_smooth.html" class="external-link">geom_smooth</a></span><span class="op">(</span>method <span class="op">=</span> <span class="st">"lm"</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/scale_continuous.html" class="external-link">scale_x_continuous</a></span><span class="op">(</span>breaks <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/seq.html" class="external-link">seq</a></span><span class="op">(</span><span class="fl">1</span>, <span class="fl">10</span><span class="op">)</span>, minor_breaks <span class="op">=</span> <span class="cn">NULL</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html" class="external-link">labs</a></span><span class="op">(</span>x <span class="op">=</span> <span class="st">"Fiscal years of award"</span>, y <span class="op">=</span> <span class="st">"Total funding"</span><span class="op">)</span></span>
<span><span class="co">#&gt; `geom_smooth()` using formula = 'y ~ x'</span></span></code></pre></div>
<p><img src="case_study_itcr_files/figure-html/unnamed-chunk-13-1.png" width="700"></p>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="va">project_summaries</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-b554da19e728208d91bb" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-b554da19e728208d91bb">{"x":{"filter":"none","vertical":false,"data":[["U24CA180922","U24CA180996","U24CA184427","U24CA180924","U24CA180918","U24CA180921","U24CA180927","U24CA180951","U24CA184407","U24CA189523","U24CA194107","U24CA194215","U24CA194354","U24CA194362","U24CA199374","U24CA199460","U24CA199461","U24CA204781","U24CA204800","U24CA204817","U24CA204854","U24CA204863","U24CA209851","U24CA209996","U24CA209999","U24CA210004","U24CA215109","U24CA215123","U24CA220242","U24CA220341","U24CA220457","U24CA231877","U24CA237617","U01CA239055","U24CA199347","U24CA220441","U24CA231846","U24CA237719","R21CA220398","U01CA180940","U01CA180956","U01CA180964","U01CA180975","U01CA180980","U01CA180982","U01CA184783","U01CA184826","U01CA188547","U01CA188590","U01CA199459","U01CA204695","U01CA204826","U01CA209936","U01CA214846","U01CA215010","U01CA220401","U01CA220413","U01CA226187","U01CA226196","U01CA231782","U01CA231840","U01CA231860","U01CA237711","U01CA242871","U01CA242879","U01CA242936","U01CA242954","R21CA209848","R21CA209874","R21CA209875","R21CA209940","R21CA209941","R21CA214845","R21CA220299","R21CA220352","R21CA220411","R21CA226106","R21CA226188","R21CA231892","R21CA231904","R21CA231911","R21CA237493","R21CA237686","R21CA242861","R21CA242933","R21CA242940","R21CA231853"],[10,8,8,6,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,4,4,4,4,4,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,1],[7263992,5281746,5658437,3479040,3722197,3291725,3433093,2939697,4376472,3207018,2798361,3274448,3479472,3591907,2898668,3467871,2646415,1904229,3747474,2493392,3040146,3824879,3905934,3962385,3743640,3802236,8140937,4207966,2627986,3392981,6365918,3850721,3639833,1045548,2315597,2737046,2570825,2942615,419651,1096706,923065,963609,1051094,1142986,913342,1265134,1095986,1171438,933878,1104852,958650,1105410,957209,1322177,1339167,1331297,1239730,1165246,1286835,980371,1239743,1518187,1127188,1096884,1191276,1075520,1339635,357679,364857,405529,402332,384399,413524,377579,352356,354792,460732,394763,362415,350073,388776,372034,368539,602867,386767,369895,195820],["Trinity: Transcriptome assembly for genetic and functional analysis of cancer","Cancer Genomics: Integrative and Scalable Solutions in R/Bioconductor","NDEx - the Network Data Exchange A Network Commons for Biologists","Tools to Analyze Morphology and Spatially Mapped Molecular Data","Quantitative Image Informatics for Cancer Research (QIICR)","Advanced Development of TIES-Enhancing Access to Tissue for Cancer Research","Informatics Tools for Optimized Imaging Biomarkers for Cancer Research&amp;Discovery","Cloud Based Resource for Data Hosting, Visualization and Analysis Using UCSC Canc","Cancer Deep Phenotype Extraction from Electronic Medical Records","Cancer imaging phenomics software suite: application to brain and breast cancer","GenePattern Notebooks for Cancer Research","Advancing Cancer Pharmacoepidemiology Research Through EHRs and Informatics","Quantitative Radiomics System Decoding the Tumor Phenotype","Advanced Development of an Open-source Platform for Web-based Integrative Digital Image Analysis in Cancer","Pathology Image Informatics Platform for visualization, analysis and management","Extensible open-source zero-footprint web viewer for oncologic imaging research","Next-Generation Clustered Heat Maps for fluent, interactive exploration of omic data","The PDX Integrator- Re-purposing KOMP2 informatics to widely disseminate data from Patient Derived Xenografts","Scalable Clinical Decision Support for Individualized Cancer Risk Management","Informatics Tools for High-throughput Analysis of Cancer Mutations","INTEGRATIVE IMAGING INFORMATICS FOR CANCER RESEARCH","Advanced Development and Dissemination of EMERSE for Cancer Phenotyping from Medical Records","TCPA: an Integrated Bioinformatics Resource for Functional Cancer Proteomic Data","Building protected data sharing networks to advance cancer risk assessment and treatment","Monitoring tumor subclonal heterogeneity over time and space","The Integrative Genomics Viewer (IGV): visualization supporting cancer research","TCIA Sustainment and Scalability - Platforms for Quantitative Imaging Informatics in Precision Medicine","The TOPAS Tool for Particle Simulation, a Monte Carlo Simulation Tool for Physics, Biology and Clinical Research","Detection of somatic, subclonal and mosaic CNVs from sequencing","Supporting and evolving Gene Set Enrichment Analysis and the Molecular Signatures Database for cancer research","The cBioPortal for Cancer Genomics","A Federated Galaxy for user-friendly large-scale cancer genomics research","Developing Informatics Technologies to Model Cancer Gene Regulation","HistoTools:  A suite of digital pathology tools for quality control, annotation and dataset identification","A Galaxy‐based multi‐omic informatics hub for cancer researchers","Developing the JBrowse Genome Browser to Visualize Structural Variants and Cancer Genomics Data","WebMeV: A Robust Platform for Intuitive Genomic Data Analysis","Standardized and Genome-Wide Clinical Interpretation of Complex Genotypes for Cancer Precision Medicine","CGDnet: Cancer Gene Drug Network: Using patient-specific drug-gene networks for recommending targeted cancer therapies.","caCDE-QA: A Quality Assurance Platform for Cancer Study Common Data Elements","Informatics Tools for High-throughput Analysis of Cancer Mutations","Informatics to enable routine personalized cancer therapy","EMR Adverse Drug Event Detection for Pharmacovigilance","Developing Informatics Technologies to Model Cancer Gene Regulation","OmniSearch: A semantic search tool for discovering microRNAs' critical roles in h","Interactive Informatics Resource for Research-driven Cancer Proteomics","Software Tools For Regulatory Analysis of Large Cancer Methylome Datasets","Informatics Links Between Histological Features and Genetics in Cancer","Computational Framework for Single-Cell Genomics of Tumors","Open source diffusion MRI technology for brain cancer research","Informatic tools for single-nucleotide analysis of cancer RNA-Seq","Informatics Tools for Tumor Heterogeneity in Multiplexed Fluorescence Images","DEVELOPMENT OF INFORMATICS RESOURCES FOR INTERPRETATION OF CLINICALLY ACTIONABLE VARIANTS IN CANCER","Accelerating Cancer Genomics with Cloud-scale Bioconductor","Integration of comprehensive cancer mutation and expression-associated data for biomarker evaluation and discovery","Informatics Tools for Quantitative Digital Pathology Profiling and Integrated Prognostic Modeling","An interactive analysis toolkit for single cell RNA-seq in cancer research","Tools for annotating mutations in the 3D cancer genome","Bioinformatics Technology to Characterize Tumor Infiltrating Immune Repertoires","A Unified Machine Learning Package for Cancer Diagnosis","Network Analysis for a Data-Driven Approach to Cancer Care","Open-access X-ray and CT simulation toolkit for research in cancer imaging and dosimetry","Informatics resources for liquid biopsy research","The Federated Tumor Segmentation (FeTS) platform: An intuitive tool facilitating secure multi-institutional collaboration","Distributed Learning of Deep Learning Models for Cancer Research","Computational pathology software for integrative cancer research with three-dimensional digital slides","Eliminating variants of uncertain significance in BRCA1, BRCA2 and beyond","Algorithms for Literature-Guided Multi-Platform Identification of Cancer Subtypes","Developing Enabling PET-CT Image Analysis Tools for Predicting Response in Radiation Cancer Therapy","A network-based approach for personalized treatment of Multiple Myeloma","Integrative genomic framework for dissecting regulatory mechanisms underlying hepatocellular carcinoma","Structure-based selection of tumor-antigens for T-cell based immunotherapy","Statistical Evaluation and Selection of Normalization Methods for microRNA Sequencing Data in Cancer Biomarker Studies","Proteomic-based integrated subject-specific networks in cancer","Predicting the Presence of Clinically Significant Prostate Cancer using Multiparametric MRI and MR-US Fusion Biopsy","Somatic variant calling and phasing using colored de Bruijn graphs in heterogeneous tumors","Conservation and functional-characterization of tumor methylation sites","Enhanced deconvolution and prediction of mutational signatures","Bayesian learning algorithms for identifying and classifying heterogeneity of cell types in variety of solid tumors","An Ontology-driven Faceted Query Engine for the Kentucky Cancer Registry","STAN-CT: Standardization and Normalization of CT images for Lung Cancer Patients","Development and Dissemination of KiNet: A Novel Imaging Informatics Tool for Gastrointestinal and Pancreatic Neuroendocrine Tumors","Personalized dosimetry for liver cancer radioembolization using fluid dynamics simulation","A statistical framework to systematically characterize cancer driver mutations in noncoding genomic regions","Data-driven QSP software for personalized colon cancer treatment","Semi-supervised Algorithms for Risk Assessment with Noisy EHR Data","Inferring Kinase Activity Profiles from Phosphoproteomic Data"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>core_project_num<\/th>\n      <th>n_years<\/th>\n      <th>amount<\/th>\n      <th>project_title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script><div class="section level3">
<h3 id="maturing-projects">Maturing projects?<a class="anchor" aria-label="anchor" href="#maturing-projects"></a>
</h3>
<p>A unique aspect of the ITCR FOA structure is that it envisions
projects starting at one FOA and ‘maturing’ to the next FOA, e.g., an
<code>Early-Stage</code> U01 matures to an <code>Advanced</code> U24. It
is not possible to assess this by tracking a project number through full
FOA, because the project number changes with FOA. Are there projects
with the same contact PI that have received awards from different FOA
tags?</p>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">tagged_pis</span> <span class="op">&lt;-</span></span>
<span>    <span class="va">projects</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/distinct.html" class="external-link">distinct</a></span><span class="op">(</span><span class="va">full_foa</span>, <span class="va">core_project_num</span>, <span class="va">contact_pi_name</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">foas</span>, by <span class="op">=</span> <span class="st">"full_foa"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="va">foa_tag</span>, <span class="va">contact_pi_name</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/distinct.html" class="external-link">distinct</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">maturing_pis</span> <span class="op">&lt;-</span></span>
<span>    <span class="va">tagged_pis</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count</a></span><span class="op">(</span><span class="va">contact_pi_name</span>, sort <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">n</span> <span class="op">&gt;</span> <span class="fl">1L</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="op">-</span><span class="va">n</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span></span>
<span>        <span class="va">projects</span> <span class="op">|&gt;</span></span>
<span>        <span class="fu"><a href="https://dplyr.tidyverse.org/reference/distinct.html" class="external-link">distinct</a></span><span class="op">(</span><span class="va">full_foa</span>, <span class="va">core_project_num</span>, <span class="va">contact_pi_name</span>, <span class="va">project_title</span><span class="op">)</span>,</span>
<span>        multiple <span class="op">=</span> <span class="st">"all"</span></span>
<span>    <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">foas</span>, by <span class="op">=</span> <span class="st">"full_foa"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="va">contact_pi_name</span>, <span class="va">project_title</span>, <span class="va">foa_tag</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="va">contact_pi_name</span>, <span class="va">project_title</span>, <span class="va">foa_tag</span><span class="op">)</span></span>
<span><span class="co">#&gt; Joining with `by = join_by(contact_pi_name)`</span></span>
<span><span class="fu">DT</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="va">maturing_pis</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-580451d1011e9d346835" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-580451d1011e9d346835">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10"],["GRIFFITH, OBI L.","GRIFFITH, OBI L.","KARCHIN, RACHEL","KARCHIN, RACHEL","LIU, XIAOLE SHIRLEY","LIU, XIAOLE SHIRLEY","LIU, XIAOLE SHIRLEY","MESIROV, JILL P.","MESIROV, JILL P.","MESIROV, JILL P."],["DEVELOPMENT OF INFORMATICS RESOURCES FOR INTERPRETATION OF CLINICALLY ACTIONABLE VARIANTS IN CANCER","Standardized and Genome-Wide Clinical Interpretation of Complex Genotypes for Cancer Precision Medicine","Informatics Tools for High-throughput Analysis of Cancer Mutations","Informatics Tools for High-throughput Analysis of Cancer Mutations","Bioinformatics Technology to Characterize Tumor Infiltrating Immune Repertoires","Developing Informatics Technologies to Model Cancer Gene Regulation","Developing Informatics Technologies to Model Cancer Gene Regulation","GenePattern Notebooks for Cancer Research","Supporting and evolving Gene Set Enrichment Analysis and the Molecular Signatures Database for cancer research","The Integrative Genomics Viewer (IGV): visualization supporting cancer research"],["Early-Stage","Advanced","Advanced","Early-Stage","Early-Stage","Advanced","Early-Stage","Advanced","Sustained","Sustained"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>contact_pi_name<\/th>\n      <th>project_title<\/th>\n      <th>foa_tag<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script><p>Direct inspection suggests that three contact PIs (Giffith, Karchin,
Liu) have each seen one project mature. All are from ‘Early-stage’ to
‘Advanced’.</p>
</div>
</div>
<div class="section level2">
<h2 id="publications">Publications<a class="anchor" aria-label="anchor" href="#publications"></a>
</h2>
<p>Grantees report publications associated with their grants, and this
information can be retrieved from NIH reporter. Queries are formulated
in a way similar to projects, as described on the NIH Reporter <a href="https://api.reporter.nih.gov/#!/Search/Publications" class="external-link">publication
search API</a>, a rich set of query criteria can be used, but fields
included in the return are strictly limited.</p>
<p>Projects funded in 2020 are</p>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">projects_2020</span> <span class="op">&lt;-</span></span>
<span>    <span class="va">projects</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">fiscal_year</span> <span class="op">==</span> <span class="st">"2020"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">project_summaries</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/distinct.html" class="external-link">distinct</a></span><span class="op">(</span></span>
<span>        <span class="va">full_foa</span>, <span class="va">core_project_num</span>,</span>
<span>        <span class="va">n_years</span>, <span class="va">amount</span>, <span class="va">project_title</span></span>
<span>    <span class="op">)</span></span>
<span><span class="co">#&gt; Joining with `by = join_by(core_project_num, project_title)`</span></span></code></pre></div>
<p>There are 42 projects receiving funds in fiscal year 2020.
Publications associated with these projects are obtained with</p>
<div class="sourceCode" id="cb18"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">publications</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/user_interface.html">reporter_publications</a></span><span class="op">(</span></span>
<span>    core_project_nums <span class="op">=</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/pull.html" class="external-link">pull</a></span><span class="op">(</span><span class="va">projects_2020</span>, <span class="st">"core_project_num"</span><span class="op">)</span></span>
<span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="op">-</span><span class="va">applid</span><span class="op">)</span> <span class="co"># we won't use 'applid'</span></span></code></pre></div>
<p>There are 704 distinct publications (not all published in 2020).</p>
<div class="section level3">
<h3 id="project-publication-and-funding">Project publication and funding<a class="anchor" aria-label="anchor" href="#project-publication-and-funding"></a>
</h3>
<p>The most prolific projects are</p>
<div class="sourceCode" id="cb19"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">publications_by_project</span> <span class="op">&lt;-</span></span>
<span>    <span class="va">publications</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count</a></span><span class="op">(</span><span class="va">coreproject</span>, sort <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span></span>
<span>        <span class="va">projects_2020</span>,</span>
<span>        by <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span>coreproject <span class="op">=</span> <span class="st">"core_project_num"</span><span class="op">)</span>,</span>
<span>        multiple <span class="op">=</span> <span class="st">"last"</span></span>
<span>    <span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="va">publications_by_project</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-3dbc8ef8026d90dea7f8" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-3dbc8ef8026d90dea7f8">{"x":{"filter":"none","vertical":false,"data":[["U01CA239055","U24CA180996","R21CA231911","U24CA209851","U24CA194215","U24CA184427","U24CA215109","U24CA180922","U24CA220341","U24CA237719","U01CA242871","U01CA231840","U01CA242936","U24CA209999","U24CA231877","U01CA242879","U24CA204800","R21CA242933","U24CA231846","U24CA204817","U24CA215123","U01CA231782","U24CA220242","U01CA237711","U01CA242954","U24CA237617","U01CA226187","U01CA226196","U24CA210004","U24CA220441","R21CA237493","U24CA220457","R21CA237686","R21CA242940","U24CA204863","R21CA242861","U24CA204854","U24CA204781","U24CA209996","R21CA226106","U01CA231860","R21CA226188"],[73,47,45,43,41,40,32,31,28,28,26,22,21,19,18,15,15,14,13,12,12,11,10,9,9,9,8,8,7,7,6,6,5,5,5,4,4,3,3,2,2,1],["PAR-15-332","PAR-15-331","PAR-15-334","PAR-15-331","PAR-13-294","PAR-15-331","PAR-15-333","PAR-15-331","PAR-15-333","PAR-15-331","PAR-15-332","PAR-15-332","PAR-15-332","PAR-15-331","PAR-15-331","PAR-15-332","PAR-15-331","PAR-15-334","PAR-15-333","PAR-13-294","PAR-15-331","PAR-15-332","PAR-15-331","PAR-15-332","PAR-15-332","PAR-15-331","PAR-15-332","PAR-15-332","PAR-15-333","PAR-15-331","PAR-15-334","PAR-15-333","PAR-15-334","PAR-15-334","PAR-15-331","PAR-15-334","PAR-13-294","PAR-13-294","PAR-15-331","PAR-15-334","PAR-15-332","PAR-15-334"],[4,8,2,5,5,8,5,10,5,4,3,3,3,5,5,3,5,2,4,5,5,3,5,3,3,5,3,3,5,4,2,5,2,2,5,2,5,5,5,2,3,2],[1045548,5281746,388776,3905934,3274448,5658437,8140937,7263992,3392981,2942615,1096884,1239743,1075520,3743640,3850721,1191276,3747474,386767,2570825,2493392,4207966,980371,2627986,1127188,1339635,3639833,1165246,1286835,3802236,2737046,372034,6365918,368539,369895,3824879,602867,3040146,1904229,3962385,460732,1518187,394763],["HistoTools:  A suite of digital pathology tools for quality control, annotation and dataset identification","Cancer Genomics: Integrative and Scalable Solutions in R/Bioconductor","STAN-CT: Standardization and Normalization of CT images for Lung Cancer Patients","TCPA: an Integrated Bioinformatics Resource for Functional Cancer Proteomic Data","Advancing Cancer Pharmacoepidemiology Research Through EHRs and Informatics","NDEx - the Network Data Exchange A Network Commons for Biologists","TCIA Sustainment and Scalability - Platforms for Quantitative Imaging Informatics in Precision Medicine","Trinity: Transcriptome assembly for genetic and functional analysis of cancer","Supporting and evolving Gene Set Enrichment Analysis and the Molecular Signatures Database for cancer research","Standardized and Genome-Wide Clinical Interpretation of Complex Genotypes for Cancer Precision Medicine","The Federated Tumor Segmentation (FeTS) platform: An intuitive tool facilitating secure multi-institutional collaboration","Network Analysis for a Data-Driven Approach to Cancer Care","Computational pathology software for integrative cancer research with three-dimensional digital slides","Monitoring tumor subclonal heterogeneity over time and space","A Federated Galaxy for user-friendly large-scale cancer genomics research","Distributed Learning of Deep Learning Models for Cancer Research","Scalable Clinical Decision Support for Individualized Cancer Risk Management","Data-driven QSP software for personalized colon cancer treatment","WebMeV: A Robust Platform for Intuitive Genomic Data Analysis","Informatics Tools for High-throughput Analysis of Cancer Mutations","The TOPAS Tool for Particle Simulation, a Monte Carlo Simulation Tool for Physics, Biology and Clinical Research","A Unified Machine Learning Package for Cancer Diagnosis","Detection of somatic, subclonal and mosaic CNVs from sequencing","Informatics resources for liquid biopsy research","Eliminating variants of uncertain significance in BRCA1, BRCA2 and beyond","Developing Informatics Technologies to Model Cancer Gene Regulation","Tools for annotating mutations in the 3D cancer genome","Bioinformatics Technology to Characterize Tumor Infiltrating Immune Repertoires","The Integrative Genomics Viewer (IGV): visualization supporting cancer research","Developing the JBrowse Genome Browser to Visualize Structural Variants and Cancer Genomics Data","Development and Dissemination of KiNet: A Novel Imaging Informatics Tool for Gastrointestinal and Pancreatic Neuroendocrine Tumors","The cBioPortal for Cancer Genomics","Personalized dosimetry for liver cancer radioembolization using fluid dynamics simulation","Semi-supervised Algorithms for Risk Assessment with Noisy EHR Data","Advanced Development and Dissemination of EMERSE for Cancer Phenotyping from Medical Records","A statistical framework to systematically characterize cancer driver mutations in noncoding genomic regions","INTEGRATIVE IMAGING INFORMATICS FOR CANCER RESEARCH","The PDX Integrator- Re-purposing KOMP2 informatics to widely disseminate data from Patient Derived Xenografts","Building protected data sharing networks to advance cancer risk assessment and treatment","Conservation and functional-characterization of tumor methylation sites","Open-access X-ray and CT simulation toolkit for research in cancer imaging and dosimetry","Enhanced deconvolution and prediction of mutational signatures"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>coreproject<\/th>\n      <th>n<\/th>\n      <th>full_foa<\/th>\n      <th>n_years<\/th>\n      <th>amount<\/th>\n      <th>project_title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":[1,3,4]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script><p>An exceptional number of publications are reported by project number
‘U01CA239055’. The relationship between publication and funding period
and amount are visualized as follows.</p>
<div class="sourceCode" id="cb20"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">publications_by_project</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span><span class="va">n_years</span>, <span class="va">n</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_point.html" class="external-link">geom_point</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_smooth.html" class="external-link">geom_smooth</a></span><span class="op">(</span>method <span class="op">=</span> <span class="st">"lm"</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/scale_continuous.html" class="external-link">scale_x_continuous</a></span><span class="op">(</span>breaks <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/seq.html" class="external-link">seq</a></span><span class="op">(</span><span class="fl">1</span>, <span class="fl">10</span><span class="op">)</span>, minor_breaks <span class="op">=</span> <span class="cn">NULL</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html" class="external-link">labs</a></span><span class="op">(</span>x <span class="op">=</span> <span class="st">"Fiscal years of award"</span>, y <span class="op">=</span> <span class="st">"Number of publications"</span><span class="op">)</span></span>
<span><span class="co">#&gt; `geom_smooth()` using formula = 'y ~ x'</span></span>
<span></span>
<span><span class="va">publications_by_project</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span><span class="va">amount</span>, <span class="va">n</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_point.html" class="external-link">geom_point</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_smooth.html" class="external-link">geom_smooth</a></span><span class="op">(</span>method <span class="op">=</span> <span class="st">"lm"</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html" class="external-link">labs</a></span><span class="op">(</span>x <span class="op">=</span> <span class="st">"Total funding"</span>, y <span class="op">=</span> <span class="st">"Number of publications"</span><span class="op">)</span></span>
<span><span class="co">#&gt; `geom_smooth()` using formula = 'y ~ x'</span></span></code></pre></div>
<p><img src="case_study_itcr_files/figure-html/unnamed-chunk-19-1.png" width="45%"><img src="case_study_itcr_files/figure-html/unnamed-chunk-19-2.png" width="45%"></p>
</div>
<div class="section level3">
<h3 id="itcr-collaboration">ITCR collaboration<a class="anchor" aria-label="anchor" href="#itcr-collaboration"></a>
</h3>
<p>ITCR emphasizes collaboration between funded projects. Are there
examples of collaboration at the level of publication, i.e.,
<code>pmid</code> associated with more than one project number?</p>
<div class="sourceCode" id="cb21"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">collaborative_pmid</span> <span class="op">&lt;-</span></span>
<span>    <span class="va">publications</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count</a></span><span class="op">(</span><span class="va">pmid</span>, sort <span class="op">=</span> <span class="cn">TRUE</span>, name <span class="op">=</span> <span class="st">"n_collab"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">n_collab</span> <span class="op">&gt;</span> <span class="fl">1</span><span class="op">)</span></span>
<span><span class="va">collaborative_pmid</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 14 × 2</span></span></span>
<span><span class="co">#&gt;        pmid n_collab</span></span>
<span><span class="co">#&gt;       <span style="color: #949494; font-style: italic;">&lt;int&gt;</span>    <span style="color: #949494; font-style: italic;">&lt;int&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span> 35<span style="text-decoration: underline;">072</span>136        3</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span> 29<span style="text-decoration: underline;">533</span>785        2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span> 30<span style="text-decoration: underline;">930</span>166        2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span> 31<span style="text-decoration: underline;">238</span>109        2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span> 32<span style="text-decoration: underline;">383</span>980        2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span> 33<span style="text-decoration: underline;">067</span>482        2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span> 33<span style="text-decoration: underline;">119</span>407        2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span> 33<span style="text-decoration: underline;">237</span>278        2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span> 33<span style="text-decoration: underline;">574</span>288        2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">10</span> 33<span style="text-decoration: underline;">720</span>842        2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">11</span> 34<span style="text-decoration: underline;">514</span>469        2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">12</span> 35<span style="text-decoration: underline;">311</span>178        2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">13</span> 35<span style="text-decoration: underline;">759</span>080        2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">14</span> 35<span style="text-decoration: underline;">760</span>813        2</span></span></code></pre></div>
<p>Which projects are collaborating through shared publication?</p>
<div class="sourceCode" id="cb22"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">collaborative_pmid</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">publications</span>, multiple <span class="op">=</span> <span class="st">"all"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count</a></span><span class="op">(</span><span class="va">coreproject</span>, sort <span class="op">=</span> <span class="cn">TRUE</span>, name <span class="op">=</span> <span class="st">"n_collab"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">project_summaries</span>, by <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span>coreproject <span class="op">=</span> <span class="st">"core_project_num"</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span>core_project_num <span class="op">=</span> <span class="va">coreproject</span>, <span class="va">n_collab</span>, <span class="va">project_title</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/distinct.html" class="external-link">distinct</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">n_collab</span> <span class="op">&gt;</span> <span class="fl">1</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; Joining with `by = join_by(pmid)`</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-b48420a6bfa40cf850d7" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-b48420a6bfa40cf850d7">{"x":{"filter":"none","vertical":false,"data":[["U01CA231840","U24CA194215","U24CA237719","U01CA242871","U01CA242954","U24CA180996","U24CA220457","U24CA231877"],[3,3,3,2,2,2,2,2],["Network Analysis for a Data-Driven Approach to Cancer Care","Advancing Cancer Pharmacoepidemiology Research Through EHRs and Informatics","Standardized and Genome-Wide Clinical Interpretation of Complex Genotypes for Cancer Precision Medicine","The Federated Tumor Segmentation (FeTS) platform: An intuitive tool facilitating secure multi-institutional collaboration","Eliminating variants of uncertain significance in BRCA1, BRCA2 and beyond","Cancer Genomics: Integrative and Scalable Solutions in R/Bioconductor","The cBioPortal for Cancer Genomics","A Federated Galaxy for user-friendly large-scale cancer genomics research"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>core_project_num<\/th>\n      <th>n_collab<\/th>\n      <th>project_title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":1}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="section level2">
<h2 id="citations">Citations<a class="anchor" aria-label="anchor" href="#citations"></a>
</h2>
<p>Query <a href="https://icite.od.nih.gov/" class="external-link">iCite</a> for all citations
to the publications associated with ITCR projects funded in 2020,
restricting output to include fields of interest.</p>
<div class="sourceCode" id="cb23"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">include_fields</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"pmid"</span>, <span class="st">"year"</span>, <span class="st">"citation_count"</span>, <span class="st">"relative_citation_ratio"</span>,</span>
<span>    <span class="st">"year"</span>, <span class="st">"title"</span>, <span class="st">"authors"</span>, <span class="st">"journal"</span>, <span class="st">"doi"</span></span>
<span><span class="op">)</span></span>
<span><span class="va">citations</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/user_interface.html">icite</a></span><span class="op">(</span><span class="va">publications</span>, <span class="va">include_fields</span><span class="op">)</span></span>
<span><span class="va">citations</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">publications</span>, <span class="va">citations</span><span class="op">)</span></span>
<span><span class="co">#&gt; Joining with `by = join_by(pmid)`</span></span></code></pre></div>
<p>The 719 publications have 36233 total citations; 98 publications have
not been cited; not surprisingly uncited publications are recent.</p>
<div class="sourceCode" id="cb24"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">citations</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/group_by.html" class="external-link">group_by</a></span><span class="op">(</span><span class="va">year</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/summarise.html" class="external-link">summarize</a></span><span class="op">(</span></span>
<span>        n_projects <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/length.html" class="external-link">length</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">coreproject</span><span class="op">)</span><span class="op">)</span>,</span>
<span>        n_pub <span class="op">=</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/context.html" class="external-link">n</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>        n_uncite <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/sum.html" class="external-link">sum</a></span><span class="op">(</span><span class="va">citation_count</span> <span class="op">==</span> <span class="fl">0L</span><span class="op">)</span>,</span>
<span>        citn_count <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/sum.html" class="external-link">sum</a></span><span class="op">(</span><span class="va">citation_count</span><span class="op">)</span></span>
<span>    <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu">DT</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-0c2da75d4a82c7dfabde" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0c2da75d4a82c7dfabde">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11"],[2010,2014,2015,2016,2017,2018,2019,2020,2021,2022,2023],[1,3,2,5,11,13,30,36,34,33,10],[1,3,4,18,44,62,103,183,167,123,11],[0,0,0,0,0,1,6,8,25,50,8],[32,2546,604,4241,8749,6471,5771,5191,2200,419,9]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>year<\/th>\n      <th>n_projects<\/th>\n      <th>n_pub<\/th>\n      <th>n_uncite<\/th>\n      <th>citn_count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script><p>Citations follow a very familiar pattern, with a few publications
cited frequently.</p>
<div class="sourceCode" id="cb25"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">citations</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">citation_count</span> <span class="op">&gt;</span> <span class="fl">0</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">citation_count</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/scale_continuous.html" class="external-link">scale_x_log10</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_density.html" class="external-link">geom_density</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html" class="external-link">xlab</a></span><span class="op">(</span><span class="st">"Number of citations"</span><span class="op">)</span> <span class="op">+</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html" class="external-link">ylab</a></span><span class="op">(</span><span class="st">"Number of publications"</span><span class="op">)</span></span></code></pre></div>
<p><img src="case_study_itcr_files/figure-html/unnamed-chunk-24-1.png" width="700"></p>
<p>Highly cited publications are in high-impact journals, and emphasize
science-related results rather than a software tool <em>per se</em> –
the tool has been used in an important study, and the authors of the
study have acknowledged the tool.</p>
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">citations</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/desc.html" class="external-link">desc</a></span><span class="op">(</span><span class="va">citation_count</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="va">coreproject</span>, <span class="va">pmid</span>, <span class="va">year</span>, <span class="va">citation_count</span>, <span class="va">journal</span>, <span class="va">title</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-6e98b2536105159f7561" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-6e98b2536105159f7561">{"x":{"filter":"none","vertical":false,"data":[["U24CA180922","U24CA180922","U24CA180922","U24CA209851","U01CA231840","U24CA180922","U24CA204817","U24CA210004","U24CA209999","U24CA209851","U24CA180922","U24CA180922","U24CA180922","U24CA180922","U24CA210004","U24CA210004","U24CA180922","U24CA209851","U24CA215109","U24CA180922","U01CA239055","U24CA180922","U24CA180996","U24CA180996","U24CA209851","U24CA180922","U24CA180922","U24CA220457","U24CA180922","U24CA209999","U24CA204817","U24CA180996","U24CA180996","U24CA180922","U24CA180922","U24CA180922","U24CA204817","U24CA209851","U24CA184427","U01CA242871","U01CA242879","U24CA184427","U24CA180996","U24CA180996","U24CA220457","U24CA231877","U24CA237719","U24CA184427","U24CA231877","U24CA204817","U24CA209851","U01CA231840","U24CA184427","U24CA180996","U24CA180922","U24CA209851","U24CA209851","U24CA209999","U24CA194215","U24CA209851","U24CA204817","U24CA210004","U01CA242871","U24CA231877","U24CA184427","U24CA204817","U24CA209851","U24CA220341","U24CA209999","U24CA237719","U24CA180922","U24CA180922","U24CA209851","U24CA209851","U01CA239055","U24CA209851","U01CA231840","U01CA231840","R21CA242861","U24CA220341","U24CA184427","U24CA180922","U24CA237719","U24CA210004","U24CA215109","U24CA209851","U24CA220341","U24CA184427","U24CA237617","U24CA209999","U24CA180996","U24CA215109","U24CA209999","U24CA209851","U24CA209851","U01CA226196","U24CA204781","U24CA180996","U24CA204817","U24CA194215","U24CA237719","U24CA184427","U24CA209999","U24CA237719","U24CA209851","U01CA239055","U24CA209851","U24CA237617","U24CA209851","U24CA204817","U24CA204781","U01CA239055","U24CA184427","U24CA215109","U01CA239055","U24CA180996","U24CA180922","U24CA180922","U24CA209851","U24CA215109","U24CA204817","U01CA239055","U24CA215123","U24CA180996","U01CA239055","U24CA209999","U24CA184427","U24CA220242","U24CA220341","U24CA180996","U24CA184427","U01CA239055","U01CA231840","U24CA215123","U24CA180996","U24CA237719","U24CA237617","U24CA231877","U24CA209851","U01CA242954","U01CA239055","U24CA204817","U24CA180922","U24CA237719","U24CA231877","U24CA184427","U24CA184427","U24CA180996","U24CA180922","U01CA239055","U24CA209851","U24CA184427","U01CA226196","U01CA226187","U24CA215109","U24CA209851","U24CA204817","U24CA194215","U24CA180922","U01CA239055","U24CA180996","U01CA242871","U01CA239055","U24CA204863","U24CA194215","U01CA231840","U01CA226196","U24CA220341","U24CA180996","U24CA237719","U24CA209851","R21CA231911","U24CA231877","U24CA220242","U24CA209851","U24CA204817","U24CA194215","U01CA226187","U24CA231877","U24CA220341","U24CA209851","U24CA184427","U01CA239055","U01CA239055","U01CA231782","U01CA226196","U24CA215109","U24CA209999","U01CA242954","U24CA220341","U24CA184427","U01CA239055","U24CA237719","U24CA215109","U24CA184427","U24CA184427","U24CA180996","U24CA231877","U24CA220341","U24CA209996","U24CA209851","U24CA194215","U24CA180996","U01CA231840","U24CA215109","U24CA194215","U01CA239055","R21CA242933","U24CA209999","U24CA184427","U01CA239055","U01CA226187","U01CA239055","R21CA242861","U24CA220341","U24CA215109","U24CA209851","U01CA242871","U01CA239055","U01CA231840","U24CA231846","U24CA220341","U24CA215109","U24CA204854","U24CA204800","U24CA180996","U01CA239055","R21CA231911","U24CA237719","U24CA231846","U24CA209999","U24CA194215","U24CA180996","U01CA242936","U01CA242871","U01CA239055","U01CA237711","R21CA237493","U24CA237617","U24CA184427","U24CA184427","U24CA184427","U24CA180996","U24CA180996","U01CA239055","U01CA239055","U01CA239055","U01CA231840","R21CA231911","U24CA237719","U24CA231877","U24CA231846","U24CA215109","U24CA209999","U24CA209851","U24CA204800","U24CA184427","U24CA180996","U01CA239055","U01CA231840","U01CA231840","R21CA237686","U24CA220457","U24CA215123","U24CA209851","U24CA204800","U24CA194215","U24CA180996","U24CA180996","U01CA239055","U01CA239055","U01CA226187","U24CA220341","U24CA220341","U24CA209851","U24CA209851","U24CA209851","U24CA204863","U24CA194215","U24CA184427","U24CA184427","U01CA242879","U01CA242871","U01CA237711","R21CA242940","R21CA242933","R21CA242933","U24CA237719","U24CA215109","U24CA209999","U24CA204854","U24CA194215","U01CA242879","U01CA239055","U01CA239055","R21CA231911","U24CA220457","U24CA220242","U24CA215109","U24CA209999","U24CA194215","U24CA194215","U24CA194215","U24CA194215","U01CA239055","U01CA239055","U01CA239055","U01CA231840","U24CA231846","U24CA220242","U24CA215109","U24CA209851","U24CA204863","U24CA204854","U24CA194215","U24CA194215","U24CA194215","U24CA194215","U24CA194215","U24CA180996","U01CA242954","U01CA239055","U01CA237711","R21CA242933","R21CA242933","R21CA231911","U24CA237617","U24CA220341","U24CA220341","U24CA215109","U24CA194215","U24CA194215","U24CA184427","U24CA180996","U24CA180996","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA226196","U24CA237719","U24CA220341","U24CA220242","U24CA215109","U24CA194215","U24CA194215","U24CA184427","U24CA180996","U01CA242954","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA226187","R21CA237686","U24CA237719","U24CA237719","U24CA237719","U24CA231877","U24CA231846","U24CA231846","U24CA220441","U24CA220441","U24CA220242","U24CA220242","U24CA215123","U24CA215109","U24CA209851","U24CA204800","U24CA204800","U24CA194215","U24CA194215","U24CA180996","U24CA180922","U01CA242954","U01CA242936","U01CA242936","U01CA242879","U01CA242879","U01CA242871","U01CA242871","U01CA239055","U01CA231782","U01CA226187","R21CA242933","R21CA242861","R21CA237493","R21CA231911","U24CA220341","U24CA209851","U24CA204800","U24CA184427","U24CA180996","U01CA239055","U01CA239055","U01CA237711","U01CA226196","U01CA226187","R21CA237493","R21CA231911","R21CA231911","R21CA231911","U24CA237719","U24CA231877","U24CA231846","U24CA220441","U24CA220341","U24CA220341","U24CA209851","U24CA204800","U24CA184427","U24CA180996","U24CA180922","U01CA242936","U01CA242871","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA237711","U01CA231840","U01CA231840","U01CA226196","R21CA242861","R21CA237686","R21CA231911","R21CA231911","R21CA231911","U24CA237719","U24CA231877","U24CA231846","U24CA231846","U24CA215123","U24CA215123","U24CA215109","U24CA215109","U24CA215109","U24CA210004","U24CA209999","U24CA209851","U24CA204863","U24CA204800","U24CA184427","U24CA184427","U24CA180996","U24CA180996","U01CA242954","U01CA239055","U01CA239055","U01CA231782","R21CA242933","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA226188","U24CA237719","U24CA220341","U24CA220341","U24CA220341","U24CA220341","U24CA215123","U24CA215123","U24CA215109","U24CA210004","U24CA204800","U24CA194215","U24CA184427","U24CA184427","U24CA180996","U24CA180996","U24CA180996","U24CA180996","U01CA242936","U01CA242879","U01CA242871","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA231840","U01CA231840","R21CA231911","R21CA231911","R21CA231911","U24CA237719","U24CA237719","U24CA231877","U24CA220341","U24CA215123","U24CA215109","U24CA215109","U24CA209999","U24CA194215","U24CA194215","U01CA242936","U01CA242871","U01CA239055","U01CA237711","U01CA237711","U01CA231782","U01CA231782","R21CA242933","R21CA237686","R21CA237493","R21CA237493","R21CA231911","R21CA231911","R21CA231911","R21CA226106","U24CA237719","U24CA237617","U24CA231877","U24CA231877","U24CA231877","U24CA231846","U24CA220441","U24CA220441","U24CA220341","U24CA215109","U24CA215109","U24CA209999","U24CA209996","U24CA209851","U24CA204800","U24CA194215","U24CA194215","U24CA194215","U24CA194215","U24CA184427","U24CA184427","U24CA184427","U24CA180996","U24CA180996","U24CA180922","U24CA180922","U24CA180922","U01CA242936","U01CA242879","U01CA242879","U01CA242871","U01CA242871","U01CA242871","U01CA242871","U01CA242871","U01CA239055","U01CA239055","U01CA239055","U01CA231860","U01CA231840","U01CA231840","U01CA231840","U01CA231840","U01CA231782","U01CA231782","U01CA226187","R21CA242933","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA231911","U24CA237719","U24CA237617","U24CA220441","U24CA220441","U24CA220341","U24CA220341","U24CA220242","U24CA220242","U24CA220242","U24CA215123","U24CA215123","U24CA215109","U24CA215109","U24CA209999","U24CA209851","U24CA209851","U24CA204800","U24CA194215","U24CA194215","U24CA194215","U24CA194215","U24CA194215","U24CA194215","U24CA180996","U01CA242936","U01CA242936","U01CA242936","U01CA242936","U01CA242936","U01CA242879","U01CA242879","U01CA242879","U01CA242879","U01CA242871","U01CA242871","U01CA242871","U01CA242871","U01CA242871","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA237711","U01CA237711","U01CA231840","U01CA231782","U01CA226196","R21CA242933","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA231911","U24CA237719","U24CA237719","U24CA237719","U24CA237719","U24CA237719","U24CA237617","U24CA237617","U24CA231877","U24CA231877","U24CA231846","U24CA231846","U24CA231846","U24CA220457","U24CA220457","U24CA220341","U24CA220341","U24CA215123","U24CA215109","U24CA215109","U24CA215109","U24CA215109","U24CA209999","U24CA209996","U24CA209851","U24CA209851","U24CA204863","U24CA204854","U24CA204800","U24CA204800","U24CA204800","U24CA204800","U24CA204781","U24CA194215","U24CA194215","U24CA184427","U24CA184427","U24CA184427","U24CA184427","U24CA180996","U24CA180996","U24CA180996","U24CA180996","U24CA180996","U24CA180996","U01CA242954","U01CA242954","U01CA242954","U01CA242936","U01CA242936","U01CA242936","U01CA242936","U01CA242936","U01CA242936","U01CA242936","U01CA242936","U01CA242936","U01CA242879","U01CA242879","U01CA242879","U01CA242871","U01CA242871","U01CA242871","U01CA242871","U01CA242871","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA239055","U01CA231860","U01CA231840","U01CA231782","U01CA231782","U01CA231782","R21CA242940","R21CA242940","R21CA242940","R21CA242940","R21CA242933","R21CA242933","R21CA242933","R21CA242933","R21CA237686","R21CA237493","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA231911","R21CA226106"],[24925914,27124452,31068700,28988769,32473681,29198524,29625053,28985562,29431738,28572459,31327527,27806376,30388455,28360267,29092934,28263302,28099853,30349115,29617659,28102262,31399699,28902842,28035024,30936548,28528867,27824854,26687005,30373752,29674595,29096012,30096302,34784344,27400279,27610572,28886383,31639029,27911828,30380420,29107334,33015372,32068507,27105112,31792435,27366979,30894748,32302568,33237278,33237278,32479607,30190408,29625054,33746047,26594663,29088129,26638175,28196595,29617665,28804138,29186491,29533785,29533785,29428417,32724046,32243801,29605183,29563323,29706454,30773340,30531870,32665297,26838501,33592174,28411194,31984309,31719058,30962452,34137799,32699031,32015527,31171722,25086664,32355025,31462330,30205044,29288495,28839186,30563849,28188128,32033573,29093439,33057217,30607307,31019089,31344359,30254248,30742113,29092942,30643296,26442818,31265066,32246132,29610481,29309061,31907209,29142225,32848206,29617661,32839606,29092939,27197156,30535239,34663898,31819800,31939789,33334576,30084834,29867192,28648452,30633903,28925987,31202631,32628863,32247964,32026945,31216884,30109124,28150243,30737237,31097695,29092936,29023449,32079590,34958894,32392626,33469025,35072136,31053864,35072136,31682236,35072136,32835732,29254494,34594037,30287923,33727309,29636367,29092941,24931973,26900662,32216127,33007258,32066900,30930166,30930166,30631215,30545954,29092935,31135882,33495604,31446979,26463000,35649412,33051342,19898898,29854252,31395609,33986545,32848210,32746888,33260207,31586392,31575946,34824477,30864654,32079540,32228266,29462669,31851943,32122539,31913126,31201206,33362771,33163952,32781640,31397859,31771646,32277893,31222198,31862013,31932491,31521603,34168282,32180547,31040006,27417679,26083491,32747365,35446428,31292160,31871109,29736411,31238109,31151886,31238109,32153349,32477646,33589781,33003193,29048539,29949979,32120229,31097707,31259199,33230298,32530570,29092946,33157050,32152622,32123864,32923847,33536619,33046443,32723384,34328530,29941415,34644528,31703155,32533855,31796060,31806877,32664994,28602904,31871106,32661341,32603860,32343872,33045922,31841948,32424124,32963239,33095781,34591642,31076212,30866806,32722082,32538136,33176026,32194985,33398948,35101336,30596645,31639475,34025103,29860504,30423406,31951474,34591612,27008010,32139401,33176160,32117976,32006268,33119407,31117056,30297703,31258969,31141421,31217513,33119407,33068415,32876839,32312293,31974273,32276990,29579225,29792826,32270762,32739208,30815153,31174177,33413539,32196092,33118182,31598384,32548637,33619294,33757216,32644817,29907888,29718142,33211330,34224163,33177518,33676102,34167765,32528219,32641410,31787246,32561748,29873782,29541700,30238070,32308921,30700301,33321450,33576688,33352373,33411620,34133929,34817058,32479186,32976776,32412846,34359696,31801515,28815141,31684865,30131701,32569358,30253799,35128509,34853071,32732909,34068946,33291412,32476708,34596551,33428749,35879302,29888078,29206922,30066643,30878356,27381624,29556541,32967377,31857377,33197281,32816791,33059736,31645350,33542242,33122304,33574288,32930711,32012237,29608663,32282230,34793697,32700021,34362928,33743483,33574288,32822291,32832663,32601676,31779674,33547198,35311178,34667258,32860050,34508353,28494756,28104629,32622340,35901164,32521512,31834820,30297783,34632321,32599144,31801529,29854175,33564398,34146471,35311178,32520928,33420322,33863957,33902889,32644158,32577629,33750343,31540699,33219054,34071939,35389777,31825019,32447262,32525984,32330068,34100949,32637990,28152060,35256515,33651718,34001525,34772935,32127393,31521104,32489227,32476702,32577045,31990579,35243422,34493595,34196689,34376580,32383980,32374631,30468912,29087446,27923063,32383980,33766651,35610333,33166198,33941504,31623652,33860966,34234141,33579955,31990580,30412202,35121878,32610459,34955584,32476709,32528216,31618044,35217711,31062858,35193937,34384063,32454570,35286309,35565277,34405297,36562559,29707261,29716986,31023520,34078380,33116744,34992246,28221771,34499096,34648520,34350068,32470903,33302246,34440778,32476705,32528218,33936486,31751240,32190408,34625425,35475554,33109540,34667113,31695841,34641956,34280910,33896732,34272388,31105932,34781009,31438083,34570431,35337019,30202918,35482478,32453635,33511368,33616620,33309994,33571780,35115363,34265509,34518361,32155093,35198648,35687337,34589556,32476700,35783088,32191543,34036230,34551429,33571541,31515016,29081725,32772385,29588547,34272955,32364754,35178539,35906241,35579955,36175411,31906978,32216637,35211363,32647406,34103097,32730116,33284750,32476704,32124223,32476701,32078366,36373660,36068320,35760813,31197310,34061826,33968133,33020189,31099383,31513970,35966128,35674341,34711268,36035065,33740422,29941416,35917480,29622869,33067482,33720842,28616579,31022372,34580671,31297189,35760813,36171387,34099491,34734986,32802270,35390737,35749336,36137534,34764084,35465060,36283200,32116512,35405081,34205005,34388099,33426704,33720842,35416993,33067482,33411619,33088907,35501416,36113481,35782394,35755403,35484692,32476706,32577044,32606488,35624339,36191105,32683444,32810130,35393416,35715421,36201590,32777815,33183232,32388464,33035214,34178560,31054503,27480386,36332624,35466854,36194411,32537571,33936520,32308886,30943955,35157022,31566474,30473781,35962988,34650348,35617723,34004004,30891467,34405896,35867397,32548281,36624800,34282411,36198326,34514469,34514469,35372431,31372599,35007352,35108202,35184149,35671506,35149536,32923903,35885617,35605261,35368616,32528212,35755405,35784009,32528211,35177877,35755404,32476707,32528217,35756897,34341766,35611186,35366592,36273232,36541006,36166553,36688700,36622853,35669346,35759080,34573415,35459940,33317447,35759080,36371231,35977718,34915451,36037173,35082552,36279873,30402669,33771218,35979414,34492340,35138907,36481564,35202205,34792472,35224632,35969459,36532299,36399494,32537570,35359803,31647918,32587977,36653526,32511379,33560368,34860667,30828438,30689687,31249680,35266509,36540993,35360665,35373174,35797891,35572741,35368366,34548902,36067783,36222817,36222820,32954255,36072353,36745141,34815983,35995898,34382033,35035351,35701399,34621541,36331281,35727603,36281473,36516556,35537220,34252041,34334397,35689952,36066461,35085099,36744216,36096127,34932493,36745147,35525068,36589620,32413171,32562264,35166342,34747010,36276480,36551627,36568126,35212794,35301128,36383499,29905905,35308983,32528213,32528215,35578386,35756345,35784397,35781919,33299548],[2014,2016,2019,2017,2020,2017,2018,2017,2018,2017,2019,2016,2018,2017,2017,2017,2017,2018,2018,2017,2019,2017,2017,2019,2017,2016,2015,2019,2018,2018,2018,2021,2016,2016,2017,2019,2016,2018,2017,2020,2020,2016,2020,2016,2019,2020,2021,2021,2020,2018,2018,2021,2015,2017,2015,2017,2018,2017,2018,2018,2018,2018,2020,2020,2018,2018,2018,2019,2019,2020,2016,2021,2017,2019,2020,2019,2021,2020,2020,2019,2014,2020,2019,2018,2018,2017,2019,2017,2020,2017,2020,2018,2019,2019,2018,2019,2017,2019,2016,2019,2020,2018,2018,2020,2017,2020,2018,2020,2017,2016,2019,2022,2019,2020,2020,2018,2018,2017,2019,2017,2019,2020,2020,2021,2019,2018,2017,2019,2019,2017,2017,2020,2022,2021,2021,2021,2019,2021,2019,2021,2021,2017,2021,2019,2021,2018,2017,2014,2016,2020,2020,2020,2019,2019,2019,2019,2017,2020,2021,2019,2016,2022,2020,2010,2018,2019,2021,2020,2020,2021,2020,2019,2022,2019,2020,2020,2018,2019,2020,2020,2019,2020,2020,2020,2019,2019,2020,2019,2019,2020,2019,2021,2020,2020,2016,2015,2020,2022,2019,2020,2018,2019,2019,2019,2020,2020,2021,2021,2017,2018,2020,2019,2019,2020,2020,2017,2020,2020,2020,2019,2021,2020,2020,2022,2018,2021,2020,2020,2019,2020,2020,2017,2020,2020,2020,2020,2021,2020,2020,2020,2020,2021,2019,2019,2020,2020,2020,2020,2021,2022,2018,2020,2019,2018,2019,2020,2021,2016,2020,2020,2020,2020,2020,2019,2018,2019,2019,2019,2020,2021,2021,2020,2020,2020,2019,2018,2020,2020,2018,2019,2021,2020,2020,2019,2020,2021,2021,2020,2018,2018,2021,2021,2020,2021,2021,2020,2020,2020,2020,2018,2017,2018,2020,2019,2021,2021,2021,2021,2021,2021,2020,2021,2020,2021,2019,2017,2019,2018,2020,2018,2021,2022,2020,2021,2020,2020,2022,2021,2022,2018,2018,2018,2019,2016,2018,2020,2020,2021,2021,2020,2019,2021,2020,2020,2020,2020,2018,2020,2021,2021,2021,2021,2020,2021,2020,2021,2019,2021,2021,2021,2020,2022,2017,2017,2020,2022,2020,2019,2018,2020,2021,2019,2018,2020,2021,2021,2020,2021,2021,2021,2020,2019,2021,2020,2021,2021,2022,2019,2020,2020,2020,2021,2021,2017,2022,2021,2021,2021,2020,2019,2019,2020,2020,2020,2022,2021,2021,2021,2020,2020,2019,2018,2016,2020,2022,2022,2020,2021,2019,2021,2021,2020,2020,2018,2021,2020,2021,2020,2020,2019,2022,2019,2022,2021,2020,2022,2022,2021,2023,2017,2018,2019,2021,2020,2022,2017,2021,2021,2021,2020,2021,2021,2020,2020,2021,2019,2019,2021,2022,2020,2022,2019,2021,2021,2021,2021,2018,2021,2019,2021,2022,2020,2022,2020,2021,2021,2021,2021,2022,2021,2021,2020,2022,2022,2021,2019,2021,2020,2021,2021,2021,2019,2017,2020,2018,2021,2020,2021,2022,2022,2022,2020,2020,2021,2020,2021,2020,2021,2020,2020,2020,2020,2023,2022,2022,2020,2021,2021,2020,2019,2019,2021,2022,2021,2022,2021,2018,2022,2018,2020,2021,2017,2019,2021,2019,2022,2022,2021,2022,2020,2022,2022,2022,2022,2022,2023,2020,2022,2021,2021,2021,2021,2022,2020,2021,2020,2022,2022,2022,2021,2022,2020,2020,2020,2022,2022,2021,2020,2022,2022,2022,2021,2020,2020,2020,2021,2019,2016,2022,2022,2022,2019,2021,2020,2019,2022,2020,2018,2022,2021,2022,2021,2019,2021,2022,2020,2022,2020,2022,2021,2021,2022,2019,2022,2022,2022,2022,2022,2020,2022,2022,2021,2019,2021,2021,2019,2021,2021,2020,2020,2021,2021,2021,2022,2022,2023,2022,2023,2023,2022,2022,2021,2022,2020,2022,2022,2022,2021,2022,2022,2022,2019,2021,2022,2022,2022,2023,2022,2021,2022,2022,2022,2023,2019,2022,2019,2020,2023,2020,2021,2021,2019,2019,2019,2022,2023,2022,2022,2022,2021,2021,2021,2022,2021,2022,2020,2022,2022,2021,2022,2020,2021,2022,2021,2022,2022,2022,2023,2022,2021,2021,2022,2022,2022,2022,2022,2022,2021,2022,2021,2021,2021,2022,2021,2022,2022,2022,2022,2022,2022,2018,2022,2020,2020,2022,2021,2021,2021,2020],[2419,2030,1187,1129,1046,1028,947,855,835,776,751,549,546,475,464,441,391,386,377,372,368,358,337,329,305,302,301,287,284,253,253,223,222,220,216,208,199,194,192,189,186,181,181,176,171,167,165,165,162,159,149,140,139,139,139,135,134,133,132,131,131,128,128,125,121,120,114,113,111,110,107,105,103,101,100,96,96,93,93,91,91,91,90,90,88,88,87,85,81,79,78,74,73,72,72,72,71,71,70,70,69,69,68,65,65,65,64,61,60,58,56,56,54,53,53,52,52,51,48,47,47,47,46,46,46,45,44,43,42,42,40,40,40,39,39,38,38,38,38,38,38,37,37,36,36,36,36,36,36,36,35,35,35,35,34,34,34,34,34,34,33,33,33,32,32,32,32,31,31,30,30,30,29,29,29,29,29,29,28,28,28,28,28,28,28,28,27,27,27,26,26,26,25,25,25,25,25,24,24,24,24,24,24,24,23,23,23,23,22,22,22,22,21,21,20,20,20,20,20,20,19,19,19,19,19,19,19,19,18,18,18,18,18,18,18,18,18,18,17,17,17,17,17,17,17,17,17,17,17,16,16,16,16,16,16,16,16,16,16,16,16,16,15,15,15,15,15,15,15,15,15,15,14,14,14,14,14,14,14,14,14,14,14,14,14,14,14,13,13,13,13,13,13,13,13,13,12,12,12,12,12,12,12,12,12,12,12,12,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,10,10,10,10,10,10,10,10,10,10,10,10,10,10,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,7,7,7,7,7,7,7,7,7,7,7,7,7,7,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],["Science","Science","Nature","Cell","Lancet","Cell","Cell","Cell","Nat Biotechnol","Cancer Discov","Cell","Nature","Cell","Science","Cancer Res","Nat Neurosci","Cell Rep","Nat Genet","Cell Rep","Nature","Nat Rev Clin Oncol","Nature","Bioinformatics","Nat Med","Cancer Cell","Nat Biotechnol","Elife","Clin Cancer Res","Science","Bioinformatics","Cell","PLoS Comput Biol","PLoS Comput Biol","Cell","Cell","Genome Biol","Proc Natl Acad Sci U S A","Cell Rep","Cell","NPJ Digit Med","Radiology","Mol Cell","Nat Methods","JAMA Oncol","Nature","Cell","Nucleic Acids Res","Nucleic Acids Res","Nucleic Acids Res","Science","Cell","Ann Oncol","Cell Syst","Nat Methods","Mol Cell","Cancer Cell","Cell Rep","Nat Rev Genet","J Am Med Inform Assoc","Cancer Cell","Cancer Cell","Cell Syst","Sci Rep","Cell","Cell Syst","Sci Transl Med","Cancer Cell","Cancer Cell","Nat Genet","Clin Cancer Res","Genome Biol","Cell","Genome Res","Nat Metab","Cancer Immunol Res","Nat Commun","JAMA Oncol","Cancer Discov","Nat Genet","J Biol Chem","Nat Genet","Science","Genome Med","Cancer Cell","J Pathol","Nat Commun","Mol Cell Proteomics","Cancer Discov","Genome Biol","Nat Commun","Nature","J Pathol Inform","Proc Natl Acad Sci U S A","Cell Syst","Nat Commun","Nat Genet","Cancer Res","Nat Neurosci","Hum Mutat","J Am Med Inform Assoc","Nat Genet","Nat Genet","Nat Methods","Cancer Immunol Res","Nat Commun","Nat Rev Nephrol","Cell Rep","Nat Genet","Cancer Res","Cancer Res","Nucleic Acids Res","Nat Rev Clin Oncol","F1000Res","Crit Care Med","Lancet Digit Health","Clin Cancer Res","Nat Methods","Trends Genet","Cell","Sci Data","Cell Syst","Circ Arrhythm Electrophysiol","Phys Med","Brief Bioinform","Circ Arrhythm Electrophysiol","NPJ Genom Med","Methods Mol Biol","Genome Res","Nat Commun","Cancer Res","PLoS Comput Biol","Clin Cancer Res","Ann Oncol","Med Phys","Nat Commun","Cell Genom","Nucleic Acids Res","Cell Genom","J Clin Invest","Cell Genom","Kidney Int","Genome Med","Nat Genet","Genet Med","Cancer Discov","Mol Cancer Ther","Cancer Res","Bioinformatics","Nat Biotechnol","J Magn Reson Imaging","Cancer Cell","Nat Rev Cancer","Dev Cell","Dev Cell","Pattern Recognit","Pharmacol Rev","Cancer Res","J Am Med Inform Assoc","Nat Med","Lung Cancer","Brief Bioinform","Cell","J Immunother Cancer","Ann Surg Oncol","AMIA Annu Symp Proc","Cancer Res","Nat Methods","Oncogene","Genome Biol","J Am Med Inform Assoc","Nucleic Acids Res","Sci Rep","Nat Methods","Nucleic Acids Res","Genome Biol","JCO Clin Cancer Inform","J Biomed Inform","Cell Rep","Methods Enzymol","Elife","Mol Cell Proteomics","Front Immunol","Lancet Digit Health","Cancers (Basel)","JAMA Netw Open","Genome Med","Am J Pathol","Gigascience","Genome Med","Clin Cancer Res","Cell Syst","Mod Pathol","Elife","Clin Radiol","Nat Commun","PLoS One","Cancer Res","Nucleic Acids Res","Cancer Res","Cancer Epidemiol Biomarkers Prev","NPJ Breast Cancer","J Biomed Inform","Ann Epidemiol","J Biomed Inform","Front Neurosci","AMIA Jt Summits Transl Sci Proc","Nat Biomed Eng","Brief Bioinform","Gigascience","Bioinformatics","Lung Cancer","Nat Commun","J Med Imaging (Bellingham)","Nat Genet","Mol Oncol","Cancer Res","Cancer Cell","Neuro Oncol","Lancet Digit Health","JCO Precis Oncol","Nature","Cancer Res","Diagn Pathol","Eur J Nucl Med Mol Imaging","J Med Internet Res","Cell","Magn Reson Med","Med Phys","Genome Med","Br J Cancer","Genome Med","J Biomed Inform","Cancer Epidemiol Biomarkers Prev","Lab Invest","Neuroimage","J Magn Reson Imaging","Epigenetics","Med Image Anal","Nat Commun","Sci Data","PLoS Comput Biol","Science","Ann Epidemiol","BMC Genomics","Cancers (Basel)","Circ Arrhythm Electrophysiol","Med Phys","NPJ Genom Med","J Biomed Opt","Genet Med","PLoS Biol","Biochim Biophys Acta Gene Regul Mech","Proc IEEE Comput Soc Conf Comput Vis Pattern Recognit","Gigascience","Cancer Lett","JCO Clin Cancer Inform","Science","Methods Mol Biol","Clin Cancer Res","Cancer Cell","Front Cell Dev Biol","Ann Biomed Eng","JCO Clin Cancer Inform","Phys Med Biol","Nat Commun","AMIA Jt Summits Transl Sci Proc","JCO Clin Cancer Inform","Sci Rep","JCO Clin Cancer Inform","Neuro Oncol","Eur Radiol","Genome Biol","Mol Cancer Ther","Mol Cancer Res","Brief Bioinform","Cell Syst","Elife","Kidney Int","AMIA Annu Symp Proc","iScience","Genome Biol","J Am Med Inform Assoc","Med Phys","Precis Clin Med","J Am Med Inform Assoc","Sci Rep","Math Biosci Eng","JCO Clin Cancer Inform","J Digit Imaging","Bioinformatics","Mov Disord","Aliment Pharmacol Ther","Sci Data","Med Image Anal","Lancet Digit Health","Proc SPIE Int Soc Opt Eng","Cancer Res","Gynecol Oncol","Sci Data","Nucleic Acids Res","JCO Clin Cancer Inform","JCO Clin Cancer Inform","AMIA Annu Symp Proc","BMC Med Inform Decis Mak","EBioMedicine","Circ Arrhythm Electrophysiol","Med Image Anal","JCO Clin Cancer Inform","Cell Rep","Gigascience","JCO Clin Cancer Inform","Cancer Cell","JCO Clin Cancer Inform","Cancers (Basel)","BMC Med Inform Decis Mak","AMIA Jt Summits Transl Sci Proc","BMC Genomics","Front Pharmacol","J Am Med Inform Assoc","Genome Biol","Cell Genom","Cancer Res","Nat Rev Clin Oncol","Cancers (Basel)","J Clin Med","Proc SPIE Int Soc Opt Eng","IEEE Trans Vis Comput Graph","Neuro Oncol","Nat Commun","AMIA Jt Summits Transl Sci Proc","J Am Med Inform Assoc","BMC Med Inform Decis Mak","Cell Syst","J Natl Cancer Inst","mSystems","Cancers (Basel)","J Clin Pathol","J Pathol","Br J Ophthalmol","Genome Biol","Cold Spring Harb Mol Case Stud","Nat Commun","Genome Res","NPJ Breast Cancer","J Am Med Inform Assoc","Clin Pharmacol Ther","Bioinformatics","JCO Clin Cancer Inform","Am J Hum Genet","Eur Radiol","NPJ Breast Cancer","Eur J Cancer","NPJ Breast Cancer","IEEE Trans Biomed Eng","Sci Adv","J Biomech Eng","Genome Med","Clin Cancer Res","Cell Genom","NPJ Precis Oncol","Bioinformatics","Nucleic Acids Res","BMC Bioinformatics","Bioinformatics","Curr Opin Genet Dev","Science","Phys Med Biol","JCO Clin Cancer Inform","Sci Rep","JAMIA Open","Spine J","BMC Med Inform Decis Mak","AMIA Annu Symp Proc","F1000Res","Cell Syst","Cell Genom","PLoS Comput Biol","Sci Rep","Sci Rep","Clin Radiol","Neuro Oncol","Brainlesion","BMC Med Imaging","Breast","Genome Res","Cancers (Basel)","Science","Med Image Comput Comput Assist Interv","Neural Netw","Nucleic Acids Res","JCO Clin Cancer Inform","J Am Med Inform Assoc","Bioinformatics","PLoS One","J Immunother Cancer","J Clin Invest","Cold Spring Harb Perspect Med","Nat Commun","Clin Cancer Res","BMC Bioinformatics","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","JCO Clin Cancer Inform","Cell Rep Med","Cancer Res","Bioinformatics","Mol Cancer Ther","JCO Clin Cancer Inform","JCO Clin Cancer Inform","J Biomed Inform","Bioinformatics","PLoS Comput Biol","JCO Clin Cancer Inform","Semin Cancer Biol","Sci Rep","JCO Clin Cancer Inform","Eur Urol Focus","Breast Cancer Res","J Magn Reson Imaging","Nat Commun","NPJ Genom Med","JCO Clin Cancer Inform","Nat Biotechnol","Nat Cancer","Bioengineering (Basel)","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","JCO Clin Cancer Inform","NPJ Precis Oncol","Bioinformatics","Genome Res","Phys Med Biol","Radiat Phys Chem Oxf Engl 1993","PLoS One","Cancers (Basel)","J Digit Imaging","Bioinformatics","J Clin Transl Sci","Mol Cell Proteomics","Clin Genitourin Cancer","BMC Health Serv Res","Clin Exp Gastroenterol","Nat Biotechnol","Anal Chem","Bioinformatics","PLoS One","IEEE J Transl Eng Health Med","Comput Methods Programs Biomed","Comput Med Imaging Graph","Cells","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","AMIA Annu Symp Proc","IEEE Trans Image Process","Vis Comput Ind Biomed Art","Cancer Res","Hum Mutat","Anticancer Res","Mol Cancer Ther","Oncotarget","Genome Med","Phys Med Biol","Brachytherapy","Sci Data","F1000Res","J Biomed Inform","Stud Health Technol Inform","Curr Protoc","Viruses","Biostatistics","Bioinformatics","JCO Clin Cancer Inform","Patterns (N Y)","Invest Ophthalmol Vis Sci","J Biomed Inform","Comput Med Imaging Graph","J Immunother Cancer","EBioMedicine","J Clin Pathol","JCO Clin Cancer Inform","Open Forum Infect Dis","JAMA Netw Open","J Med Imaging (Bellingham)","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","JCO Clin Cancer Inform","JCO Precis Oncol","Nucleic Acids Res","Cancer Lett","Phys Med","Proc IEEE Int Conf Clust Comput","Med Phys","Leukemia","J Am Med Inform Assoc","JCO Clin Cancer Inform","Proc ACM SIGSPATIAL Int Conf Adv Inf","Sci Data","J Pathol","Nat Commun","J Transl Med","JCO Clin Cancer Inform","IEEE Access","J Math Imaging Vis","BMC Res Notes","JCO Clin Cancer Inform","IEEE Trans Med Imaging","Proc SPIE Int Soc Opt Eng","Med Biol Eng Comput","Proc SPIE Int Soc Opt Eng","JCO Clin Cancer Inform","Nucleic Acids Res","Nat Methods","Nat Commun","Bioinformatics","PLoS Comput Biol","Front Genet","Genetics","Bioinformatics","iScience","IEEE Access","Bioinformatics","Genome Med","Harv Data Sci Rev","Cancer Cell","J Med Internet Res","JCO Clin Cancer Inform","Indian J Hematol Blood Transfus","Sci Rep","JMIR Med Inform","Mol Cell Oncol","Cell Syst","bioRxiv","F1000Res","Nat Commun","Nat Protoc","Cancer Res","Bioinformatics","Proc IEEE Int Symp Biomed Imaging","Med Image Anal","IEEE J Biomed Health Inform","Phys Med Biol","AJNR Am J Neuroradiol","Diabet Foot Ulcers Grand Chall (2021)","Med Image Anal","Front Neurosci","Circulation","Cancers (Basel)","IEEE J Biomed Health Inform","Med Phys","JMIR Med Inform","JAMA Netw Open","Sci Rep","JCO Clin Cancer Inform","Acad Pathol","J Digit Imaging","Mol Cell","SoftwareX","Proc SPIE Int Soc Opt Eng","J Biomed Opt","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","Nat Cancer","IEEE Trans Vis Comput Graph","Bioinformatics","PLoS Comput Biol","Nat Commun","NPJ Syst Biol Appl","Science","Bioinformatics","BMC Bioinformatics","Phys Med","PLoS One","IEEE Access","Comput Biol Med","Leukemia","Cancer Cell","Expert Rev Proteomics","JAMA Netw Open","IEEE Int Conf Healthc Inform","AMIA Annu Symp Proc","AMIA Annu Symp Proc","BMC Med Inform Decis Mak","Bioinformatics","Health Informatics J","F1000Res","Bioinformatics","IEEE Int Conf Mob Data Manag","Comput Biol Med","Transl Vis Sci Technol","J Med Imaging (Bellingham)","Med Phys","Circ Arrhythm Electrophysiol","Tomography","Proc IEEE Comput Soc Conf Comput Vis Pattern Recognit","Mach Learn Med Imaging","Phys Med Biol","Brainlesion","Brainlesion","Front Med (Lausanne)","J Med Artif Intell","J Pathol","IEEE Trans Med Imaging","Mod Pathol","J Comput Biol","Clin Cancer Res","JCO Precis Oncol","Diagnostics (Basel)","Cancer Immunol Res","Inverse Probl","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","ArXiv","Cancer Res Commun","Cancer Genet","Sci Rep","Pac Symp Biocomput","IEEE Trans Vis Comput Graph","Bioinformatics","PLoS Comput Biol","Bioinform Adv","Cancer Causes Control","Genes (Basel)","Nat Methods","BMC Bioinformatics","Cancer Causes Control","J Transl Med","J Proteome Res","Phys Med Biol","PLoS One","Comput Netw","Phys Med Biol","J Digit Imaging","Genome Med","Computer (Long Beach Calif)","Genomics Proteomics Bioinformatics","Sci Adv","J Psychiatr Res","Tomography","J Med Internet Res","J Am Med Inform Assoc","JMIR Med Inform","PEC Innov","Nucleic Acids Res","IEEE Int Conf Healthc Inform","iScience","Cell Syst","bioRxiv","Nat Protoc","bioRxiv","Bioinformatics","J Med Internet Res","F1000Res","Am J Epidemiol","F1000Res","Bioinformatics","Pac Symp Biocomput","IEEE Trans Technol Soc","Cell Genom","Comput Biol Med","Proc Int Conf Data Eng","Proc IEEE Int Symp Biomed Imaging","Proc IEEE Int Symp Biomed Imaging","Phys Med Biol","Med Image Comput Comput Assist Interv","Adv Database Technol","Front Big Data","ACM Trans Spat Algorithms Syst","Distrib Collab Fed Learn Afford AI Healthc Resour Div Glob Health (2022)","ArXiv","J Digit Imaging","Mach Learn Med Imaging","Front Neurosci","Sci Data","Appl Sci (Basel)","Brainlesion","Clin Cancer Res","Cancer Med","Med Image Anal","Comput Biol Med","Cancer Res","J Alzheimers Dis","Oral Oncol","Cancer","IEEE J Biomed Health Inform","Front Ophthalmol (Lausanne)","Phys Med Biol","J Med Internet Res","Proc IAPR Int Conf Pattern Recogn","Comput Biol Med","IEEE EMBS Int Conf Biomed Health Inform","Biometrics","Biometrics","Biostatistics","Biometrics","AIP Conf Proc","Cancers (Basel)","Commun Stat Simul Comput","J Math Biol","J Vasc Interv Radiol","Med Image Comput Comput Assist Interv","Amino Acids","AMIA Annu Symp Proc","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","J Biomed Opt","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","Proc SPIE Int Soc Opt Eng","F1000Res"],["Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma.","Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq.","Next-generation characterization of the Cancer Cell Line Encyclopedia.","Comprehensive Molecular Characterization of Muscle-Invasive Bladder Cancer.","Clinical impact of COVID-19 on patients with cancer (CCC19): a cohort study.","Single-Cell Transcriptomic Analysis of Primary and Metastatic Tumor Ecosystems in Head and Neck Cancer.","Comprehensive Characterization of Cancer Driver Genes and Mutations.","Cohesin Loss Eliminates All Loop Domains.","Nanopore sequencing and assembly of a human genome with ultra-long reads.","AACR Project GENIE: Powering Precision Medicine through an International Consortium.","An Integrative Model of Cellular States, Plasticity, and Genetics for Glioblastoma.","Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma.","A Cancer Cell Program Promotes T Cell Exclusion and Resistance to Checkpoint Blockade.","Decoupling genetics, lineages, and microenvironment in IDH-mutant gliomas by single-cell RNA-seq.","Variant Review with the Integrative Genomics Viewer.","Whole genome sequencing resource identifies 18 new candidate genes for autism spectrum disorder.","A Tissue-Mapped Axolotl De Novo Transcriptome Enables Identification of Limb Regeneration Factors.","Long noncoding RNA MALAT1 suppresses breast cancer metastasis.","Spatial Organization and Molecular Correlation of Tumor-Infiltrating Lymphocytes Using Deep Learning on Pathology Images.","Scaling single-cell genomics from phenomenology to mechanism.","Artificial intelligence in digital pathology - new tools for diagnosis and precision oncology.","The neuropeptide NMU amplifies ILC2-driven allergic lung inflammation.","Preprocessing, normalization and integration of the Illumina HumanMethylationEPIC array with minfi.","Metagenomic analysis of colorectal cancer datasets identifies cross-cohort microbial diagnostic signatures and a link with choline degradation.","A Pan-Cancer Proteogenomic Atlas of PI3K/AKT/mTOR Pathway Alterations.","Revealing the vectors of cellular identity with single-cell genomics.","Many lncRNAs, 5'UTRs, and pseudogenes are translated and some are likely to express functional proteins.","Prospective Genotyping of Hepatocellular Carcinoma: Clinical Implications of Next-Generation Sequencing for Matching Patients to Targeted and Immune Therapies.","Developmental and oncogenic programs in H3K27M gliomas dissected by single-cell RNA-seq.","Mosdepth: quick coverage calculation for genomes and exomes.","Comprehensive Characterization of Cancer Driver Genes and Mutations.","Multivariable association discovery in population-scale meta-omics studies.","Machine Learning Meta-analysis of Large Metagenomic Datasets: Tools and Biological Insights.","A Distinct Gene Module for Dysfunction Uncoupled from Activation in Tumor-Infiltrating T Cells.","Anatomically and Functionally Distinct Lung Mesenchymal Populations Marked by Lgr5 and Lgr6.","Accuracy assessment of fusion transcript detection via read-mapping and de novo fusion transcript assembly-based methods.","Evaluating the evaluation of cancer driver genes.","Comprehensive Molecular Characterization of the Hippo Signaling Pathway in Cancer.","MHC-I Genotype Restricts the Oncogenic Mutational Landscape.","The future of digital health with federated learning.","Preparing Medical Imaging Data for Machine Learning.","Methylome-wide Analysis of Chronic HIV Infection Reveals Five-Year Increase in Biological Age and Epigenetic Targeting of HLA.","Orchestrating single-cell analysis with Bioconductor.","Racial/Ethnic Disparities in Genomic Sequencing.","The expanding landscape of 'oncohistone' mutations in human cancers.","The Human Tumor Atlas Network: Charting Tumor Transitions across Space and Time at Single-Cell Resolution.","Integration of the Drug-Gene Interaction Database (DGIdb 4.0) with open crowdsource efforts.","Integration of the Drug-Gene Interaction Database (DGIdb 4.0) with open crowdsource efforts.","The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2020 update.","Minimal functional driver gene heterogeneity among untreated metastases.","A Pan-Cancer Analysis of Enhancer Expression in Nearly 9000 Patient Samples.","Association of clinical factors and recent anticancer therapy with COVID-19 severity among patients with cancer: a report from the COVID-19 and Cancer Consortium.","NDEx, the Network Data Exchange.","Accessible, curated metagenomic data through ExperimentHub.","A Regression-Based Analysis of Ribosome-Profiling Data Reveals a Conserved Complexity to Mammalian Translation.","Characterization of Human Cancer Cell Lines by Reverse-phase Protein Arrays.","Molecular Characterization and Clinical Relevance of Metabolic Expression Subtypes in Human Cancers.","Settling the score: variant prioritization and Mendelian disease.","CLAMP - a toolkit for efficiently building customized clinical natural language processing pipelines.","Systematic Functional Annotation of Somatic Mutations in Cancer.","Systematic Functional Annotation of Somatic Mutations in Cancer.","Juicebox.js Provides a Cloud-Based Visualization System for Hi-C Data.","Federated learning in medicine: facilitating multi-institutional collaborations without sharing patient data.","How Machine Learning Will Transform Biomedicine.","Systematic Evaluation of Molecular Networks for Discovery of Disease Genes.","Evaluation of liquid from the Papanicolaou test and other liquid biopsies for the detection of endometrial and ovarian cancers.","A-to-I RNA Editing Contributes to Proteomic Diversity in Cancer.","A Platform of Synthetic Lethal Gene Interaction Networks Reveals that the GNAQ Uveal Melanoma Oncogene Controls the Hippo Pathway through FAK.","A map of constrained coding regions in the human genome.","Neoadjuvant and Adjuvant Pembrolizumab in Resectable Locally Advanced, Human Papillomavirus-Unrelated Head and Neck Cancer: A Multicenter, Phase II Trial.","Evolutionary analysis across mammals reveals distinct classes of long non-coding RNAs.","Inhibitory CD161 receptor identified in glioma-infiltrating T cells by single-cell analysis.","Systematic characterization of A-to-I RNA editing hotspots in microRNAs across human cancers.","Characterization of Hypoxia-associated Molecular Features to Aid Hypoxia-Targeted Therapy.","Changes in CT Radiomic Features Associated with Lymphocyte Distribution Predict Overall Survival and Response to Immunotherapy in Non-Small Cell Lung Cancer.","Breast cancer quantitative proteome and proteogenomic landscape.","Association of Convalescent Plasma Therapy With Survival in Patients With Hematologic Cancers and COVID-19.","Utilization of COVID-19 Treatments and Clinical Outcomes among Patients with Cancer: A COVID-19 and Cancer Consortium (CCC19) Cohort Study.","Identification of cancer driver genes based on nucleotide context.","Illuminating the Onco-GPCRome: Novel G protein-coupled receptor-driven oncocrine networks and targets for cancer immunotherapy.","Multi-tiered genomic analysis of head and neck cancer ties TP53 mutation to 3p loss.","Regenerative potential of prostate luminal cells revealed by single-cell analysis.","Best practices for bioinformatic characterization of neoantigens for clinical utility.","Proteomics, Post-translational Modifications, and Integrative Analyses Reveal Molecular Heterogeneity within Medulloblastoma Subgroups.","PanCancer insights from The Cancer Genome Atlas: the pathologist's perspective.","An integrated bioinformatics platform for investigating the human E3 ubiquitin ligase-substrate interaction network.","A Curated Resource for Phosphosite-specific Signature Analysis.","Interaction Landscape of Inherited Polymorphisms with Somatic Events in Cancer.","Lisa: inferring transcriptional regulators through integrative modeling of public chromatin accessibility and ChIP-seq data.","Combating subclonal evolution of resistant cancer phenotypes.","Transparency and reproducibility in artificial intelligence.","Twenty Years of Digital Pathology: An Overview of the Road Travelled, What is on the Horizon, and the Emergence of Vendor-Neutral Archives.","Overlooked roles of DNA damage and maternal age in generating human germline mutations.","Before and After: Comparison of Legacy and Harmonized TCGA Genomic Data Commons' Data.","RNA editing derived epitopes function as cancer antigens to elicit immune responses.","Landscape of B cell immunity and related immune evasion in human cancers.","PDX-MI: Minimal Information for Patient-Derived Tumor Xenograft Models.","Neuronal brain-region-specific DNA methylation and chromatin accessibility are associated with neuropsychiatric trait heritability.","Assessing the Pathogenicity of Insertion and Deletion Variants with the Variant Effect Scoring Tool (VEST-Indel).","Enhancing clinical concept extraction with contextual embeddings.","A harmonized meta-knowledgebase of clinical interpretations of somatic genomic variants in cancer.","A global transcriptional network connecting noncoding mutations to changes in tumor gene expression.","GIGGLE: a search engine for large-scale integrated genome analysis.","pVACtools: A Computational Toolkit to Identify and Visualize Cancer Neoantigens.","Comparative genomic analysis of esophageal squamous cell carcinoma between Asian and Caucasian patient populations.","Digital pathology and computational image analysis in nephropathology.","Integrated Genomic Analysis of the Ubiquitin Pathway across Cancer Types.","Dynamic incorporation of multiple in silico functional annotations empowers rare variant association analysis of large whole-genome sequencing studies at scale.","Explore, Visualize, and Analyze Functional Cancer Proteomic Data Using the Cancer Proteome Atlas.","Exome-Scale Discovery of Hotspot Mutation Regions in Human Cancer Using 3D Protein Structure.","PDX Finder: A portal for patient-derived tumor xenograft model discovery.","Predicting cancer outcomes with radiomics and artificial intelligence in radiology.","RCy3: Network biology using Cytoscape from within R.","Early Prediction of Sepsis From Clinical Data: The PhysioNet/Computing in Cardiology Challenge 2019.","CT derived radiomic score for predicting the added benefit of adjuvant chemotherapy following surgery in stage I, II resectable non-small cell lung cancer: a retrospective multicohort study for outcome prediction.","Consensus on Molecular Subtypes of High-Grade Serous Ovarian Carcinoma.","Comprehensive comparative analysis of 5'-end RNA-sequencing methods.","Advances in Decoding Axolotl Limb Regeneration.","Bacteria-to-Human Protein Networks Reveal Origins of Endogenous DNA Damage.","The public cancer radiology imaging collections of The Cancer Imaging Archive.","CHASMplus Reveals the Scope of Somatic Missense Mutations Driving Human Cancers.","Artificial Intelligence and Machine Learning in Arrhythmias and Cardiac Electrophysiology.","The TOPAS tool for particle simulation, a Monte Carlo simulation tool for physics, biology and clinical research.","Toward a gold standard for benchmarking gene set enrichment analysis.","Machine Learning Prediction of Response to Cardiac Resynchronization Therapy: Improvement Versus Current Guidelines.","Whole-genome analysis for effective clinical diagnosis and gene discovery in early infantile epileptic encephalopathy.","NDEx: A Community Resource for Sharing and Publishing of Biological Networks.","Comprehensive, integrated, and phased whole-genome analysis of the primary ENCODE cell line K562.","Chromatin dysregulation and DNA methylation at transcription start sites associated with transcriptional repression in cancers.","Software for the Integration of Multiomics Experiments in Bioconductor.","Network propagation in the cytoscape cyberinfrastructure.","Radiogenomic-Based Survival Risk Stratification of Tumor Habitat on Gd-T1w MRI Is Associated with Biological Processes in Glioblastoma.","COVID-19 vaccination and breakthrough infections in patients with cancer.","Report on G4-Med, a Geant4 benchmarking system for medical physics applications developed by the Geant4 Medical Simulation Benchmarking Group.","A spatially resolved brain region- and cell type-specific isoform atlas of the postnatal mouse brain.","GA4GH: International policies and standards for data sharing across genomic research and healthcare.","Cistrome-GO: a web server for functional enrichment analysis of transcription factor ChIP-seq peaks.","GA4GH: International policies and standards for data sharing across genomic research and healthcare.","A-to-I-edited miRNA-379-5p inhibits cancer cell proliferation through CD97-induced apoptosis.","GA4GH: International policies and standards for data sharing across genomic research and healthcare.","Development and evaluation of deep learning-based segmentation of histologic structures in the kidney cortex with multiple histologic stains.","Mapping genetic variations to three-dimensional protein structures to enhance variant interpretation: a proposed framework.","Epigenetic encoding, heritability and plasticity of glioma transcriptional cell states.","Standard operating procedure for somatic variant refinement of sequencing data with paired tumor and normal samples.","Leukocyte Heterogeneity in Pancreatic Ductal Adenocarcinoma: Phenotypic and Spatial Features Associated with Clinical Outcome.","Disruption of &lt;i&gt;NSD1&lt;/i&gt; in Head and Neck Cancer Promotes Favorable Chemotherapeutic Responses Linked to Hypomethylation.","NDEx 2.0: A Clearinghouse for Research on Cancer Pathways.","Cross-study validation for the assessment of prediction algorithms.","Transcriptome-scale RNase-footprinting of RNA-protein complexes.","Radiomic Features of Primary Rectal Cancers on Baseline T&lt;sub&gt;2&lt;/sub&gt; -Weighted MRI Are Associated With Pathologic Complete Response to Neoadjuvant Chemoradiation: A Multisite Study.","A High-Resolution Map of Human Enhancer RNA Loci Characterizes Super-enhancer Activities in Cancer.","A census of pathway maps in cancer systems biology.","The Pediatric Cell Atlas: Defining the Growth Phase of Human Development at Single-Cell Resolution.","The Pediatric Cell Atlas: Defining the Growth Phase of Human Development at Single-Cell Resolution.","Sparse Autoencoder for Unsupervised Nucleus Detection and Representation in Histopathology Images.","Personal Mutanomes Meet Modern Oncology Drug Discovery and Precision Health.","CRAVAT 4: Cancer-Related Analysis of Variants Toolkit.","A study of deep learning approaches for medication and adverse drug event extraction from clinical text.","Opposing immune and genetic mechanisms shape oncogenic programs in synovial sarcoma.","Predicting pathologic response to neoadjuvant chemoradiation in resectable stage III non-small cell lung cancer patients using computed tomography radiomic features.","Public data and open source tools for multi-assay genomic investigation of disease.","Glioma progression is shaped by genetic evolution and microenvironment interactions.","Novel, non-invasive imaging approach to identify patients with advanced non-small cell lung cancer at risk of hyperprogressive disease with immune checkpoint blockade.","Obesity and angiolymphatic invasion in primary breast cancer.","Clinical Named Entity Recognition Using Deep Learning Models.","Use of Natural Language Processing to Extract Clinical Cancer Phenotypes from Electronic Medical Records.","TRUST4: immune repertoire reconstruction from bulk and single-cell RNA-seq data.","HPV E2, E4, E5 drive alternative carcinogenic pathways in HPV positive cancers.","Assessment of statistical methods from single cell, bulk RNA-seq, and metagenomics applied to microbiome data.","Use of electronic health records to support a public health response to the COVID-19 pandemic in the United States: a perspective from 15 academic medical centers.","APAatlas: decoding alternative polyadenylation across human tissues.","Head and Neck Cancer Detection in Digitized Whole-Slide Histology Using Convolutional Neural Networks.","MCMICRO: a scalable, modular image-processing pipeline for multiplexed tissue imaging.","Haplotype-resolved and integrated genome analysis of the cancer cell line HepG2.","Comprehensive assessment of computational algorithms in predicting cancer driver mutations.","Integrated Informatics Analysis of Cancer-Related Variants.","Assessing the practice of biomedical ontology evaluation: Gaps and opportunities.","Spatial Genome Re-organization between Fetal and Adult Hematopoietic Stem Cells.","High-dimensional multiplexed immunohistochemical characterization of immune contexture in human cancers.","STRIPAK directs PP2A activity toward MAP4K4 to promote oncogenic transformation of human cells.","TCPA v3.0: An Integrative Platform to Explore the Pan-Cancer Analysis of Functional Proteomic Data.","A Network-Based Analysis Reveals the Mechanism Underlying Vitamin D in Suppressing Cytokine Storm and Virus in SARS-CoV-2 Infection.","A prognostic model for overall survival of patients with early-stage non-small cell lung cancer: a multicentre, retrospective study.","Combination of Peri-Tumoral and Intra-Tumoral Radiomic Features on Bi-Parametric MRI Accurately Stratifies Prostate Cancer Risk: A Multi-Site Study.","Assessment of Machine Learning of Breast Pathology Structures for Automated Differentiation of Breast Cancer and High-Risk Proliferative Lesions.","Immune receptor repertoires in pediatric and adult acute myeloid leukemia.","Utilizing Automated Breast Cancer Detection to Identify Spatial Distributions of Tumor-Infiltrating Lymphocytes in Invasive Breast Cancer.","Duphold: scalable, depth-based annotation and curation of high-confidence structural variant calls.","Recommendations for the collection and use of multiplexed functional data for clinical variant interpretation.","Cannabinoids Promote Progression of HPV-Positive Head and Neck Squamous Cell Carcinoma via p38 MAPK Activation.","Proteogenomic Network Analysis of Context-Specific KRAS Signaling in Mouse-to-Human Cross-Species Translation.","Quality control stress test for deep learning-based diagnostic model in digital pathology.","Wikidata as a knowledge graph for the life sciences.","Open access image repositories: high-quality data to enable machine learning research.","Challenges in identifying cancer genes by analysis of exome sequencing data.","ERCC1 and TS Expression as Prognostic and Predictive Biomarkers in Metastatic Colon Cancer.","Multiomic Analysis of Subtype Evolution and Heterogeneity in High-Grade Serous Ovarian Carcinoma.","The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2022 update.","Metformin Inhibits Progression of Head and Neck Squamous Cell Carcinoma by Acting Directly on Carcinoma-Initiating Cells.","Prevalence of Inherited Mutations in Breast Cancer Predisposition Genes among Women in Uganda and Cameroon.","A functional genomic screen in vivo identifies CEACAM5 as a clinically relevant driver of breast cancer metastasis.","HemOnc: A new standard vocabulary for chemotherapy regimen representation in the OMOP common data model.","Sociodemographic variation in the oral microbiome.","HemOnc: A new standard vocabulary for chemotherapy regimen representation in the OMOP common data model.","Segmentation and Classification in Digital Pathology for Glioma Research: Challenges and Deep Learning Approaches.","BERT-based Ranking for Biomedical Entity Normalization.","Harnessing non-destructive 3D pathology.","A review of digital cytometry methods: estimating the relative abundance of cell types in a bulk of cells.","Indexcov: fast coverage quality control for whole-genome sequencing.","Classifying tumors by supervised network propagation.","Stable and discriminating radiomic predictor of recurrence in early stage non-small cell lung cancer: Multi-site study.","Optimal control nodes in disease-perturbed networks as targets for combination therapy.","Multisite evaluation of radiomic feature reproducibility and discriminability for identifying peripheral zone prostate tumors on MRI.","Integrated molecular drivers coordinate biological and clinical states in melanoma.","Role of ultraviolet mutational signature versus tumor mutation burden in predicting response to immunotherapy.","A Containerized Software System for Generation, Management, and Exploration of Features from Whole Slide Tissue Images.","Large-Scale Characterization of Drug Responses of Clinically Relevant Proteins in Cancer Cell Lines.","AI-based prognostic imaging biomarkers for precision neuro-oncology: the ReSPOND consortium.","CT derived radiomic score for predicting the added benefit of adjuvant chemotherapy following surgery in Stage I, II resectable Non-Small Cell Lung Cancer: a retrospective multi-cohort study for outcome prediction.","Next-Generation Sequencing and the Clinical Oncology Workflow: Data Challenges, Proposed Solutions, and a Call to Action.","BRCA1 and RNAi factors promote repair mediated by small RNAs and PALB2-RAD52.","Functional Precision Medicine Identifies New Therapeutic Candidates for Medulloblastoma.","Deep learning-based image analysis methods for brightfield-acquired multiplex immunohistochemistry images.","Co-clinical FDG-PET radiomic signature in predicting response to neoadjuvant chemotherapy in triple-negative breast cancer.","A Deep Learning Method to Automatically Identify Reports of Scientifically Rigorous Clinical Research from the Biomedical Literature: Comparative Analytic Study.","A mouse-specific retrotransposon drives a conserved Cdk2ap1 isoform essential for development.","Repeatability of radiomics and machine learning for DWI: Short-term repeatability study of 112 patients with prostate cancer.","Graph-convolutional-network-based interactive prostate segmentation in MR images.","Text-mining clinically relevant cancer biomarkers for curation into the CIViC database.","Nongenic cancer-risk SNPs affect oncogenes, tumour-suppressor genes, and immune function.","Somalier: rapid relatedness estimation for cancer and germline studies using efficient genome sketches.","A hybrid approach to automatic de-identification of psychiatric notes.","The Impact of Stroma Admixture on Molecular Subtypes and Prognostic Gene Signatures in Serous Ovarian Cancer.","Deep-learning-based accurate hepatic steatosis quantification for histological assessment of liver biopsies.","Brain extraction on MRI scans in presence of diffuse glioma: Multi-institutional performance evaluation of deep learning methods and robust modality-agnostic training.","Noncontrast Radiomics Approach for Predicting Grades of Nonfunctional Pancreatic Neuroendocrine Tumors.","Cell-free DNA Methylation and Transcriptomic Signature Prediction of Pregnancies with Adverse Outcomes.","Graph temporal ensembling based semi-supervised convolutional neural network with noisy labels for histopathology image analysis.","Determinants of transcription factor regulatory range.","Consensus transcriptional regulatory networks of coronavirus-infected human cells.","Multiscale community detection in Cytoscape.","A protein network map of head and neck cancer reveals PIK3CA mutant drug sensitivity.","Tobacco exposure associated with oral microbiota oxygen utilization in the New York City Health and Nutrition Examination Study.","Linear models enable powerful differential activity analysis in massively parallel reporter assays.","Radiomic Texture and Shape Descriptors of the Rectal Environment on Post-Chemoradiation T2-Weighted MRI are Associated with Pathologic Tumor Stage Regression in Rectal Cancers: A Retrospective, Multi-Institution Study.","Machine Learning of 12-Lead QRS Waveforms to Identify Cardiac Resynchronization Therapy Patients With Differential Outcomes.","Technical Note: MRQy - An open-source tool for quality control of MR imaging data.","FHIR Genomics: enabling standardization for precision medicine use cases.","Comprehensive review of surgical microscopes: technology development and medical applications.","Standards for the classification of pathogenicity of somatic variants in cancer (oncogenicity): Joint recommendations of Clinical Genome Resource (ClinGen), Cancer Genomics Consortium (CGC), and Variant Interpretation for Cancer Consortium (VICC).","Enabling precision medicine via standard communication of HTS provenance, analysis, and results.","Gaussian and Mixed Graphical Models as (multi-)omics data analysis tools.","Robust Histopathology Image Analysis: to Label or to Synthesize?","SV-plaudit: A cloud-based framework for manually curating thousands of structural variants.","SRC-3 inhibition blocks tumor growth of pancreatic ductal adenocarcinoma.","Standards-Based Clinical Decision Support Platform to Manage Patients Who Meet Guideline-Based Criteria for Genetic Evaluation of Familial Cancer.","A protein interaction landscape of breast cancer.","Genomic Annotation Resources in R/Bioconductor.","Computationally Derived Image Signature of Stromal Morphology Is Prognostic of Prostate Cancer Recurrence Following Prostatectomy in African American Patients.","A Systematic Framework to Rapidly Obtain Data on Patients with Cancer and COVID-19: CCC19 Governance, Protocol, and Quality Assurance.","A Review of Precision Oncology Knowledgebases for Determining the Clinical Actionability of Genetic Variants.","Personalized Dosimetry for Liver Cancer Y-90 Radioembolization Using Computational Fluid Dynamics and Monte Carlo Simulation.","Multiomic Integration of Public Oncology Databases in Bioconductor.","The microdosimetric extension in TOPAS: development and comparison with published data.","Biosynthetic energy cost for amino acids decreases in cancer evolution.","Determining Onset for Familial Breast and Colorectal Cancer from Family History Comments in the Electronic Health Record.","Discovery of Noncancer Drug Effects on Survival in Electronic Health Records of Patients With Cancer: A New Paradigm for Drug Repurposing.","MetaGxData: Clinically Annotated Breast, Ovarian and Pancreatic Cancer Datasets and their Use in Generating a Multi-Cancer Gene Signature.","Multiomic Integration of Public Oncology Databases in Bioconductor.","Sexually dimorphic radiogenomic models identify distinct imaging and biological pathways that are prognostic of overall survival in glioblastoma.","T1 and T2 MR fingerprinting measurements of prostate cancer and prostatitis correlate with deep learning-derived estimates of epithelium, lumen, and stromal composition on corresponding whole mount histopathology.","scATAC-pro: a comprehensive workbench for single-cell chromatin accessibility sequencing data.","Macrophage Syk-PI3Kγ Inhibits Antitumor Immunity: SRX3207, a Novel Dual Syk-PI3K Inhibitory Chemotype Relieves Tumor Immunosuppression.","Mesenchymal and MAPK Expression Signatures Associate with Telomerase Promoter Mutations in Multiple Cancers.","Distinct telomere length and molecular signatures in seminoma and non-seminoma of testicular germ cell tumor.","Fast-Evolving Human-Specific Neural Enhancers Are Associated with Aging-Related Diseases.","Differential expression of MAGEA6 toggles autophagy to promote pancreatic cancer progression.","Podocyte stress and detachment measured in urine are related to mean arterial pressure in healthy humans.","Combine Factual Medical Knowledge and Distributed Word Representation to Improve Clinical Named Entity Recognition.","A Fast and Flexible Framework for Network-Assisted Genomic Association.","HiDeF: identifying persistent structures in multiscale 'omics data.","Accounting for data variability in multi-institutional distributed deep learning for medical imaging.","Reproducibility analysis of multi-institutional paired expert annotations and radiomic features of the Ivy Glioblastoma Atlas Project (Ivy GAP) dataset.","The size of cell-free mitochondrial DNA in blood is inversely correlated with tumor burden in cancer patients.","sureLDA: A multidisease automated phenotyping method for the electronic health record.","Immune classification of clear cell renal cell carcinoma.","Immune classification of osteosarcoma.","Collaborative, Multidisciplinary Evaluation of Cancer Variants Through Virtual Molecular Tumor Boards Informs Local Clinical Practices.","Reengineering Workflow for Curation of DICOM Datasets.","hts-nim: scripting high-performance genomic analyses.","Resting-State Functional Connectivity Predicts STN DBS Clinical Response.","Nonselective beta-blockers are associated with a lower risk of hepatocellular carcinoma among cirrhotic patients in the United States.","CT-ORG, a new dataset for multiple organ segmentation in computed tomography.","Computerized spermatogenesis staging (CSS) of mouse testis sections via quantitative histomorphological analysis.","An integrated nomogram combining deep learning, Prostate Imaging-Reporting and Data System (PI-RADS) scoring, and clinical variables for identification of clinically significant prostate cancer on biparametric MRI: a retrospective multicentre study.","Hyperspectral imaging and deep learning for the detection of breast cancer cells in digitized histological images.","Leveraging Systematic Functional Analysis to Benchmark an &lt;i&gt;In Silico&lt;/i&gt; Framework Distinguishes Driver from Passenger MEK Mutants in Cancer.","Combining copy number, methylation markers, and mutations as a panel for endometrial cancer detection via intravaginal tampon collection.","Dataset of segmented nuclei in hematoxylin and eosin stained histopathology images of ten cancer types.","Coloc-stats: a unified web interface to perform colocalization analysis of genomic features.","Automating the Determination of Prostate Cancer Risk Strata From Electronic Medical Records.","Computerized Approach to Creating a Systematic Ontology of Hematology/Oncology Regimens.","Relation Extraction from Clinical Narratives Using Pre-trained Language Models.","Integrating shortest dependency path and sentence sequence into a deep learning framework for relation extraction in clinical text.","A novel imaging based Nomogram for predicting post-surgical biochemical recurrence and adverse pathology of prostate cancer from pre-operative bi-parametric MRI.","Machine Learning-Derived Fractal Features of Shape and Texture of the Left Atrium and Pulmonary Veins From Cardiac Computed Tomography Scans Are Associated With Risk of Recurrence of Atrial Fibrillation Postablation.","Feature-driven local cell graph (FLocK): New computational pathology-based descriptors for prognosis of lung cancer and HPV status of oropharyngeal cancers.","Extending the OMOP Common Data Model and Standardized Vocabularies to Support Observational Cancer Research.","Using graph convolutional neural networks to learn a representation for glycans.","CNVpytor: a tool for copy number variation detection and analysis from read depth and allele imbalance in whole-genome sequencing.","PRISM: A Platform for Imaging in Precision Medicine.","Next-Generation Analytics for Omics Data.","Electronic Medical Record Search Engine (EMERSE): An Information Retrieval Tool for Supporting Cancer Research.","Deep Learning Segmentation of Triple-Negative Breast Cancer (TNBC) Patient Derived Tumor Xenograft (PDX) and Sensitivity of Radiomic Pipeline to Tumor Probability Boundary.","Natural language processing for populating lung cancer clinical research data.","Identifying Metastases-related Information from Pathology Reports of Lung Cancer Patients.","Cox regression increases power to detect genotype-phenotype associations in genomic studies using the electronic health record.","Detecting Pharmacovigilance Signals Combining Electronic Medical Records With Spontaneous Reports: A Case Study of Conventional Disease-Modifying Antirheumatic Drugs for Rheumatoid Arthritis.","COVID-19 TestNorm: A tool to normalize COVID-19 testing names to LOINC codes.","Continuity of transcriptomes among colorectal cancer subtypes based on meta-analysis.","International federation of genomic medicine databases using GA4GH standards.","Prostate Cancer Risk Stratification via Nondestructive 3D Pathology with Deep Learning-Assisted Gland Analysis.","Methylation extends the reach of liquid biopsy in cancer detection.","Data-Driven Mathematical Model of Osteosarcoma.","Data Driven Mathematical Model of Colon Cancer Progression.","Hyperspectral Microscopic Imaging for Automatic Detection of Head and Neck Squamous Cell Carcinoma Using Histologic Image and Machine Learning.","Gosling: A Grammar-based Toolkit for Scalable and Interactive Genomics Data Visualization.","PTEN deficiency leads to proteasome addiction: a novel vulnerability in glioblastoma.","Lymphatic-preserving treatment sequencing with immune checkpoint inhibition unleashes cDC1-dependent antitumor immunity in HNSCC.","Comparison of Different Classifiers with Active Learning to Support Quality Control in Nucleus Segmentation in Pathology Images.","PIE: A prior knowledge guided integrated likelihood estimation method for bias reduction in association studies using electronic health records data.","Identifying direct temporal relations between time and events from clinical notes.","DDOT: A Swiss Army Knife for Investigating Data-Driven Biological Ontologies.","The Doppelgänger Effect: Hidden Duplicates in Databases of Transcriptome Profiles.","Data and Statistical Methods To Analyze the Human Microbiome.","Computer Extracted Features from Initial H&amp;amp;E Tissue Biopsies Predict Disease Progression for Prostate Cancer Patients on Active Surveillance.","Current opinion, status and future development of digital pathology in Switzerland.","Assessment of a computerized quantitative quality control tool for whole slide images of kidney biopsies.","Radiomics-based assessment of ultra-widefield leakage patterns and vessel network architecture in the PERMEATE study: insights into treatment durability.","Clonal tracing reveals diverse patterns of response to immune checkpoint blockade.","A case for expert curation: an overview of cancer curation in the Clinical Genome Resource (ClinGen).","SPT6 promotes epidermal differentiation and blockade of an intestinal-like phenotype through control of transcriptional elongation.","Complex mosaic structural variations in human fetal brains.","Report on computational assessment of Tumor Infiltrating Lymphocytes from the International Immuno-Oncology Biomarker Working Group.","Representation of EHR data for predictive modeling: a comparison between UMLS and other terminologies.","Electronic Health Records for Drug Repurposing: Current Status, Challenges, and Future Directions.","pyNBS: a Python implementation for network-based stratification of tumor mutations.","Reliable Analysis of Clinical Tumor-Only Whole-Exome Sequencing Data.","Closing the gap: Systematic integration of multiplexed functional data resolves variants of uncertain significance in BRCA1, TP53, and PTEN.","Test-retest repeatability of a deep learning architecture in detecting and segmenting clinically significant prostate cancer on apparent diffusion coefficient (ADC) maps.","Collagen fiber orientation disorder from H&amp;amp;E images is prognostic for early stage breast cancer: clinical trial validation.","Distinguishing granulomas from adenocarcinomas by integrating stable and discriminating radiomic features on non-contrast computed tomography scans.","Report on computational assessment of Tumor Infiltrating Lymphocytes from the International Immuno-Oncology Biomarker Working Group.","Imaging Features of Vessels and Leakage Patterns Predict Extended Interval Aflibercept Dosing Using Ultra-Widefield Angiography in Retinal Vascular Disease: Findings From the PERMEATE Study.","Diverse noncoding mutations contribute to deregulation of cis-regulatory landscape in pediatric cancers.","Multiscale Computational Fluid Dynamics Modeling for Personalized Liver Cancer Radioembolization Dosimetry.","Standard operating procedure for curation and clinical interpretation of variants in cancer.","Yap1 Mediates Trametinib Resistance in Head and Neck Squamous Cell Carcinomas.","The GA4GH Variation Representation Specification: A computational framework for variation representation and federated identification.","Multiomics analysis of serial PARP inhibitor treated metastatic TNBC inform on rational combination therapies.","PUMA: PANDA Using MicroRNA Associations.","GRAND: a database of gene regulatory network models across human conditions.","Solving the master equation for Indels.","Historian: accurate reconstruction of ancestral sequences and evolutionary rates.","The role of somatic mosaicism in brain disease.","Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability.","Intensity modulated Ir-192 brachytherapy using high-Z 3D printed applicators.","Call for Data Standardization: Lessons Learned and Recommendations in an Imaging Study.","Personalized Integrated Network Modeling of the Cancer Proteome Atlas.","Physicians' strategies for using family history data: having the data is not the same as using the data.","SMART on FHIR in spine: integrating clinical prediction models into electronic health records for precision medicine at the point of care.","Applying a deep learning-based sequence labeling approach to detect attributes of medical concepts in clinical text.","Leveraging existing corpora for de-identification of psychiatric notes using domain adaptation.","HGNChelper: identification and correction of invalid gene symbols for human and mouse.","A community challenge to evaluate RNA-seq, fusion detection, and isoform quantification methods for cancer discovery.","The GA4GH Variation Representation Specification: A computational framework for variation representation and federated identification.","Tensile force-induced cytoskeletal remodeling: Mechanics before chemistry.","Convolutional autoencoder based model HistoCAE for segmentation of viable tumor regions in liver whole-slide images.","Data valuation for medical imaging using Shapley value and application to a large-scale chest X-ray dataset.","Challenges and opportunities for artificial intelligence in oncological imaging.","iGLASS: imaging integration into the Glioma Longitudinal Analysis Consortium.","Skull-Stripping of Glioblastoma MRI Scans Using 3D Deep Learning.","A new machine learning approach for predicting likelihood of recurrence following ablation for atrial fibrillation from CT.","Pathways to breast cancer screening artificial intelligence algorithm validation.","Predictive modeling of single-cell DNA methylome data enhances integration with transcriptome data.","Data Driven Mathematical Model of FOLFIRI Treatment for Colon Cancer.","Genome-wide analysis of somatic noncoding mutation patterns in cancer.","Adversarial Domain Adaptation and Pseudo-Labeling for Cross-Modality Microscopy Image Quantification.","Sequential vessel segmentation via deep channel attention network.","Tissue- and development-stage-specific mRNA and heterogeneous CNV signatures of human ribosomal proteins in normal and cancer samples.","Predicting Cancer Cell Line Dependencies From the Protein Expression Data of Reverse-Phase Protein Arrays.","Contemporary clinical decision support standards using Health Level Seven International Fast Healthcare Interoperability Resources.","The Minimum Information about a Molecular Interaction CAusal STatement (MI2CAST).","Expression Quantitative Trait loci (QTL) in tumor adjacent normal breast tissue and breast tumor tissue.","Novel imaging biomarkers predict outcomes in stage III unresectable non-small cell lung cancer treated with chemoradiation and durvalumab.","Computerized tumor multinucleation index (MuNI) is prognostic in p16+ oropharyngeal carcinoma.","Early Diagnosis and Screening for Lung Cancer.","Fast alignment and preprocessing of chromatin profiles with Chromap.","Dissecting the Tumor-Immune Landscape in Chimeric Antigen Receptor T-cell Therapy: Key Challenges and Opportunities for a Systems Immunology Approach.","Towards pixel-to-pixel deep nucleus detection in microscopy images.","Cancer Detection Using Hyperspectral Imaging and Evaluation of the Superficial Tumor Margin Variance with Depth.","Segmentation of uterus and placenta in MR images using a fully convolutional neural network.","Abdominal muscle segmentation from CT using a convolutional neural network.","Evidence-Based Network Approach to Recommending Targeted Cancer Therapies.","An omic and multidimensional spatial atlas from serial biopsies of an evolving metastatic breast cancer.","Regulatory Network of PD1 Signaling Is Associated with Prognosis in Glioblastoma Multiforme.","JBrowseR: An R Interface to the JBrowse 2 Genome Browser.","KIT&lt;sup&gt;low&lt;/sup&gt; Cells Mediate Imatinib Resistance in Gastrointestinal Stromal Tumor.","Imaging-AMARETTO: An Imaging Genomics Software Tool to Interrogate Multiomics Networks for Relevance to Radiography and Histopathology Imaging Biomarkers of Clinical Outcomes.","Personalized Network Modeling of the Pan-Cancer Patient and Cell Line Interactome.","Automatic identification of recent high impact clinical articles in PubMed to support clinical decision making using time-agnostic features.","ndexr-an R package to interface with the network data exchange.","Control of Gene Expression by RNA Binding Protein Action on Alternative Translation Initiation Sites.","Imaging-AMARETTO: An Imaging Genomics Software Tool to Interrogate Multiomics Networks for Relevance to Radiography and Histopathology Imaging Biomarkers of Clinical Outcomes.","Polyploid giant cancer cell characterization: New frontiers in predicting response to chemotherapy in breast cancer.","Clinical measures, radiomics, and genomics offer synergistic value in AI-based prediction of overall survival in patients with glioblastoma.","Reimagining T Staging Through Artificial Intelligence and Machine Learning Image Processing Approaches in Digital Pathology.","Computationally Derived Cribriform Area Index from Prostate Cancer Hematoxylin and Eosin Images Is Associated with Biochemical Recurrence Following Radical Prostatectomy and Is Most Prognostic in Gleason Grade Group 2.","Quantitative nuclear histomorphometric features are predictive of Oncotype DX risk categories in ductal carcinoma in situ: preliminary findings.","Prospective Evaluation of Repeatability and Robustness of Radiomic Descriptors in Healthy Brain Tissue Regions In Vivo Across Systematic Variations in T2-Weighted Magnetic Resonance Imaging Acquisition Parameters.","Sensitive detection of tumor mutations from blood and its application to immunotherapy prognosis.","FHIR Genomics: enabling standardization for precision medicine use cases.","Standardizing Chemotherapy Regimen Nomenclature: A Proposal and Evaluation of the HemOnc and National Cancer Institute Thesaurus Regimen Content.","Evaluation of immune repertoire inference methods from RNA-seq data.","Integrating molecular profiles into clinical frameworks through the Molecular Oncology Almanac to prospectively guide precision oncology.","Computational Modeling of the Liver Arterial Blood Flow for Microsphere Therapy: Effect of Boundary Conditions.","Automatic detection of head and neck squamous cell carcinoma on pathologic slides using polarized hyperspectral imaging and machine learning.","Using a 22-Layer U-Net to Perform Segmentation of Squamous Cell Carcinoma on Digitized Head and Neck Histological Images.","Augmented Reality-Assisted Biopsy of Soft Tissue Lesions.","Open-Sourced CIViC Annotation Pipeline to Identify and Annotate Clinically Relevant Variants Using Single-Molecule Molecular Inversion Probes.","Single-cell spatial architectures associated with clinical outcome in head and neck squamous cell carcinoma.","Identification of differentially expressed gene sets using the Generalized Berk-Jones statistic.","Predicting genotype-specific gene regulatory networks.","Monte Carlo methods for device simulations in radiation therapy.","DNA Condensation with a Boron-Containing Cationic Peptide for Modeling Boron Neutron Capture Therapy.","AI-driven quantification of ground glass opacities in lungs of COVID-19 patients using 3D computed tomography imaging.","Spatial Characterization of Tumor-Infiltrating Lymphocytes and Breast Cancer Progression.","A DICOM Framework for Machine Learning and Processing Pipelines Against Real-time Radiology Images.","igv.js: an embeddable JavaScript implementation of the Integrative Genomics Viewer (IGV).","Rapid clinical diagnostic variant investigation of genomic patient sequencing data with &lt;i&gt;iobio&lt;/i&gt; web tools.","Credentialing Individual Samples for Proteogenomic Analysis.","Cardiovascular and Metabolic Toxicity of Abiraterone in Castration-resistant Prostate Cancer: Post-marketing Experience.","Comparing models of delivery for cancer genetics services among patients receiving primary care who meet criteria for genetic evaluation in two healthcare systems: BRIDGE randomized controlled trial.","Identification of Novel Therapeutic Molecular Targets in Inflammatory Bowel Disease by Using Genetic Databases.","Deep distributed computing to reconstruct extremely large lineage trees.","Joint Bounding of Peaks Across Samples Improves Differential Analysis in Mass Spectrometry-Based Metabolomics.","PsiNorm: a scalable normalization for single-cell RNA-seq data.","A survey of U.S. public perspectives on facial recognition technology and facial imaging data practices in health and research contexts.","Multi-Compartment Spatially-Derived Radiomics From Optical Coherence Tomography Predict Anti-VEGF Treatment Durability in Macular Edema Secondary to Retinal Vascular Disease: Preliminary Findings.","Automated gleason grading on prostate biopsy slides by statistical representations of homology profile.","Machine learning techniques for mitoses classification.","Investigating Optimal Chemotherapy Options for Osteosarcoma Patients through a Mathematical Model.","&lt;i&gt;In Vivo&lt;/i&gt; Cancer Detection in Animal Model Using Hyperspectral Image Classification with Wavelet Feature Extraction.","Conditional generative adversarial network for synthesizing hyperspectral images of breast cancer cells from digitized histology.","STAN-CT: Standardizing CT Image using Generative Adversarial Networks.","Phase asymmetry ultrasound despeckling with fractional anisotropic diffusion and total variation.","Adaptive deep learning for head and neck cancer detection using hyperspectral imaging.","The Mutational Signature Comprehensive Analysis Toolkit (musicatk) for the Discovery, Prediction, and Exploration of Mutational Signatures.","Large scale genotype- and phenotype-driven machine learning in Von Hippel-Lindau disease.","WNT Signaling Driven by R-spondin 1 and LGR6 in High-grade Serous Ovarian Cancer.","A FZD7-specific Antibody-Drug Conjugate Induces Ovarian Tumor Regression in Preclinical Models.","The multikinase inhibitor RXDX-105 is effective against neuroblastoma &lt;i&gt;in vitro&lt;/i&gt; and &lt;i&gt;in vivo&lt;/i&gt;.","Intra-patient stability of tumor mutational burden from tissue biopsies at different time points in advanced cancers.","The relation between microdosimetry and induction of direct damage to DNA by alpha particles.","Validation of the TOPAS Monte Carlo toolkit for HDR brachytherapy simulations.","A DICOM dataset for evaluation of medical image de-identification.","An accessible GenePattern notebook for the copy number variation analysis of Illumina Infinium DNA methylation arrays.","CASIDE: A data model for interoperable cancer survivorship information based on FHIR.","Developing Customizable Cancer Information Extraction Modules for Pathology Reports Using CLAMP.","NDEx: Accessing Network Models and Streamlining Network Biology Workflows.","A BioID-Derived Proximity Interactome for SARS-CoV-2 Proteins.","The impact of different sources of heterogeneity on loss of accuracy from genomic prediction models.","SpatialExperiment: infrastructure for spatially-resolved transcriptomics data in R using Bioconductor.","Global Alliance for Genomics and Health Meets Bioconductor: Toward Reproducible and Agile Cancer Genomics at Cloud Scale.","SIMON: Open-Source Knowledge Discovery Platform.","Morphometric Analysis of Retinal Pigment Epithelial Cells From C57BL/6J Mice During Aging.","Comparison of segmentation-free and segmentation-dependent computer-aided diagnosis of breast masses on a public mammography dataset.","Analyzing magnetic resonance imaging data from glioma patients using deep learning.","Computational image features of immune architecture is associated with clinical benefit and survival in gynecological cancers across treatment modalities.","A prognostic and predictive computational pathology image signature for added benefit of adjuvant chemotherapy in early stage non-small-cell lung cancer.","Update on the current opinion, status and future development of digital pathology in Switzerland in light of COVID-19.","Quantitative Assessment of the Effects of Compression on Deep Learning in Digital Pathology Image Analysis.","Coinfections in Patients With Cancer and COVID-19: A COVID-19 and Cancer Consortium (CCC19) Study.","Cancer Therapy Approval Timings, Review Speed, and Publication of Pivotal Registration Trials in the US and Europe, 2010-2019.","Deep learning-based segmentation of the placenta and uterus on MR images.","Detection of Squamous Cell Carcinoma in Digitized Histological Images from the Head and Neck Using Convolutional Neural Networks.","Hyperspectral Microscopic Imaging for the Detection of Head and Neck Squamous Cell Carcinoma on Histologic Slides.","CIViCpy: A Python Software Development and Analysis Toolkit for the CIViC Knowledgebase.","Impact of a 40-Gene Targeted Panel Test on Physician Decision Making for Patients With Acute Myeloid Leukemia.","High resolution copy number inference in cancer using short-molecule nanopore sequencing.","TORC1/2 kinase inhibition depletes glutathione and synergizes with carboplatin to suppress the growth of MYC-driven medulloblastoma.","Monte Carlo Processing on a Chip (MCoaC)-preliminary experiments toward the realization of optimal-hardware for TOPAS/Geant4 to drive discovery.","Parallel and Efficient Sensitivity Analysis of Microscopy Image Segmentation Workflows in Hybrid Systems.","DICOM re-encoding of volumetrically annotated Lung Imaging Database Consortium (LIDC) nodules.","Ongoing clonal evolution in chronic myelomonocytic leukemia on hypomethylating agents: a computational perspective.","Are synthetic clinical notes useful for real natural language processing tasks: A case study on clinical entity recognition.","Efficient and Accurate Extracting of Unstructured EHRs on Cancer Therapy Responses for the Development of RECIST Natural Language Processing Tools: Part I, the Corpus.","GPU-based Real-time Contact Tracing at Scale.","The University of Pennsylvania glioblastoma (UPenn-GBM) cohort: advanced MRI, clinical, genomics, &amp;amp; radiomics.","The state of the art for artificial intelligence in lung digital pathology.","Cost-effective methylome sequencing of cell-free DNA for accurately detecting and locating cancer.","Rapid diagnosis and comprehensive bacteria profiling of sepsis based on cell-free DNA.","MLCD: A Unified Software Package for Cancer Diagnosis.","Scale-Aware Transformers for Diagnosing Melanocytic Lesions.","Sparse reconstruction of log-conductivity in current density impedance tomography.","Realistic boundary conditions in SimVascular through inlet catheter modeling.","Generative Adversarial Domain Adaptation for Nucleus Quantification in Images of Tissue Immunohistochemically Stained for Ki-67.","Bidirectional Mapping-Based Domain Adaptation for Nucleus Detection in Cross-Modality Microscopy Images.","Renal biopsy under augmented reality guidance.","A multi-level similarity measure for the retrieval of the common CT imaging signs of lung diseases.","Incorporating minimal user input into deep learning based image segmentation.","Epigenetic Conservation Is a Beacon of Function: An Analysis Using Methcon5 Software for Studying Gene Methylation.","CIViCdb 2022: evolution of an open-access cancer variant interpretation knowledgebase.","MIRA: joint regulatory modeling of multimodal expression and chromatin accessibility in single cells.","GenomicSuperSignature facilitates interpretation of RNA-seq experiments through robust, efficient comparison to public databases.","Cloud bursting galaxy: federated identity and access management.","Galaxy-ML: An accessible, reproducible, and scalable machine learning toolkit for biomedicine.","Gene Targeting in Disease Networks.","A Model of Indel Evolution by Finite-State, Continuous-Time Machines.","Cram-JS: reference-based decompression in node and the browser.","Cyclin E Overexpression in Human Mammary Epithelial Cells Promotes Epithelial Cancer-Specific Copy Number Alterations.","Understanding Scanner Utilization With Real-Time DICOM Metadata Extraction.","Deep learning for survival analysis in breast cancer with whole slide image data.","Novel temporal and spatial patterns of metastatic colonization from breast cancer rapid-autopsy tumor biopsies.","Sharing Begins at Home: How Continuous and Ubiquitous FAIRness Can Enhance Research Productivity and Data Reuse.","More reliable breast cancer risk assessment for every woman.","Interactive Visual Displays for Interpreting the Results of Clinical Trials: Formative Evaluation With Case Vignettes.","Assessment of Electronic Health Record for Cancer Research and Patient Care Through a Scoping Review of Cancer Natural Language Processing.","Infusion Related Hypersensitivity Reactions with Bio-similar Anti CD-20 Monoclonal Antibody Rituximab in Indian Patients: A Retrospective Study.","Seven decades of chemotherapy clinical trials: a pan-cancer social network analysis.","Characterizing the Anticancer Treatment Trajectory and Pattern in Patients Receiving Chemotherapy for Cancer Using Harmonized Observational Databases: Retrospective Study.","Common genetic variation in the germline influences where and how tumors develop.","Strategies for Network GWAS Evaluated Using Classroom Crowd Science.","A BioID-derived proximity interactome for SARS-CoV-2 proteins.","TFutils: Data structures for transcription factor bioinformatics.","GenomicSuperSignature facilitates interpretation of RNA-seq experiments through robust, efficient comparison to public databases.","Metagenome analysis using the Kraken software suite.","Gene Fusions Create Partner and Collateral Dependencies Essential to Cancer Cell Survival.","PhyloCSF++: a fast and user-friendly implementation of PhyloCSF with annotation tools.","AUTOMATIC BRAIN ORGAN SEGMENTATION WITH 3D FULLY CONVOLUTIONAL NEURAL NETWORK FOR RADIATION THERAPY TREATMENT PLANNING.","Handling data heterogeneity with generative replay in collaborative learning for medical imaging.","SplitAVG: A Heterogeneity-Aware Federated Deep Learning Method for Medical Imaging.","The federated tumor segmentation (FeTS) tool: an open-source solution to further solid tumor research.","Quantifying T2-FLAIR Mismatch Using Geographically Weighted Regression and Predicting Molecular Status in Lower-Grade Gliomas.","Classification of Infection and Ischemia in Diabetic Foot Ulcers Using VGG Architectures.","CrossMoDA 2021 challenge: Benchmark of cross-modality domain adaptation techniques for vestibular schwannoma and cochlea segmentation.","Systematic Evaluation of Image Tiling Adverse Effects on Deep Learning Semantic Segmentation.","Computational Analysis of Routine Biopsies Improves Diagnosis and Prediction of Cardiac Allograft Vasculopathy.","A Novel Nodule Edge Sharpness Radiomic Biomarker Improves Performance of Lung-RADS for Distinguishing Adenocarcinomas from Granulomas on Non-Contrast CT Scans.","Integrated Clinical and CT Based Artificial Intelligence Nomogram for Predicting Severity and Need for Ventilator Support in COVID-19 Patients: A Multi-Site Study.","Semiempirical, parameterized spectrum estimation for x-ray computed tomography.","Characterizing the Anticancer Treatment Trajectory and Pattern in Patients Receiving Chemotherapy for Cancer Using Harmonized Observational Databases: Retrospective Study.","Evaluation of Information Theoretic Network Meta-analysis to Rank First-Line Anticancer Regimens for Hormone Receptor-Positive, ERBB2-Negative Metastatic Breast Cancer.","Seven decades of chemotherapy clinical trials: a pan-cancer social network analysis.","Chemotherapy Knowledge Base Management in the Era of Precision Oncology.","Pathology Trainees' Experience and Attitudes on Use of Digital Whole Slide Images.","Segmenting Skin Biopsy Images with Coarse and Sparse Annotations using U-Net.","Control of ribosomal RNA synthesis by hematopoietic transcription factors.","TumorDecon: A digital cytometry software.","Pixel-level Tumor Margin Assessment of Surgical Specimen with Hyperspectral Imaging and Deep Learning Classification.","Automatic detection of head and neck squamous cell carcinoma on histologic slides using hyperspectral microscopic imaging.","Fully automated segmentation of the right ventricle in patients with repaired Tetralogy of Fallot using U-Net.","Development of a new polarized hyperspectral imaging microscope.","Augmented reality visualization of hyperspectral imaging classifications for image-guided brain tumor phantom resection.","A community approach to the cancer-variant-interpretation bottleneck.","GenoREC: A Recommendation System for Interactive Genomics Data Visualization.","Machine Boss: rapid prototyping of bioinformatic automata.","JBrowse Connect: A server API to connect JBrowse instances and users.","CHMP2A regulates tumor sensitivity to natural killer cell-mediated cytotoxicity.","Single-cell characterization of step-wise acquisition of carboplatin resistance in ovarian cancer.","A noncoding single-nucleotide polymorphism at 8q24 drives &lt;i&gt;IDH1&lt;/i&gt;-mutant glioma formation.","LongAGE: defining breakpoints of genomic structural variants through optimal and memory efficient alignments of long reads.","SCELLECTOR: ranking amplification bias in single cells using shallow sequencing.","DICOM-RT Ion interface to utilize MC simulations in routine clinical workflow for proton pencil beam radiotherapy.","Monte Carlo simulation of the effect of magnetic fields on brachytherapy dose distributions in lung tissue material.","&lt;i&gt;TilGAN&lt;/i&gt;: GAN for Facilitating Tumor-Infiltrating Lymphocyte Pathology Image Synthesis With Improved Image Classification.","Sensitivity analysis in digital pathology: Handling large number of parameters with compute expensive workflows.","Rapid conversion of chronic myeloid leukemia to chronic myelomonocytic leukemia in a patient on imatinib therapy.","Tumor-intrinsic SIRPA promotes sensitivity to checkpoint inhibition immunotherapy in melanoma.","Characterization of anticancer drug resistance by reverse-phase protein array: new targets and strategies.","Association of Disparities in Family History and Family Cancer History in the Electronic Health Record With Sex, Race, Hispanic or Latino Ethnicity, and Language Preference in 2 Large US Health Care Systems.","Information Extraction for Populating Lung Cancer Clinical Research Data.","Normalizing Clinical Document Titles to LOINC Document Ontology: an Initial Study.","Achievability to Extract Specific Date Information for Cancer Research.","Parsing clinical text using the state-of-the-art deep learning based parsers: a systematic comparison.","Cox regression is robust to inaccurate EHR-extracted event time: an application to EHR-based GWAS.","Recognizing software names in biomedical literature using machine learning.","Orchestrating a community-developed computational workshop and accompanying training materials.","A spatial attention guided deep learning system for prediction of pathological complete response using breast cancer histopathology images.","IDEAL: a Vector-Raster Hybrid Model for Efficient Spatial Queries over Complex Polygons.","MultiHeadGAN: A deep learning method for low contrast retinal pigment epithelium cell segmentation with fluorescent flatmount microscopy images.","Computational Model-Based Estimation of Mouse Eyeball Structure From Two-Dimensional Flatmount Microscopy Images.","Effective nuclei segmentation with sparse shape prior and dynamic occlusion constraint for glioblastoma pathology images.","MRI pulse sequence integration for deep-learning-based brain metastases segmentation.","Machine Learning-Enabled Multimodal Fusion of Intra-Atrial and Body Surface Signals in Prediction of Atrial Fibrillation Ablation Outcomes.","Clinical Trial Design and Development Work Group Within the Quantitative Imaging Network.","Rethinking Architecture Design for Tackling Data Heterogeneity in Federated Learning.","O-Net: An Overall Convolutional Network for Segmentation Tasks.","OpenFL: the open federated learning library.","Estimating Glioblastoma Biophysical Growth Parameters Using Deep Learning Regression.","Estimating Glioblastoma Biophysical Growth Parameters Using Deep Learning Regression.","Clinically Deployed Computational Assessment of Multiple Sclerosis Lesions.","The revolving door for AI and pathologists-docendo discimus?","Computer-extracted features of nuclear morphology in hematoxylin and eosin images distinguish stage II and IV colon tumors.","Radiomic Deformation and Textural Heterogeneity (R-DepTH) Descriptor to Characterize Tumor Field Effect: Application to Survival Prediction in Glioblastoma.","Oropharyngeal cancer outcomes correlate with p16 status, multinucleation and immune infiltration.","Computational Analysis of High-Dimensional DNA Methylation Data for Cancer Prognosis.","cfTrack: A Method of Exome-Wide Mutation Analysis of Cell-free DNA to Simultaneously Monitor the Full Spectrum of Cancer Treatment Outcomes Including MRD, Recurrence, and Evolution.","Opportunities and Challenges for Analyzing Cancer Data at the Inter- and Intra-Institutional Levels.","Improving the Diagnosis of Skin Biopsies Using Tissue Segmentation.","Comprehensive Characterizations of Immune Receptor Repertoire in Tumors and Cancer Immunotherapy Studies.","Sparsity-based nonlinear reconstruction of optical parameters in two-photon photoacoustic computed tomography.","A semiautomatic approach for prostate segmentation in MR images using local texture classification and statistical shape modeling.","Three-dimensional prostate CT segmentation through fine-tuning of a pre-trained neural network using no reference labeling.","Development of a real-time spectral imaging system using in-site micro-LED-based illumination and high-speed micro-camera for endoscopic applications.","Radiomics Analysis of MRI for Predicting Molecular Subtypes of Breast Cancer in Young Women.","Virtual Reality Assisted Cardiac Catheterization.","Automatic Segmentation of the Prostate on MR Images based on Anatomy and Deep Learning.","A complex dual-modality kidney phantom for renal biopsy studies.","Image Guided Mitral Valve Replacement: Registration of 3D Ultrasound and 2D X-ray Images.","Multiparametric Radiomics for Predicting the Aggressiveness of Papillary Thyroid Carcinoma Using Hyperspectral Images.","Bam-readcount -- rapid generation of basepair-resolution sequence metrics.","&lt;i&gt;In silico&lt;/i&gt; epitope prediction analyses highlight the potential for distracting antigen immunodominance with allogeneic cancer vaccines.","Standardized evidence-based approach for assessment of oncogenic and clinical significance of NTRK fusions.","Personalized ctDNA micro-panels can monitor and predict clinical outcomes for patients with triple-negative breast cancer.","Accessing clinical-grade genomic classification data through the ClinGen Data Platform.","Multi-View Design Patterns and Responsive Visualization for Genomics Data.","Cistrome Explorer: an interactive visual analysis tool for large-scale epigenomic data.","Galaxy Training: A powerful framework for teaching!","Expanding the Galaxy's reference data.","Proceedings of the fifth international Molecular Pathological Epidemiology (MPE) meeting.","Environmental Influences Measured by Epigenetic Clock and Vulnerability Components at Birth Impact Clinical ASD Heterogeneity.","An online notebook resource for reproducible inference, analysis and publication of gene regulatory networks.","annoFuse: an R Package to annotate, prioritize, and interactively explore putative oncogenic RNA fusions.","Proceedings of the fifth international Molecular Pathological Epidemiology (MPE) meeting.","Differential regulation of TNFα and IL-6 expression contributes to immune evasion in prostate cancer.","Subgroup-Enriched Pathways and Kinase Signatures in Medulloblastoma Patient-Derived Xenografts.","Pre- and post-treatment image-based dosimetry in&lt;sup&gt;90&lt;/sup&gt;Y-microsphere radioembolization using the TOPAS Monte Carlo toolkit.","Deep learning-based approach to the characterization and quantification of histopathology in mouse models of colitis.","Towards an internet-scale overlay network for latency-aware decentralized workflows at the edge.","The PRISM semantic cohort builder: a novel tool to search and access clinical data in TCIA imaging collections.","Multi-objective Parameter Auto-tuning for Tissue Image Segmentation Workflows.","OncoGEMINI: software for investigating tumor variants from multiple biopsies with integrated cancer annotations.","CUF-Links: Continuous and Ubiquitous FAIRness Linkages for reproducible research.","Convergent Usage of Amino Acids in Human Cancers as A Reversed Process of Tissue Development.","A functional genomic approach to actionable gene fusions for precision oncology.","Depression, suicide intent, and plan among adults presenting in an emergency department after making a suicide attempt: Exploratory differences by psychosis symptom experience.","Integrating the OHIF Viewer into XNAT: Achievements, Challenges and Prospects for Quantitative Imaging Studies.","Patient Interactions With an Automated Conversational Agent Delivering Pretest Genetics Education: Descriptive Study.","GARDE: a standards-based clinical decision support platform for identifying population health management cohorts.","Identifying Patients Who Meet Criteria for Genetic Testing of Hereditary Cancers Based on Structured and Unstructured Family Health History Data in the Electronic Health Record: Natural Language Processing Approach.","Barriers to family history collection among Spanish-speaking primary care patients: a BRIDGE qualitative study.","PDCM Finder: an open global research platform for patient-derived cancer models.","Detect Attributes of Medical Concepts via Sequence Labeling.","Development of a bayesian toxo-equivalence model between docetaxel and paclitaxel.","Strategies for Network GWAS Evaluated Using Classroom Crowd Science.","Identifying persistent structures in multiscale 'omics data.","Mapping the common gene networks that underlie related diseases.","Consensus transcriptional regulatory networks of coronavirus-infected human cells.","Per-sample standardization and asymmetric winsorization lead to accurate clustering of RNA-seq expression profiles.","Open-source Software Sustainability Models: Initial White Paper From the Informatics Technology for Cancer Research Sustainability and Industry Partnership Working Group.","restfulSE: A semantically rich interface for cloud-scale genomics with Bioconductor.","Waldron et al. Reply to \"Commentary on the HMP16SData Bioconductor Package\".","BiocPkgTools: Toolkit for mining the &lt;i&gt;Bioconductor&lt;/i&gt; package ecosystem.","NewWave: a scalable R/Bioconductor package for the dimensionality reduction and batch effect removal of single-cell RNA-seq data.","Development and application of a computable genotype model in the GA4GH Variation Representation Specification.","U.S. Adult Perspectives on Facial Images, DNA, and Other Biometrics.","Federated analysis of BRCA1 and BRCA2 variation in a Japanese cohort.","Virtual liver needle biopsy from reconstructed three-dimensional histopathological images: Quantification of sampling error.","SPEAR: Dynamic Spatio-Temporal Query Processing over High Velocity Data Streams.","IMAGE REGISTRATION WITH OPTIMAL REGULARIZATION PARAMETER SELECTION BY LEARNED AUTO ENCODER FEATURES.","HISTOPATHOLOGY IMAGE REGISTRATION BY INTEGRATED TEXTURE AND SPATIAL PROXIMITY BASED LANDMARK SELECTION AND MODIFICATION.","An integrative web-based software tool for multi-dimensional pathology whole-slide image analytics.","Spatial Attention-Based Deep Learning System for Breast Cancer Pathological Complete Response Prediction with Serial Histopathology Images in Multiple Stains.","3DPro: Querying Complex Three-Dimensional Data with Progressive Compression and Refinement.","Accelerating Spatial Cross-Matching on CPU-GPU Hybrid Platform With CUDA and OpenACC.","Efficient 3D Spatial Queries for Complex Objects.","Towards More Efficient Data Valuation in Healthcare Federated Learning using Ensembling.","Advancing COVID-19 Diagnosis with Privacy-Preserving Collaboration in Artificial Intelligence.","Highdicom: a Python Library for Standardized Encoding of Image Annotations and Machine Learning Model Outputs in Pathology and Radiology.","A Deep Network for Joint Registration and Reconstruction of Images with Pathologies.","Robust, Primitive, and Unsupervised Quality Estimation for Segmentation Ensembles.","Enhancing the REMBRANDT MRI collection with expert segmentation labels and quantitative radiomic features.","Interactive Machine Learning-Based Multi-Label Segmentation of Solid Tumors and Organs.","Optimization of Deep Learning Based Brain Extraction in MRI for Low Resource Environments.","Novel Radiomic Measurements of Tumor-Associated Vasculature Morphology on Clinical Imaging as a Biomarker of Treatment Response in Multiple Cancers.","Automated analysis of computerized morphological features of cell clusters associated with malignancy on bile duct brushing whole slide images.","Multi-site cross-organ calibrated deep learning (MuSClD): Automated diagnosis of non-melanoma skin cancer.","Survival prediction on intrahepatic cholangiocarcinoma with histomorphological analysis on the whole slide images.","Artificial Intelligence in Surveillance of Barrett's Esophagus.","Amyloid Deposition Is Greater in Cerebral Gyri than in Cerebral Sulci with Worsening Clinical Diagnosis Across the Alzheimer's Disease Spectrum.","A machine learning model for separating epithelial and stromal regions in oral cavity squamous cell carcinomas using H&amp;amp;E-stained histology images: A multi-center, retrospective study.","Image analysis reveals differences in tumor multinucleations in Black and White patients with human papillomavirus-associated oropharyngeal squamous cell carcinoma.","RADIomic Spatial TexturAl Descriptor (RADISTAT): Quantifying Spatial Organization of Imaging Heterogeneity Associated With Tumor Response to Treatment.","Evaluating the utility of deep learning for predicting therapeutic response in diabetic eye disease.","XCIST-an open access x-ray/CT simulation toolkit.","SMART COVID Navigator, a Clinical Decision Support Tool for COVID-19 Treatment: Design and Development Study.","Classifying Breast Histopathology Images with a Ductal Instance-Oriented Pipeline.","Automated quantitative assessment of amorphous calcifications: Towards improved malignancy risk stratification.","Analysis of Regions of Interest and Distractor Regions in Breast Biopsy Images.","Robust and efficient semi-supervised estimation of average treatment effects with application to electronic health records data.","Developing and evaluating risk prediction models with panel current status data.","Semisupervised Calibration of Risk with Noisy Event Times (SCORNET) using electronic health record data.","Risk prediction with imperfect survival outcome information from electronic health records.","A Fokker-Planck Framework for Parameter Estimation and Sensitivity Analysis in Colon Cancer.","Bio-Mechanical Model of Osteosarcoma Tumor Microenvironment: A Porous Media Approach.","A New Non-Linear Conjugate Gradient Algorithm for Destructive Cure Rate Model and a Simulation Study: Illustration with Negative Binomial Competing Risks.","A Fokker-Planck feedback control framework for optimal personalized therapies in colon cancer-induced angiogenesis.","The Impact of Injection Distance to Bifurcations on Yttrium-90 Distribution in Liver Cancer Radioembolization.","Low-Resource Adversarial Domain Adaptation for Cross-Modality Nucleus Detection.","Selective modification of fluciclovine (&lt;sup&gt;18&lt;/sup&gt;F) transport in prostate carcinoma xenografts.","Cross-Vendor CT Image Data Harmonization Using CVH-CT.","Development of a polarized hyperspectral microscope for cardiac fiber orientation imaging.","Siamese neural networks for the classification of high-dimensional radiomic features.","Unsupervised super-resolution reconstruction of hyperspectral histology images for whole-slide imaging.","GPU-Based Simulation of Echocardiography Volumes Using Quantitative Fiber-Angle-to-Backscatter Measurements.","Assessing reproducibility in Magnetic Resonance (MR) Radiomics features between Deep-Learning segmented and Expert Manual segmented data and evaluating their diagnostic performance in Pregnant Women with suspected Placenta Accreta Spectrum (PAS).","Computer-aided Classification of Lung Nodules on CT Images with Expert Knowledge.","&lt;i&gt;iMutSig&lt;/i&gt;: a web application to identify the most similar mutational signature using shiny."]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>coreproject<\/th>\n      <th>pmid<\/th>\n      <th>year<\/th>\n      <th>citation_count<\/th>\n      <th>journal<\/th>\n      <th>title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":[1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script><p>Citations per project are summarized below.</p>
<div class="sourceCode" id="cb27"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">citations</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/group_by.html" class="external-link">group_by</a></span><span class="op">(</span><span class="va">coreproject</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/summarise.html" class="external-link">summarize</a></span><span class="op">(</span></span>
<span>        n_pub <span class="op">=</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/context.html" class="external-link">n</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>        n_uncite <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/sum.html" class="external-link">sum</a></span><span class="op">(</span><span class="va">citation_count</span> <span class="op">==</span> <span class="fl">0L</span><span class="op">)</span>,</span>
<span>        citn_count <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/sum.html" class="external-link">sum</a></span><span class="op">(</span><span class="va">citation_count</span><span class="op">)</span></span>
<span>    <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/desc.html" class="external-link">desc</a></span><span class="op">(</span><span class="va">citn_count</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">project_summaries</span>, by <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span>coreproject <span class="op">=</span> <span class="st">"core_project_num"</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-3c76601457be35bc8c2f" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-3c76601457be35bc8c2f">{"x":{"filter":"none","vertical":false,"data":[["U24CA180922","U24CA209851","U24CA180996","U24CA204817","U24CA210004","U24CA209999","U01CA231840","U24CA184427","U01CA239055","U24CA215109","U24CA237719","U24CA231877","U24CA194215","U24CA220341","U24CA220457","U01CA242871","U01CA242879","U24CA237617","R21CA231911","U01CA226196","U24CA215123","R21CA242861","U01CA226187","U24CA204781","U24CA220242","U01CA242954","U24CA231846","R21CA242933","U24CA204800","U01CA237711","U24CA204863","U01CA242936","U01CA231782","U24CA204854","R21CA237493","R21CA237686","U24CA220441","U24CA209996","R21CA242940","R21CA226188","R21CA226106","U01CA231860"],[31,43,47,12,7,19,22,40,73,32,28,18,41,28,6,26,15,9,45,8,12,4,8,3,10,9,13,14,15,9,5,21,11,4,6,5,7,3,5,1,2,2],[0,2,6,0,0,1,1,4,10,4,5,2,2,2,2,5,3,2,8,0,1,0,0,1,0,3,3,4,4,0,1,9,3,1,1,1,0,1,4,0,1,1],[12309,4574,2293,2084,1987,1716,1581,1565,1461,897,732,653,585,580,485,446,241,210,202,191,131,128,127,127,123,98,98,92,91,64,62,54,52,43,39,34,28,26,14,5,3,2],[10,5,8,5,5,5,3,8,4,5,4,5,5,5,5,3,3,5,2,3,5,2,3,5,5,3,4,2,5,3,5,3,3,5,2,2,4,5,2,2,2,3],[7263992,3905934,5281746,2493392,3802236,3743640,1239743,5658437,1045548,8140937,2942615,3850721,3274448,3392981,6365918,1096884,1191276,3639833,388776,1286835,4207966,602867,1165246,1904229,2627986,1339635,2570825,386767,3747474,1127188,3824879,1075520,980371,3040146,372034,368539,2737046,3962385,369895,394763,460732,1518187],["Trinity: Transcriptome assembly for genetic and functional analysis of cancer","TCPA: an Integrated Bioinformatics Resource for Functional Cancer Proteomic Data","Cancer Genomics: Integrative and Scalable Solutions in R/Bioconductor","Informatics Tools for High-throughput Analysis of Cancer Mutations","The Integrative Genomics Viewer (IGV): visualization supporting cancer research","Monitoring tumor subclonal heterogeneity over time and space","Network Analysis for a Data-Driven Approach to Cancer Care","NDEx - the Network Data Exchange A Network Commons for Biologists","HistoTools:  A suite of digital pathology tools for quality control, annotation and dataset identification","TCIA Sustainment and Scalability - Platforms for Quantitative Imaging Informatics in Precision Medicine","Standardized and Genome-Wide Clinical Interpretation of Complex Genotypes for Cancer Precision Medicine","A Federated Galaxy for user-friendly large-scale cancer genomics research","Advancing Cancer Pharmacoepidemiology Research Through EHRs and Informatics","Supporting and evolving Gene Set Enrichment Analysis and the Molecular Signatures Database for cancer research","The cBioPortal for Cancer Genomics","The Federated Tumor Segmentation (FeTS) platform: An intuitive tool facilitating secure multi-institutional collaboration","Distributed Learning of Deep Learning Models for Cancer Research","Developing Informatics Technologies to Model Cancer Gene Regulation","STAN-CT: Standardization and Normalization of CT images for Lung Cancer Patients","Bioinformatics Technology to Characterize Tumor Infiltrating Immune Repertoires","The TOPAS Tool for Particle Simulation, a Monte Carlo Simulation Tool for Physics, Biology and Clinical Research","A statistical framework to systematically characterize cancer driver mutations in noncoding genomic regions","Tools for annotating mutations in the 3D cancer genome","The PDX Integrator- Re-purposing KOMP2 informatics to widely disseminate data from Patient Derived Xenografts","Detection of somatic, subclonal and mosaic CNVs from sequencing","Eliminating variants of uncertain significance in BRCA1, BRCA2 and beyond","WebMeV: A Robust Platform for Intuitive Genomic Data Analysis","Data-driven QSP software for personalized colon cancer treatment","Scalable Clinical Decision Support for Individualized Cancer Risk Management","Informatics resources for liquid biopsy research","Advanced Development and Dissemination of EMERSE for Cancer Phenotyping from Medical Records","Computational pathology software for integrative cancer research with three-dimensional digital slides","A Unified Machine Learning Package for Cancer Diagnosis","INTEGRATIVE IMAGING INFORMATICS FOR CANCER RESEARCH","Development and Dissemination of KiNet: A Novel Imaging Informatics Tool for Gastrointestinal and Pancreatic Neuroendocrine Tumors","Personalized dosimetry for liver cancer radioembolization using fluid dynamics simulation","Developing the JBrowse Genome Browser to Visualize Structural Variants and Cancer Genomics Data","Building protected data sharing networks to advance cancer risk assessment and treatment","Semi-supervised Algorithms for Risk Assessment with Noisy EHR Data","Enhanced deconvolution and prediction of mutational signatures","Conservation and functional-characterization of tumor methylation sites","Open-access X-ray and CT simulation toolkit for research in cancer imaging and dosimetry"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>coreproject<\/th>\n      <th>n_pub<\/th>\n      <th>n_uncite<\/th>\n      <th>citn_count<\/th>\n      <th>n_years<\/th>\n      <th>amount<\/th>\n      <th>project_title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script><div class="section level3">
<h3 id="itcr-collaborations">ITCR collaborations<a class="anchor" aria-label="anchor" href="#itcr-collaborations"></a>
</h3>
<p>An opportunity for (indirect) collaboration occurs when one project
cites the work of another project. Thus we query <a href="https://icite.od.nih.gov/" class="external-link">iCite</a> for the publications that
cited ITCR publications, and exclude publications that are not
themselves ITCR publications.</p>
<div class="sourceCode" id="cb28"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">include_fields</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"pmid"</span>, <span class="st">"cited_by"</span><span class="op">)</span></span>
<span><span class="va">cited_by</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/user_interface.html">icite</a></span><span class="op">(</span><span class="va">publications</span>, <span class="va">include_fields</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## 'cited_by' is a space-delimited character vector of pmid ids; expand these...</span></span>
<span><span class="va">cited_by_pmids</span> <span class="op">&lt;-</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/pull.html" class="external-link">pull</a></span><span class="op">(</span><span class="va">cited_by</span>, <span class="st">"cited_by"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">lapply</a></span><span class="op">(</span><span class="va">strsplit</span>, <span class="st">"[[:blank:]]"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/unlist.html" class="external-link">unlist</a></span><span class="op">(</span>recursive <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## ...construct a tibble of pmid / cited_by pmid</span></span>
<span><span class="va">pmid_cited_by_itcr</span> <span class="op">&lt;-</span></span>
<span>    <span class="fu"><a href="https://tibble.tidyverse.org/reference/tibble.html" class="external-link">tibble</a></span><span class="op">(</span></span>
<span>        pmid <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/rep.html" class="external-link">rep</a></span><span class="op">(</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/pull.html" class="external-link">pull</a></span><span class="op">(</span><span class="va">cited_by</span>, <span class="st">"pmid"</span><span class="op">)</span>, <span class="fu"><a href="https://rdrr.io/r/base/lengths.html" class="external-link">lengths</a></span><span class="op">(</span><span class="va">cited_by_pmids</span><span class="op">)</span><span class="op">)</span>,</span>
<span>        cited_by <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/unlist.html" class="external-link">unlist</a></span><span class="op">(</span><span class="va">cited_by_pmids</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/base/integer.html" class="external-link">as.integer</a></span><span class="op">(</span><span class="op">)</span></span>
<span>    <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="co">## and exclude citations by non-ITCR publications</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">cited_by</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="va">pmid</span><span class="op">)</span></span></code></pre></div>
<p>There are 840 citations of ITCR publications by ITCR projects.</p>
<div class="sourceCode" id="cb29"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">core_project_citations</span> <span class="op">&lt;-</span></span>
<span>    <span class="va">pmid_cited_by_itcr</span> <span class="op">|&gt;</span></span>
<span>    <span class="co">## map pmid to coreproject</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">publications</span>, multiple <span class="op">=</span> <span class="st">"all"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/rename.html" class="external-link">rename</a></span><span class="op">(</span>core_project_num <span class="op">=</span> <span class="va">coreproject</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="co">## map cited_by to coreproject</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span></span>
<span>        <span class="va">publications</span>,</span>
<span>        by <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span>cited_by <span class="op">=</span> <span class="st">"pmid"</span><span class="op">)</span>,</span>
<span>        multiple <span class="op">=</span> <span class="st">"all"</span></span>
<span>    <span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/rename.html" class="external-link">rename</a></span><span class="op">(</span>cited_by_core_project <span class="op">=</span> <span class="va">coreproject</span><span class="op">)</span></span>
<span><span class="co">#&gt; Joining with `by = join_by(pmid)`</span></span></code></pre></div>
<p>The following summarizes citations at the project level</p>
<div class="sourceCode" id="cb30"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">project_project_citations</span> <span class="op">&lt;-</span></span>
<span>    <span class="va">core_project_citations</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/count.html" class="external-link">count</a></span><span class="op">(</span><span class="va">core_project_num</span>, cited_by <span class="op">=</span> <span class="va">cited_by_core_project</span>, name <span class="op">=</span> <span class="st">"n_citns"</span><span class="op">)</span></span></code></pre></div>
<p>Self-citations are</p>
<div class="sourceCode" id="cb31"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">project_project_citations</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">core_project_num</span> <span class="op">==</span> <span class="va">cited_by</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/desc.html" class="external-link">desc</a></span><span class="op">(</span><span class="va">n_citns</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">project_summaries</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="op">-</span><span class="va">cited_by</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/rename.html" class="external-link">rename</a></span><span class="op">(</span>self_citn <span class="op">=</span> <span class="st">"n_citns"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; Joining with `by = join_by(core_project_num)`</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-8e766935e15d84041fb3" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-8e766935e15d84041fb3">{"x":{"filter":"none","vertical":false,"data":[["U01CA239055","U24CA180922","U24CA184427","U24CA215109","U24CA180996","R21CA231911","U01CA242871","U24CA194215","U01CA231840","R21CA242933","U24CA237719","U24CA209851","U24CA204800","U24CA204817","U24CA209999","U24CA231877","R21CA237686","U01CA231782","U01CA242954","U24CA215123","U24CA220341","U01CA226196","U01CA242879","U24CA231846","R21CA237493","U24CA237617","R21CA242861","U24CA204781","U24CA220242","U24CA220441","U24CA210004","R21CA242940","U01CA226187","U01CA231860","U01CA237711","U01CA242936","U24CA204863"],[107,63,63,40,37,36,36,35,32,29,29,27,23,18,15,13,10,9,9,9,9,8,8,7,5,5,3,3,3,3,2,1,1,1,1,1,1],[4,10,8,5,8,2,3,5,3,2,4,5,5,5,5,5,2,3,3,5,5,3,3,4,2,5,2,5,5,4,5,2,3,3,3,3,5],[1045548,7263992,5658437,8140937,5281746,388776,1096884,3274448,1239743,386767,2942615,3905934,3747474,2493392,3743640,3850721,368539,980371,1339635,4207966,3392981,1286835,1191276,2570825,372034,3639833,602867,1904229,2627986,2737046,3802236,369895,1165246,1518187,1127188,1075520,3824879],["HistoTools:  A suite of digital pathology tools for quality control, annotation and dataset identification","Trinity: Transcriptome assembly for genetic and functional analysis of cancer","NDEx - the Network Data Exchange A Network Commons for Biologists","TCIA Sustainment and Scalability - Platforms for Quantitative Imaging Informatics in Precision Medicine","Cancer Genomics: Integrative and Scalable Solutions in R/Bioconductor","STAN-CT: Standardization and Normalization of CT images for Lung Cancer Patients","The Federated Tumor Segmentation (FeTS) platform: An intuitive tool facilitating secure multi-institutional collaboration","Advancing Cancer Pharmacoepidemiology Research Through EHRs and Informatics","Network Analysis for a Data-Driven Approach to Cancer Care","Data-driven QSP software for personalized colon cancer treatment","Standardized and Genome-Wide Clinical Interpretation of Complex Genotypes for Cancer Precision Medicine","TCPA: an Integrated Bioinformatics Resource for Functional Cancer Proteomic Data","Scalable Clinical Decision Support for Individualized Cancer Risk Management","Informatics Tools for High-throughput Analysis of Cancer Mutations","Monitoring tumor subclonal heterogeneity over time and space","A Federated Galaxy for user-friendly large-scale cancer genomics research","Personalized dosimetry for liver cancer radioembolization using fluid dynamics simulation","A Unified Machine Learning Package for Cancer Diagnosis","Eliminating variants of uncertain significance in BRCA1, BRCA2 and beyond","The TOPAS Tool for Particle Simulation, a Monte Carlo Simulation Tool for Physics, Biology and Clinical Research","Supporting and evolving Gene Set Enrichment Analysis and the Molecular Signatures Database for cancer research","Bioinformatics Technology to Characterize Tumor Infiltrating Immune Repertoires","Distributed Learning of Deep Learning Models for Cancer Research","WebMeV: A Robust Platform for Intuitive Genomic Data Analysis","Development and Dissemination of KiNet: A Novel Imaging Informatics Tool for Gastrointestinal and Pancreatic Neuroendocrine Tumors","Developing Informatics Technologies to Model Cancer Gene Regulation","A statistical framework to systematically characterize cancer driver mutations in noncoding genomic regions","The PDX Integrator- Re-purposing KOMP2 informatics to widely disseminate data from Patient Derived Xenografts","Detection of somatic, subclonal and mosaic CNVs from sequencing","Developing the JBrowse Genome Browser to Visualize Structural Variants and Cancer Genomics Data","The Integrative Genomics Viewer (IGV): visualization supporting cancer research","Semi-supervised Algorithms for Risk Assessment with Noisy EHR Data","Tools for annotating mutations in the 3D cancer genome","Open-access X-ray and CT simulation toolkit for research in cancer imaging and dosimetry","Informatics resources for liquid biopsy research","Computational pathology software for integrative cancer research with three-dimensional digital slides","Advanced Development and Dissemination of EMERSE for Cancer Phenotyping from Medical Records"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>core_project_num<\/th>\n      <th>self_citn<\/th>\n      <th>n_years<\/th>\n      <th>amount<\/th>\n      <th>project_title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":[1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script><p>One can view collaboration as the number of projects that cite a
particular project…</p>
<div class="sourceCode" id="cb32"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">project_project_citations</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">core_project_num</span> <span class="op">!=</span> <span class="va">cited_by</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/group_by.html" class="external-link">group_by</a></span><span class="op">(</span><span class="va">core_project_num</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/summarise.html" class="external-link">summarize</a></span><span class="op">(</span>n_collaborators <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/length.html" class="external-link">length</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">cited_by</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/desc.html" class="external-link">desc</a></span><span class="op">(</span><span class="va">n_collaborators</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">project_titles_h</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; Joining with `by = join_by(core_project_num)`</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-c07b9fe26dd374628d5f" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-c07b9fe26dd374628d5f">{"x":{"filter":"none","vertical":false,"data":[["U24CA180922","U24CA209851","U24CA204817","U24CA184427","U24CA180996","U24CA215109","U24CA210004","U24CA231877","U24CA237719","U01CA242871","U01CA231840","U01CA242954","U24CA209999","U24CA220341","R21CA231911","U01CA239055","U01CA242936","U24CA194215","U24CA204800","U24CA220242","U24CA231846","U24CA237617"],[12,10,9,7,5,5,4,4,4,3,2,2,2,2,1,1,1,1,1,1,1,1],["Trinity: Transcriptome assembly for genetic and functional analysis of cancer","TCPA: an Integrated Bioinformatics Resource for Functional Cancer Proteomic Data","Informatics Tools for High-throughput Analysis of Cancer Mutations","NDEx - the Network Data Exchange A Network Commons for Biologists","Cancer Genomics: Integrative and Scalable Solutions in R/Bioconductor","TCIA Sustainment and Scalability - Platforms for Quantitative Imaging Informatics in Precision Medicine","The Integrative Genomics Viewer (IGV): visualization supporting cancer research","A Federated Galaxy for user-friendly large-scale cancer genomics research","Standardized and Genome-Wide Clinical Interpretation of Complex Genotypes for Cancer Precision Medicine","The Federated Tumor Segmentation (FeTS) platform: An intuitive tool facilitating secure multi-institutional collaboration","Network Analysis for a Data-Driven Approach to Cancer Care","Eliminating variants of uncertain significance in BRCA1, BRCA2 and beyond","Monitoring tumor subclonal heterogeneity over time and space","Supporting and evolving Gene Set Enrichment Analysis and the Molecular Signatures Database for cancer research","STAN-CT: Standardization and Normalization of CT images for Lung Cancer Patients","HistoTools:  A suite of digital pathology tools for quality control, annotation and dataset identification","Computational pathology software for integrative cancer research with three-dimensional digital slides","Advancing Cancer Pharmacoepidemiology Research Through EHRs and Informatics","Scalable Clinical Decision Support for Individualized Cancer Risk Management","Detection of somatic, subclonal and mosaic CNVs from sequencing","WebMeV: A Robust Platform for Intuitive Genomic Data Analysis","Developing Informatics Technologies to Model Cancer Gene Regulation"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>core_project_num<\/th>\n      <th>n_collaborators<\/th>\n      <th>project_title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":1}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script><p>…or the number of projects a project cites</p>
<div class="sourceCode" id="cb33"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">project_project_citations</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">core_project_num</span> <span class="op">!=</span> <span class="va">cited_by</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/group_by.html" class="external-link">group_by</a></span><span class="op">(</span><span class="va">cited_by</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/summarise.html" class="external-link">summarize</a></span><span class="op">(</span>n_collaborators <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/length.html" class="external-link">length</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">core_project_num</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/desc.html" class="external-link">desc</a></span><span class="op">(</span><span class="va">n_collaborators</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">project_titles_h</span>, by <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span>cited_by <span class="op">=</span> <span class="st">"core_project_num"</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-0b9f5df62a962ee6a744" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0b9f5df62a962ee6a744">{"x":{"filter":"none","vertical":false,"data":[["U24CA237719","U24CA220457","U24CA231846","U24CA231877","R21CA242861","U01CA231840","U01CA239055","U01CA242954","U24CA180996","U24CA209851","U24CA220341","U01CA226187","U24CA180922","U24CA204817","U01CA226196","U01CA242871","U01CA242879","U24CA194215","U24CA220441","U01CA231782","U01CA242936","U24CA184427","U24CA204863","U24CA209999","U24CA220242"],[7,6,6,6,5,4,4,4,4,4,4,3,3,3,2,2,2,2,2,1,1,1,1,1,1],["Standardized and Genome-Wide Clinical Interpretation of Complex Genotypes for Cancer Precision Medicine","The cBioPortal for Cancer Genomics","WebMeV: A Robust Platform for Intuitive Genomic Data Analysis","A Federated Galaxy for user-friendly large-scale cancer genomics research","A statistical framework to systematically characterize cancer driver mutations in noncoding genomic regions","Network Analysis for a Data-Driven Approach to Cancer Care","HistoTools:  A suite of digital pathology tools for quality control, annotation and dataset identification","Eliminating variants of uncertain significance in BRCA1, BRCA2 and beyond","Cancer Genomics: Integrative and Scalable Solutions in R/Bioconductor","TCPA: an Integrated Bioinformatics Resource for Functional Cancer Proteomic Data","Supporting and evolving Gene Set Enrichment Analysis and the Molecular Signatures Database for cancer research","Tools for annotating mutations in the 3D cancer genome","Trinity: Transcriptome assembly for genetic and functional analysis of cancer","Informatics Tools for High-throughput Analysis of Cancer Mutations","Bioinformatics Technology to Characterize Tumor Infiltrating Immune Repertoires","The Federated Tumor Segmentation (FeTS) platform: An intuitive tool facilitating secure multi-institutional collaboration","Distributed Learning of Deep Learning Models for Cancer Research","Advancing Cancer Pharmacoepidemiology Research Through EHRs and Informatics","Developing the JBrowse Genome Browser to Visualize Structural Variants and Cancer Genomics Data","A Unified Machine Learning Package for Cancer Diagnosis","Computational pathology software for integrative cancer research with three-dimensional digital slides","NDEx - the Network Data Exchange A Network Commons for Biologists","Advanced Development and Dissemination of EMERSE for Cancer Phenotyping from Medical Records","Monitoring tumor subclonal heterogeneity over time and space","Detection of somatic, subclonal and mosaic CNVs from sequencing"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>cited_by<\/th>\n      <th>n_collaborators<\/th>\n      <th>project_title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":1}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="section level2">
<h2 id="session-information">Session information<a class="anchor" aria-label="anchor" href="#session-information"></a>
</h2>
<div class="sourceCode" id="cb34"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; R version 4.2.2 (2022-10-31)</span></span>
<span><span class="co">#&gt; Platform: x86_64-pc-linux-gnu (64-bit)</span></span>
<span><span class="co">#&gt; Running under: Ubuntu 22.04.2 LTS</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Matrix products: default</span></span>
<span><span class="co">#&gt; BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3</span></span>
<span><span class="co">#&gt; LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.20.so</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; locale:</span></span>
<span><span class="co">#&gt;  [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       </span></span>
<span><span class="co">#&gt;  [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   </span></span>
<span><span class="co">#&gt;  [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          </span></span>
<span><span class="co">#&gt; [10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; attached base packages:</span></span>
<span><span class="co">#&gt; [1] stats     graphics  grDevices utils     datasets  methods   base     </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; other attached packages:</span></span>
<span><span class="co">#&gt; [1] DT_0.27                 ggplot2_3.4.1           grantpubcite_0.0.0.9002</span></span>
<span><span class="co">#&gt; [4] dplyr_1.1.0            </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; loaded via a namespace (and not attached):</span></span>
<span><span class="co">#&gt;  [1] lattice_0.20-45   rprojroot_2.0.3   digest_0.6.31     utf8_1.2.3       </span></span>
<span><span class="co">#&gt;  [5] R6_2.5.1          evaluate_0.20     httr_1.4.5        highr_0.10       </span></span>
<span><span class="co">#&gt;  [9] pillar_1.8.1      rlang_1.0.6       curl_5.0.0        jquerylib_0.1.4  </span></span>
<span><span class="co">#&gt; [13] Matrix_1.5-1      rmarkdown_2.20    pkgdown_2.0.7     textshaping_0.3.6</span></span>
<span><span class="co">#&gt; [17] desc_1.4.2        labeling_0.4.2    splines_4.2.2     readr_2.1.4      </span></span>
<span><span class="co">#&gt; [21] stringr_1.5.0     htmlwidgets_1.6.1 bit_4.0.5         munsell_0.5.0    </span></span>
<span><span class="co">#&gt; [25] compiler_4.2.2    xfun_0.37         pkgconfig_2.0.3   systemfonts_1.0.4</span></span>
<span><span class="co">#&gt; [29] mgcv_1.8-41       htmltools_0.5.4   tidyselect_1.2.0  tibble_3.1.8     </span></span>
<span><span class="co">#&gt; [33] fansi_1.0.4       crayon_1.5.2      tzdb_0.3.0        withr_2.5.0      </span></span>
<span><span class="co">#&gt; [37] rjsoncons_1.0.0   grid_4.2.2        nlme_3.1-160      jsonlite_1.8.4   </span></span>
<span><span class="co">#&gt; [41] gtable_0.3.1      lifecycle_1.0.3   magrittr_2.0.3    scales_1.2.1     </span></span>
<span><span class="co">#&gt; [45] cli_3.6.0         stringi_1.7.12    vroom_1.6.1       cachem_1.0.7     </span></span>
<span><span class="co">#&gt; [49] farver_2.1.1      fs_1.6.1          bslib_0.4.2       ellipsis_0.3.2   </span></span>
<span><span class="co">#&gt; [53] ragg_1.2.5        generics_0.1.3    vctrs_0.5.2       tools_4.2.2      </span></span>
<span><span class="co">#&gt; [57] bit64_4.0.5       glue_1.6.2        purrr_1.0.1       hms_1.1.2        </span></span>
<span><span class="co">#&gt; [61] crosstalk_1.2.0   parallel_4.2.2    fastmap_1.1.1     yaml_2.3.7       </span></span>
<span><span class="co">#&gt; [65] colorspace_2.1-0  memoise_2.0.1     knitr_1.42        sass_0.4.5</span></span></code></pre></div>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by Martin Morgan.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
